%%%-------------------------------------------------%%%
%%% Sub document for material and methods %%%
%%%-------------------------------------------------%%%

\chapter{Material And Methods}
Within this section percentages refer to volume per volume (v/v) percentages unless otherwise specified.
%%%% materials (chemicals, inbstrumenmts, software etc)
\section{Materials}
\subsection{Chemicals}
Enzymes and buffers used for molecular cloning were obtained from Life Technologies (Darmstadt, Germany), unless otherwise noted. 
Flavonoid HPLC standards were purchased from Extrasynthese (Genay, France). 
Deuterated solvents were aquired from Deutero GmbH (Kastellaun, Germany). 
Solvents, purchased from VWR (Poole, England), were distilled in-house before use. \\
All other chemicals were obtained from either Sigma-Aldrich (Steinheim, Germany), Applichem (Darmstadt, Germany), Carl Roth (Karlsruhe, Germany) or Merck (Darmstadt, Germany).

\subsection{Commonly used solutions and buffers}

\begin{lstTabularx}{\linewidth}{>{\small}p{0.25\textwidth}<{\small}X}
  50$\times$ 5052 & 25\,\% glycerol, 2.5\,\% (w/v) glucose, 10\,\% (w/v) \textalpha-lactose \\
  binding buffer & 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 2.5\,mM imidazole pH 7\\
  elution buffer & 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 250\,mM imidazole pH 7 \\
  lysis buffer & 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 2.5\,mM imidazole, 0.2\,\% Tween-20 pH 7\\
  1\,M \gls{mmt} pH 4 (10$\times$) & \SI{26.8}{\g\per\l} \textsc{L}-malic acid, \SI{78.1}{\g\per\l} \gls{mes}, \SI{26.8}{\g\per\l} \gls{tris}, 2.1\,\% 10\,M HCl \\
    1\,M \gls{mmt} pH 9 (10$\times$) & \SI{26.8}{\g\per\l} \textsc{L}-malic acid, \SI{78.1}{\g\per\l} \gls{mes}, \SI{26.8}{\g\per\l} \gls{tris}, 6.7\,\% 10\,M NaOH \\
  20$\times$ \gls{nps} & 1\,M \cf{Na2HPO4}, 1\,M \cf{KH2PO4}, 0.5\,M \cf{(NH4)2SO4} \\
  5$\times$ SDS sample buffer &  10\,\% (w/v) \gls{sds}, 10\,mM \textbeta-mercapto ethanol, 20\,\% glycerol, 0.2\,M Tris/HCl pH\,6.8, 0.05\,\% (w/v) bromophenolblue\\
  1000$\times$ trace elements & 50\,mM \cf{FeCl3}, 20\,mM \cf{CaCl2}, 10\,mM \cf{MnCl2}, 10\,mM \cf{ZnSO4}, 2\,mM \cf{CoCl2}, 2\,mM \cf{CuCl2}, 2\,mM \cf{NiCl2}, 2\,mM \cf{Na2MoO4}, 2\,mM \cf{Na2SeO3}, 2\,mM \cf{H3BO3} \\
  \addtocounter{table}{-1} 
\end{lstTabularx}

\subsubsection{Preparation of \glsentryfirst{des}}

\Gls{des} were prepared by adding each component in a round-bottom flask with a stirrer and stirring the mixture at \degC{50} with intermittent sonication treatments until a clear solution was obtained.

 \ostTableEnv[label=tab:des]{%
   \begin{tabularx}{\linewidth}{X p{0.3\textwidth} X p{0.3\textwidth}}
   \toprule
   name & composition & mole ratio & mass fraction (w/w) \\
   \midrule
   PCH & propane-1,2-diol \newline choline chloride \newline water & 1:1:1 &  0.326 \newline 0.597 \newline 0.077 \\
   GCH & \textsc{D}-glucose \newline choline chloride \newline water & 2:5:5 & 0.314 \newline 0.608 \newline 0.078 \\
   \bottomrule
 \end{tabularx}
 }{%
   \gls{nades}-mixtures
 }

\subsection{Culture media used to grow bacteria}

\begin{lstTabularx}{\linewidth}{>{\small}p{0.25\textwidth}<{\small}X}
  LB-medium & \SI{10}{\g\per\l} NaCl, \SI{10}{\g\per\l} tryptone, \SI{5}{\g\per\l} yeast extract, pH 7.5 \\
  LB-agar & LB + 1.5\,\% (w/v) agar-agar \\ 
  TB-medium & \SI{12}{\g\per\l} tryptone, \SI{24}{\g\per\l} yeaxst extract, 0.4\,\% glycerol, 72\,mM \cf{K2HPO4}, 17\,mM \cf{KH2PO4}\\
  ZY & \SI{10}{\g\per\l} tryptone, \SI{5}{\g\per\l} yeast extract \\ 
  ZYP-5052 & in proportions: 0.928 ZY, 0.05 20$\times$ NPS, 0.02 50$\times$ 5052, 0.002 1\,M \cf{MgSO4}, 0.0002 1000$\times$ trace elements\\
  \addtocounter{table}{-1} 
\end{lstTabularx}

  
\subsection{Bacterial strains}

\subsubsection{\textit{E.coli}}
\begin{lstTabularx}{\linewidth}{>{\small}p{0.25\textwidth}<{\small}X}
  BL21(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Invitrogen, Karslruhe (Germany) \\
  C41(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Lucigen, Wisconsin (USA) \\
  C43(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3)\newline Lucigen, Wisconsin (USA) \\
  DH5$\alpha$ & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{lacZYA-argF}) U169 \textit{recA}1 \textit{endA}1 \textit{hsdR}17(r$_K^-$m$_K^+$) \textit{phoA supE}44 $\lambda^-$ \textit{thi}-1 \textit{gyrA}96 \textit{relA}1 \newline Invitrogen, Karlsruhe (Germany) \\
  JM110 & \textit{rpsL thr leu thi lacY galK galT ara tonA tsx dam dcm glnV}44 $\Delta$(\textit{lac-proAB}) e14- [F' \textit{traD}36 \textit{proAB}$^+$ \textit{lacI}$^q$ \textit{lacZ}$\Delta$M15] \textit{hsdR}17(r$_K^-$m$_K^+$) \newline Martin-Luther-University Halle-Wittenberg \\
  JW1593 \newline (BW25113 derivative) & \textit{rrnB} $\Delta$\textit{lacZ}4787 \textit{HsdR}514 $\Delta$(\textit{araBAD})568 \textit{rph}-1 $\Delta$\textit{ydgG} (Kan$^R$) \newline Keio Collection, National Institute of Genetics (Japan) \\
  MG1655 & F$^-$ $\lambda^-$ \textit{ilvG}$^-$ \textit{rfb}-50 \textit{rph}-1 \newline DSMZ, Hamburg (Germany) \\
  One Shot TOP10 & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{mrr-hsdRMS-mcrBC}) \textit{recA}1 \textit{endA}1 \textit{mcrA} $\Delta$\textit{lacX}74 \textit{araD}139 $\Delta$(\textit{ara-leu})7697 \textit{galU galK rpsL} (Str$^R$) $\lambda^-$ \textit{nupG} \newline Invitrogen, Karlsruhe (Germany) \\
  Origami(DE3) & $\Delta$(\textit{ara-leu})7697 $\Delta$\textit{lacX}74 $\Delta$\textit{phoA} \textit{Pvull phoR araD}139 \textit{ahpC galE galK rpsL} F'[\textit{lac + lacI q pro}] (DE3)\textit{gor}522::\textit{Tn}10 \textit{trxB} (Kan$^R$, Str$^R$, Tet$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) pLysS & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pLysSRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  T7 Express & \textit{fhuA}2 \textit{lacZ::T}7 \textit{gene}1 \textit{[lon] ompT gal sulA}11 \textit{R}(\textit{mcr-}73\textit{::miniTn}10--Tet$^S$)2 \textit{[dcm] R}(\textit{zgb}-210\textit{::Tn}10--Tet$^S$) \textit{endA}1 $\Delta$(\textit{mcrC-mrr})114\textit{::IS}10 \newline NEB, Massachusetts (USA) \\
\addtocounter{table}{-1} 
\end{lstTabularx}

\subsubsection{\textit{Agrobacterium tumefaciens}}
\begin{lstTabularx}{\linewidth}{>{\small}p{0.25\textwidth}<{\small}X}
GV3101 & chromosomal background: C58, marker gene: \textit{rif}, \gls{tip}: cured, opine: nopaline \newline Sylvestre Marillonet, IPB \\
\addtocounter{table}{-1} 
\end{lstTabularx}


\subsection{Plasmids}
%% \ostTableEnv[label = tab:plasmids]{%
%%   \begin{tabularx}{\textwidth}{p{0.3\textwidth}X}
%%     \toprule
%% 	name & supplier/source \\ 
%%     \midrule
%%     pACYCDuet-1 & Merck, Darmstadt (Germany)  \\
%%     pCDFDuet-1 & Merck, Darmstadt (Germany) \\
%%     pET-20b(+) & Merck, Darmstadt (Germany) \\
%%     pET-28a(+) & Merck, Darmstadt (Germany) \\
%%     pET-32a(+) & Merck, Darmstadt (Germany) \\
%%     pET-41a(+) & Merck, Darmstadt (Germany) \\
%%     pQE30 & QIAGEN, Hilden (Germany) \\
%%     pUC19 & Invitrogen, Karslruhe (Germany) \\
%%     pQE30-PFOMT & Thomas Vogt (\gls{ipb}, Halle, Germany) \\
%%     \bottomrule
%%   \end{tabularx}
%% }{%
%% Plasmids used in this work.
%% }

\begin{lstTabularx}[1]{\linewidth}{p{0.3\textwidth}X}
  \caption{Plasmids used in this work.} \\
  \label{tab:plasmids}
  \small
  \hfill \\
     \toprule
	name & supplier/source \\ 
     \midrule
     \endfirsthead
     \toprule
	name & supplier/source \\ 
     \midrule
     \endhead
     pACYCDuet-1 & Merck, Darmstadt (Germany)  \\
     pCDFDuet-1 & Merck, Darmstadt (Germany) \\
     pET-20b(+) & Merck, Darmstadt (Germany) \\
     pET-28a(+) & Merck, Darmstadt (Germany) \\
     pET-32a(+) & Merck, Darmstadt (Germany) \\
     pET-41a(+) & Merck, Darmstadt (Germany) \\
     pQE30 & QIAGEN, Hilden (Germany) \\
     pUC19 & Invitrogen, Karslruhe (Germany) \\
     \bottomrule
\end{lstTabularx}

\subsection{Oligonucleotides and synthetic genes}
Oligonucleotides and primers were ordered from Eurofins Genomics (Ebersberg, Germany). The purity grade was \emph{high purity salt free} (HPSF).
Synthetic genes or gene fragments were obtained from GeneArt\textsuperscript{\textregistered} (Life Technologies, Darmstadt, Germany) or Eurofins Genomics (Ebersberg, Germany).

\ostTableEnv[label = tab:primers]{%
  \begin{tabularx}{\textwidth}{X >{\footnotesize}p{0.70\textwidth} X}
    \toprule
    	name & sequence (5'$\rightarrow$3') & cloning site \\ 
    \midrule
%    	T7 & TAATACGACTCACTATAGGGT & n/a \\
%       T7.term & GCTAGTTATTGCTCAGCGGT & n/a \\
        somt1 & TT\underline{G AAG AC}A AAA TGG CTT CTT CAT TAA ACA ATG GCC G & BpiI \\
        somt2 & TT\underline{G AAG AC}A AGG ACA CCC CAA ATA CTG TGA GAT CTT CC & BpiI \\
        somt3 & TT\underline{G AAG AC}A AGT CCT TAG GAA CAC CTT TCT GGG AC  & BpiI \\
        somt4 & TT\underline{G AAG AC}A AAA GCT CAA GGA TAG ATC TCA ATA AGA GAC & BpiI \\
        pfomt1.fw & CAG AGA GGC cTA TGA GAT TGG CTT GC & \\
        pfomt1.rv & GCA AGC CAA TCT CAT AgG CCT CTC TG & \\
        pfomt2.fw & \underline{CAT ATG} GAT TTT GCT GTG ATG AAG CAG GTC & NdeI \\
        pfomt2.rv & \underline{GAA TTC} AAT AAA GAC GCC TGC AGA AAG TG & EcoRI \\
        pRha1.fw & CTC TAG C\underline{AG ATC T}CG GTG AGC ATC ACA TCA CCA CAA TTC & BglII \\
        pRha1.rv & CAA TTG A\underline{GG ATC C}CC ATT TTA ACC TCC TTA GTG & BamHI \\
        pUC1.fw & GCG TAT TGG Gag aTC TTC CGC TTC CTC & \\
        pUC1.rv & GAG GAA GCG GAA GAt ctC CCA ATA CGC & \\
    \bottomrule
  \end{tabularx}
}{%
Primers used in this work. Recognition sites for endonucleases are underlined. Positions used for site directed mutagenesis are in lower case font.
}

\subsection{Instruments}

%\begin{small}
\begin{tabularx}{\linewidth}{>{\small}p{0.3\textwidth}>{\small}X}
  \glssymbol{cd}-spectrometer & Jasco J-815 (Eaton, USA) \\
  electrophoresis (horizontal) & Biometra Compact XS/S (Göttingen, Germany)\\
  electrophoresis (vertical) & Biometra Compact M (Göttingen, Germany) \newline Biometra Minigel-Twin (Göttingen, Germany) \\
  \glsentrytext{fplc} & ÄKTA purifier (GE Healthcare, Freiburg, Germany) \\
  \glsentrytext{gcms} & GC-MS-QP2010 Ultra (Shimadzu, Duisburg, Germany)\\
  \glsentrytext{hplc} & VWR-Hitachi LaChrom Elite (VWR, Darmstadt, Germany) \\
  \glsentrytext{itc} & MicroCal iTC200 (Malvern, Worcestershire, UK) \\
  plate-reader & SpectraMax M5 (Molecular Devices, Biberach, Germany) \\
  \glsentrytext{nmr}-spectrometer & Varian Unity 400 (Agilent, Böblingen, Germany) \newline Varian VNMRS 600 (Agilent, Böblingen, Germany) \\
  photospectrometer & Eppendorf Biophotometer Plus (Hamburg, Germany) \newline JASCO V-560 (Eaton, USA) \newline Colibri Microvolume Spectrometer (Biozym, Hess. Oldendorf, Germany) \\
  centrifuges & Eppendorf 5424 (Hamburg, Germany) \newline Hettich Mikro 120 (Kirchlengern, Germany) \newline Beckman Avanti J-E, Beckman Allegra X-30R (Krefeld, Germany)  \\
  centrifuge rotors & Beckman JA-10, JA-16.250, JS-4.3 (Krefeld, Germany)    
\end{tabularx}
%\end{small}

\subsection{Software}
All mathematical and statistical computations and graphics were done with the \verb+R+ software (versions 3.1.X, \url{http://cran.r-project.org/}) \citep{RCoreTeam2015}. Visualizations of macromolecules were arranged using the PyMol Molecular Graphics System, version 1.7.0.0 (Schrödinger, New York, USA).\\
Physicochemical calculations and calculations of different molecular descriptors were performed using Marvin Beans 15.4.13.0 (ChemAxon, Budapest, Hungary) and Molecular Operating Environment 2008.10 (Chemical Computing Group, Montreal, Canada).
Special software used for X-ray crystal structure solution is discussed seperately in the corresponding section (\ref{sec:xraymethods}).

%%%% Microbiologicyl Methods
\section{Microbiology}


%%%% Molecular Biology methods
\section{Molecular Biology}
\label{sec:molbio}
Basic molecular biology methods like \gls{pcr}, DNA restriction/ligation, DNA gel electrophoresis, preparation of competent cells and transformation were performed based on the protocols summarized by Sambrook and Russell \citep{Sambrook2001}.\\
Plasmid DNA was isolated using the QIAprep\textsuperscript{\textregistered} Spin Miniprep Kit (QIAGEN, Hilden, Germany) according to the manufacturer's instructions. \\
In vitro site-directed mutatgenesis was set-up according to the protocol of the \emph{QuikChange\textsuperscript{\texttrademark} Site-Directed Mutagenesis} kit \cite{Technologies} offered by Agilent Technologies (Santa Clara, USA). \\
Nucleotide fragments obtained by \gls{pcr}, restriction/ligation procedures or excision from electrophoresis gels were purified and concentrated using the \emph{Nucleospin Gel and PCR Clean-up} kit provided by Machery-Nagel (Düren, Germany) according to the instructions provided by the manufacturer.

\subsection{Golden Gate Cloning}
\label{sec:GGcloning}
The Golden Gate cloning procedure is a one-pot method, meaning the restriction digestion and ligation are carried out in the same reaction vessel at the same time \cite{Kondou2011,Engler2008}. Consequently \gls{pcr}-fragments, destination vector, restriction endonuclease and ligase are added together in this reaction. The methodology employs type II restriction enzymes, which together with proper design of the fragments allow for a ligation product lacking the original restriction sites.  \\
For digestion/ligation reactions of fragments containing BpiI sites, 20\,fmol of each fragment or vector, together with 5\,U of BpiI and 5\,U of T4 ligase were combined in a total volume of \ul{15} 1$\times$ ligase buffer.
For fragments to be cloned via BsaI sites, BpiI in the above reaction was substituted by 5\,U BsaI. \\
The reaction mixture was placed in a thermocycler and inbcubated at \degC{37} for 2\,min and \degC{16} for 5\,min. These two first steps were repeated 50 times over. Finally, the temperature was raised to \degC{50} (5\,min) and \degC{80} (10\,min) to inactivate the enzymes.

\subsection{Subcloning of genes}
All subcloning procedures were performed according to \autoref{sec:molbio} and specifically \autoref{sec:GGcloning}.
Specific steps for the subcloning of any genes discussed can be found in the appendix (p.\pageref{tab:molecularcloning}).
The \textit{pfomt} gene was subcloned from the pQE-30 vector kindly provided by Thomas Vogt (\gls{ipb}, Halle, Germany) into the pET-28a(+) vector.
The \textit{somt-2} gene was subcloned from the pQE-30 vector kindly provided by Martin Dippe (\gls{ipb}, Halle, Germany) into the pET-28-MC vector.


%% \subsection{Cloning of \textit{p-lsr} reporter constructs} --> evtl Ergebnisse



%%%% protein biochemical methods
\section{Protein biochemistry}
Stock solutions of antibiotics, IPTG or sugars were prepared according to the pET System Manual by Novagen \citep{Novagen2011}, unless otherwise noted.

\subsection{Protein production test (expression test)}
The heterologous production of proteins in \eco was assessed in a small scale protein production test, henceforth called expression test.
Single colonies of \eco transformed with the constructs to be studied were used to inoculate a 2\,ml starter culture in \gls{lb}-medium containing the appropriate antibiotics. 
The concentrations of antiobiotics used was as follows:
\ugperml{200} ampicillin, \ugperml{150} kanamycin, \ugperml{50} chloramphenicol, \ugperml{20} tetracycline. \\
The starter culture was allowed to grow at \degC{37} and 200\,rpm over night.
A 5\,ml sampling culture of the medium to be studied containing the appropriate antibiotics was prepared. The media tested included \gls{lb}, \gls{tb} and auto-induction media like ZYP-5052.
The sampling culture was inoculated to an \odx{600} of 0.075 using the starter culture and incubated at different temperatures and 200\,rpm in a shaking incubator.
1\,mM \gls{iptg} was added when the \odx{600} reached 0.6-0.8, if appropriate for the studied construct.
1\,ml samples were removed after different times of incubation (e.g. 4, 8, 12 hours), subfractionated (\ref{sec:subfractionation}) and analyzed via \gls{sds}-\gls{page} (\ref{sec:sdspage}).\\
Exact specifications of growth conditions (e.g. temperature, time, constructs) are discussed in the individual sections.

\subsection{Protein subfractionation}
\label{sec:subfractionation}
The protein subfractionation procedure described herein was adapted from the protocol described in the pET Manual \citep{Novagen2011}. 
Overall 5 protein subfractions can be obtained, including \emph{total cell protein, culture supernatant (medium) protein, periplasmic protein, solube cytoplasmic protein} and \emph{insoluble protein}. \\
The \odx{600} of the culture sample was measured and the cells harvested by centrifugation at \timesg{10000}, \degC{4} for 5\,minutes.
The protein in the supernatant medium was concentrated by precipitation with \gls{tca} (\ref{sec:tcaprecipitation}) for \sdspage analysis.
The periplasmic protein was prepared (\ref{sec:ppprep}) and also concentrated by \gls{tca} precipitation for \sdspage.
Cells were lysed by resuspending the cell pellet in (\odx{600}$\times~\mathrm{V}~\times~50$)\,\si{\micro\litre} of \gls{bper} and vortexing vigorously for 30\,s. 
The suspension was incubated at \gls{rt} for 30\,min to assure complete lysis. To separate insoluble protein and cell debris from the soluble cytosolic protein, the suspension was centrifuged at \timesg{10000} and \degC{4} for 10\,min.
Soluble cytoplasmic protein was contained in the supernatant, whereas insoluble protein remained in the pellet. For \sdspage analysis of the insoluble protein, the pellet was resuspended in the same volume of \gls{bper}.
To obtain only the total cell protein fraction, the preparation of periplasmic and soluble cytosolic protein was omitted. Sample volumes of \ul{10} of each fraction were used for \sdspage analysis.


\subsection{Protein sample concentration by \glsentrytext{tca} precipitation}
\label{sec:tcaprecipitation}
Diluted protein samples were concetrated by \gls{tca} precipitation in microcentrifuge tubes.
Therefore 0.1\,\gls{v} of 100\,\%~(w/v) \gls{tca} in water was added to the clarified sample, which was then vortexed for 15\,s and placed on ice for a minimum of 15\,min.
The sample was centrifuged at \timesg{14000}, \degC{4} for 15\,min. The supernatant was discarded and the pellet was washed twice with 0.2\,\gls{v} ice-cold acetone.
The acetone was removed and the pellet set to air-dry in an open tube.
After drying, the protein pellet was resuspended in 0.1\,\gls{v} \gls{pbs} containing 1\,$\times$ \gls{sds}-sample buffer by heating to \degC{85} and vigorous vortexing, to achieve a 10\,$\times$ concentration.
After resuspension the sample was analyzed by \sdspage or stored at \degC{-20} until use.

\subsection{Preparation of periplasmic protein}
\label{sec:ppprep}
Target proteins may be directed to the periplasmic space by N-terminal signal sequences like \textit{pelB} or \textit{DsbA/C} \citep{Makrides1996}. The periplasma is, other than the cytosol, an oxidizing environment and often used for the production of proteins containing dilsufide linkages.
The preparation of periplasmic protein was accomplished by an osmotic shock protocol modified from Current Protocols in Molecular Biology \citep{Ausubel2008}.\\
The cell pellet was resuspended in the same volume as the culture sample of 30\,mM tris-HCl, 20\,\% (w/v) sucrose, pH\,8 and 1\,mM \gls{edta} was added. The suspension was stirred for 10\,min at \gls{rt} and the cells were collected by centrifugation at \timesg{10000}, \degC{4} for 10\,min.
The supernatant was discarded and the cell pellet was resuspended in the same volume of ice-cold 5\,mM \cf{MgSO4}.
The suspension was stirred for 10\,min on ice, while the periplasmic proteins were released into the solution.
The cells were collected by centrifugation as before. Periplasmic proteins were contained in the supernatant.

\subsection{Discontinous SDS-polyacrylamide gel electrophoresis (\glsentrytext{sds}-\glsentrytext{page})}
\label{sec:sdspage}

The analysis of samples via \sdspage was realized via the discontinous system first described by Laemmli, which allows separation of proteins based on their electrophoretic mobility, which in turn depends on their size \citep{Laemmli1970}.\\
The \sdspage procedure was carried out according to standard protocols described by Sambrook and Russell \citep{Sambrook2001}.
Very dilute and/or samples with high ionic strength were concentrated and/or desalted by the \gls{tca} precipitation procedure described in \autoref{sec:tcaprecipitation}.
Generally a 10\,\% (acrylamide/bisacrylamide) running gel combined with a 4\,\% stacking gel was used. Reducing \sdspage sample buffer was added to the protein sample to be analyzed, whereafter the sample was heated to \degC{95} for 5\,min, to allow for total unfolding of the protein.
After cooling to \gls{rt} the samples were transferred into the gel pockets for analysis.
The \emph{PageRuler\textsuperscript{\texttrademark} Prestained Protein Ladder} (Life Technologies GmbH, Darmstadt, Germany) was used as a \gls{mw} marker and run alongside every analysis as a reference. 
Gels were stained using a staining solution of 0.25\,\% Coomassie Brilliant Blue G-250 (w/v) in water:methanol:acetic acid (4:5:1) and destained by treatment with water:methanol:acetic acid (6:3:1). 

\subsection{Buffer change of protein samples}
\label{sec:rebuffer}
The buffer in protein samples was exchanged either by dialysis, or by centrifugal filter concentrators (Amicon\textsuperscript{\textregistered} Ultra Centrifugal Filter; Merck, Darmstadt, Germany).
Large volumes of highly concentrated protein solutions were preferably dialyzed.
Respectively, very dilute samples were concentrated and rebuffered using centrifugal concentrators. \\
Dialysis was carried out at least twice against a minimum of 100 times the sample volume.
Dialysis steps were carried out at \gls{rt} for 2 hours, or over-night at \degC{4}.
Centrifugal concentrators were used according to the manufacturers instructions.

\subsection{Production of recombinant protein}
\subsubsection{Heterologous production of \glsentrytext{pfomt}}
\Gls{pfomt} was produced as a N-terminally (His)$_6$-tagged fusion protein.
A 2\,ml starter culture of \gls{lb} containing \ugperml{100} kanamycin was inoculated with a single colony of \eco BL21(DE3) transformed with pET28-pfomt and incubated at \degC{37}, 220\,rpm for 6\,hours.
The main culture (\gls{zyp} containing \ugperml{200} kanamycin) was inoculated with the starter culture such that \odx{600} was 0.05.
The culture was incubated in a shaking incubator at \degC{37}, 220\,rpm over night ($\approx$16\,h).
Due to the autoinducing nature of the \gls{zyp} medium, addition of \gls{iptg} was not neccesary.
Cells were harvested by centrifugation at \timesg{10000}, \degC{4} for 10\,min and the supernatant discarded.
The pellet was resuspended in 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 2.5\,mM imidazole, 10\,\% glycerol pH\,7 using a volume of $\approx$\SI{10}{\milli\litre\per\gram} of cell pellet.
The cells were lysed by sonication (70\,\% amplitude, 1\,s on-off-cycle) for 30\,seconds, which was repeated twice.
The crude lysate was clarified by centrifugation at \timesg{15000}, \degC{4} for 15\,minutes followed by filtration through a \SI{0.45}{\micro\metre} filter.
Consequently, the His-tagged \gls{pfomt} was purified by \gls{imac} (\ref{sec:imac}).
The eluted \gls{pfomt} protein was dialyzed (\ref{sec:rebuffer}) against 25\,mM \glslink{hepes}{HEPES}, 100\,mM NaCl, 5\,\% glycerol pH 7 and stored at \degC{-20} until use.

\subsubsection{Heterologous production of \glsentrytext{somt}}
\Gls{somt} was produced as a fusion protein with an N-terminal His-tag.
A starter \gls{lb}-culture ($\approx$\,2\,ml) containing \ugperml{100} kanamycin was inoculated with a single colony of \eco BL21(DE3) transformed with pET28MC-somt \bweigel{Überprüfen!} and incubated at \degC{37}, 220\,rpm for 6\,hours.
The starter culture was used to inoculate the main culture (\gls{lb}-medium containing \ugperml{100} kanamycin), such that \odx{600}$\approx$\,0.05.
The culture was incubated at \degC{37}, 220\,rpm in a shaking incubator until \odx{600}$\approx$\,0.6.
Expression was induced by addition of 1\,mM \gls{iptg}.
Incubation continued at \degC{37}, 220\,rpm for 6 hours.
Cells were harvested by centrifugation (\timesg{10000}, \degC{4}, 10\,min) and used, or stored at \degC{-20} until use.
\Gls{somt} was produced in \glspl{ib}, which were prepared as laid out in \autoref{sec:ibprep}.

\subsection{Preparation of \glsentryfirst{ib}}
\label{sec:ibprep}
Often, when recombinant protein is produced in high levels in \eco it is accumulated in so-called \glsentryfirst{ib}. 
The accumulating \glspl{ib} consist mainly of the overproduced target protein, which is inherently quite pure already.
\Glspl{ib} can be selectively recovered from \eco cell lysates and can consequently be refolded. 
\Glspl{ib} were prepared according to a modified protocol by Palmer \citep{Palmer2012}. \\
The cells were resusupended in 5\,ml/$\mathrm{g_{cells}}$ IB lysis buffer (100\,mM \gls{tris}/HCl, 5\,mM \gls{edta}, 5\,mM \gls{dtt} pH\,7) and 5\,mM benzamidine was added as protease inhibitor.
The solution was homogenized using a tissue grinder homogenizer.
\ugperml{200} lysozyme was added to aid in the breakage of cells and the cells were lysed by sonicating thrice at 70\,\% amplitude (1\,s on-off-cycle) for 30 seconds.
The lysate was clarified by centrifuging for 1\,h at \timesg{20000}, \degC{4}.
The supernatant was discarded and the pellet was resuspended in 5\,ml/$\mathrm{g_{cells}}$ IB wash buffer (100\,mM \gls{tris}/HCl, 5\,mM \gls{edta}, 5\,mM \gls{dtt}, 2\,M urea, 2\,\% (w/v) Triton X-100 pH\,7), followed by thorough homogenization.
The solution was centrifuged (30\,min at \timesg{20000}, \degC{4}), the supernatant discarded and the pellet was washed twice more.
To remove detergent, the pellet was washed twice again. However, for this step the Triton X-100 was omitted from the IB-wash buffer.
The \glspl{ib} were resuspended in extraction buffer (100\,mM \gls{tris}/HCl, 5\,mM \gls{edta}, 5\,mM \gls{dtt}, 8\,M \gls{gua} pH\,7) and stored at \degC{-20} until use.




\subsection{Purification of His-tagged proteins using \glsentryfirst{imac}}
\label{sec:imac}
N- or C-terminal oligo-histidine tags (His-tags) are a common tool to ease purification of recombinantly produced proteins.
The free electron pairs of the imidazol nitrogens of histidines can complex divalent cations such as \cf{Mg^2+} or \cf{Ni^2+}, which are usually immobilized on a matrix of \gls{nta}-derivatives.
The affinity of the His-tag is correlated with its length and tagged proteins can simply be eluted by increasing the concentration of competing molecules (e.g. imidazole). \\
His-tagged protein was purified by \gls{fplc} via \cf{Ni^2+}- (HisTrap FF crude)  or \cf{Co^2+}-NTA (HiTrap Talon FF crude) columns, obtained from GE Healthcare (Freiburg, Germany), following modified suppliers instructions.
First the column was equilibrated with 5\,\gls{cv} of binding buffer (50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 2.5\,mM imidazole pH 7).
The sample (generally clarified lysate) was applied to the column using a flow of \SI{0.75}{\milli\litre\per\minute}. Unound protein was removed by washing with 3\,\gls{cv} binding buffer.
Unspecifically bound proteins were washed away by increasing the amount of elution buffer (50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 250\,mM imidazole pH 7) to 10\,\% (constant for 3 to 5\,\gls{cv}). 
Highly enriched and purified target protein was eluted with 6 to 10\,\gls{cv} of 100\,\% elution buffer. 

\section{Crystallographic Procedures}
\label{sec:xraymethods}

\subsection{Crystallization of proteins}

Commercially available crystallization screens were used to find initial crystallization conditions.
The tested screens included kits available from Hampton Research (Aliso Viejo, USA) and Jena Bioscience (Jena, Germany).
Crystallization screens were processed in 96-well \glspl{mtp}, where each well possessed 4 subwells aligned in a 2$\times$2 matrix. The subwells were divided into 3 shallow wells for sitting drop vapour diffusion experimental setups and a fourth subwell, which was deep enough to act as buffer reservoir.
This way the performance of each crystallization buffer could be assessed using three different protein solutions with varying concentrations, effectors etc.
A pipetting robot (Cartesian Microsys, Zinsser-Analystik; Frankfurt, Germany) was used to mix \SI{200}{\nano\litre} of each, protein and buffer solution, for a final volume of \SI{400}{\nano\litre}.
The crystallization preparations were incubated at \degC{16} and the progress of the experiment was documented by an automated imaging-system (Desktop Minstrel UV, Rigaku Europe, Kent, UK).
Furthermore, fine screens (e.g. for refinement of crystallization conditions) were set up by hand in 24-well \glspl{mtp} using the hanging drop vapour diffusion method.

\subsubsection{\glsentrytext{pfomt}}
\Gls{pfomt} protein was concentrated to \SIrange{6}{8}{\mg\per\ml} and rebuffered to 10\,mM \gls{tris}/HCl pH 7.5 using Amicon\textsuperscript{\textregistered} Ultracel centrifugal concentrators (10\,kDa \gls{mwco}).
The concentrated protein solution was centrifuged at \timesg{14000}, \degC{4} for 10\,min to remove any insoluble material or aggregates.
Crystallization screens were set up as described above. \\
\Gls{pfomt} was crystallized using the following conditions \bweigel{nochma genau guckn}  2\,M \cf{(NH4)2SO4}, 20\,\%glycerol. The protein solution contained 0.25\,mM \gls{sah}, 0.25\,mM \cf{MgCl2}, 0.25\,mM ferulic acid and \SI{7.53}{\mg\per\ml} \gls{pfomt}.

\bweigel{add DES crystallization}

\subsection{Data collection and processing}
Crystallographic data were collected at the beamline of the group of Professor Stubbs (\gls{mlu}, Halle, Germany). 
The beamline was equipped with a rotating anode X-ray source MicroMax007 (Rigaku/MSC, Tokio, Japan), which had a maximum power of \SI{0.8}{\kilo\watt} (\SI{40}{\kilo\volt}, \SI{20}{\milli\ampere}) and supplied monochromatic \cf{Cu}-K\textsubscript{\textalpha}-radiation with a wavelength of 1.5418\,\gls{a}.
Diffraction patterns were detected with a Saturn 944+ detector (CCD++, Rigaku/MSC, Tokio, Japan). \\
Indexing and integration of the reflexes via Fourier transformation was accomplished using \emph{XDS} \citep{Kabsch2010,Kabsch1993,Kabsch2010a} or \emph{MOSFLM} \citep{Powell1999}.
\emph{Scala} \citep{Evans2006}, which is integrated in the \gls{ccp4}-Suite, was used for scaling of the intensities.


\subsection{Structure solution}
For the determination of the electron density $\rho(\textrm{\textbf{r}})$, where \textbf{r} is the positional vector, from the diffraction images by Fourier transformation two terms are neccessary as coefficients; the \emph{structure factor amplitudes}, $F_{\mathrm{obs}}(\mathrm{\textbf{h}}$) and the \emph{phase angles} or \emph{phases}, $\alpha\mathrm{(\textbf{h})}$, where \textbf{h} is the reciprocal index vector.
The structure factor amplitudes can be directly determined from the measured and corrected diffraction intensities of each spot.
However, the phase information is lost during the detection of the diffracted photons and there is no direct way to determine the phases.
This constitutes the so-called \emph{phase problem}.
Thus, additional phasing experiments are necessary in order to obtain the phases.
A variety of phasing experiments are available, which include \emph{marker atom substructure methods}, \emph{density modification} and \emph{\gls{mr}} techniques \citep{Rupp2009}.
Phases of the structures herein were exclusively determined by \gls{mr} \citep{Rossmann1962,Rossmann2001}. \\
\Gls{mr} was performed using the software \emph{Phaser} \citep{McCoy2007,McCoy2007a}, which is included in the \gls{ccp4}-Suite \citep{Winn2011}. 
A previously published \gls{pfomt} structure (\gls{pdb}-code: 3C3Y \citep{Kopycki2008b}) was used as a template during \gls{mr} procedure for the \gls{pfomt} structure solution.
%%
\bweigel{Lysozym struktur}
For the \gls{mr} of the lysozyme structure the \gls{pdb}-entry XXXX was used.


\subsection{Model building, refinement and validation}
Macromolecular model building and manipulation, as well as real space refinement and Ramachandran idealization were performed using the \gls{coot} software \citep{emsley2010:coot}.
Structure refinement was done using the software REFMAC5 \citep{Murshudov1997,Vagin2004} as part of the \gls{ccp4}-suite or the \gls{phenix} \citep{Adams2010}.
Validation of the structures was carried out using the web service MolProbity (\url{http://molprobity.biochem.duke.edu/}) \citep{Chen2010}.
Structure visualization and the preparation of figures was performed using PyMOL (Schrödinger, New York, USA).

\subsection{\textit{In silico} substrate docking}
\textit{In silico} molecular docking studies were performed using the AutoDock Vina 1.1.2 or AutoDock 4.2.6 software in combination with the AutoDockTools-Suite (\url{http://autodock.scripps.edu/}) \citep{Huey2007,Morris2009,Trott2010}.
Substrates were docked into the \gls{pfomt} structure with the \gls{pdb}-code 3C3Y.
The grid box, which determines the search space, was manually assigned to center at 1.581, 5.196, 25.718 (x,y,z) and had size of 22, 20, 25 \gls{a} (x, y, z)).
The exhaustiveness of the global search for AutoDock Vina was set to 25, whereas the rest of the input parameters were kept at their defaults.

%%%% analytical methods
\section{Analytics}
\subsection{In vitro determination of glucose}
The glucose concentration in clarified, aqueous samples was determined by a modified version of the glucose assay kit procedure provided by Sigma-Aldrich \citep{Sigma-Aldrich}. \Gls{god} oxidizes D-glucose to gluconic acid, whereby hydrogen peroxide is produced. The hydrogen peroxide can be detected and quantified by \gls{hrp}, which reduces the produced \cf{H2O2} and thereby oxidizes its chromogenic substrate \textit{o}-dianisidine. The oxidized form of \textit{o}-dianisidine can then be measured photospectrometrically \citep{Claiborne1979}.\\
The methodology employs a coupled photospectrometric assay using \gls{god} and \gls{hrp} with \textit{o}-dianisidine as reporter substrate. 
The assay was prepared in \gls{mtp}-format. A reaction solution containing \SI{12.5}{U\per\milli\litre} \gls{god}, \SI{2.5}{U\per\milli\litre} \gls{hrp} and \SI{0.125}{\milli\gram\per\milli\litre} \textit{o}-dianisidine dihydrochloride in \SI{50}{mM} sodium acetate pH\,5.1 was prepared. \\
Sample solutions from culture supernatants were typically diluted in 9\,volumes of water.
The reaction was started, by adding \ul{50} reaction solution to \ul{25} of sample and was incubated at \degC{37} and 200\,rpm for 30\,min in a shaking incubator.
\ul{50} 6\,M sulfuric acid was added to stop the reaction and achieve maximum color development (full oxidation of any \textit{o}-dianisidine charge transfer complexes). \\
The developed pink color was measured at 540\,nm in a \gls{mtp}-reader.
A calibration curve of a standard D-glucose solutions (0 to 100\,\si{\micro\gram\per\milli\litre}), that was always part of the experiments, was used to quantify the sample measurements.


\framedfigure[label = fig:godassay]{
  \begin{subfigure}{0.3\linewidth}\centering
    \begin{chemEnv}[0.5]
      \schemestart
      D-Glc\arrow{-y>[\chemfig{O_2}, \chemfig{H_2O}][\chemfig{H_2O_2}][GOD]}[,1.5]gluconic acid 
      \arrow(@c1.south east--.north east){0}[-90,.7]
      o-dianisidine\arrow{-y>[\chemfig{H_2O_2}][\chemfig{2 H_2O}][HRP]}[,1.5]CTC 
      \arrow(@c3.south east--.north east){0}[-90,.7]
      CTC\arrow{->[][\chemfig{H_2SO_4}]}[,1.5]diimine 
      \schemestop
 \end{chemEnv}
  \end{subfigure}
  \begin{subfigure}{0.6\linewidth}\centering
    \begin{chemEnv}[0.75]
      \setarrowdefault{-90}
      \schemestart
    	\Mchemname{\chemfig[][scale=1]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}}{\textbf{diamine}}\arrow{->[*{0}{-2\cf{e^-},2\cf{H^+}}]}
    	\chemleft[
	  \Mchemname{\subscheme{
	      \chemfig[]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}\arrow{0}[0,0.1]
              \chemfig[]{*6((=HN)-(-OMe)=-(-*6(-=-(=NH)-(-OMe)=-))-=--)}    
          }}{Charge transfer complex}\chemright]
        \arrow{<=>}
	\Mchemname{\chemfig[]{*6((=HN)-(-OMe)=-(-*6(-=-(=NH)-(-OMe)=-))-=--)}}{quinonediimine}
    \schemestop
    \end{chemEnv}
  \end{subfigure}
  
}{GOD assay}



\subsection{\glsentrytext{hplc} measurements}
