%%%-------------------------------------------------%%%
%%% Sub document for material and methods %%%
%%%-------------------------------------------------%%%

\chapter{Material And Methods}

%%%% materials (chemicals, inbstrumenmts, software etc)
\section{Materials}
\subsection{Chemicals}
Enzymes and buffers used for molecular cloning were obtained from Life Technologies (Darmstadt, Germany), unless otherwise noted. 
Flavonoid HPLC standards were purchased from Extrasynthese (Genay, France). 
Deuterated solvents were aquired from Deutero GmbH (Kastellaun, Germany). 
Solvents, purchased from VWR (Poole, England), were distilled in-house before use. \\
All other chemicals were obtained from either Sigma-Aldrich (Steinheim, Germany), Applichem (Darmstadt, Germany), Carl Roth (Karlsruhe, Germany) or Merck (Darmstadt, Germany).

\subsection{Instruments}

\begin{small}
\begin{longtable}{p{0.35\textwidth}p{0.65\textwidth}}
  \gls{cd}-spectrometer & Jasco J-815 (Eaton, USA) \\
  electrophoresis (horizontal) & Biometra Compact XS/S (Göttingen, Germany)\\
  electrophoresis (vertical) & Biometra Compact M (Göttingen, Germany) \newline Biometra Minigel-Twin (Göttingen, Germany) \\
  \gls{fplc} & ÄKTA purifier (GE Healthcare, Freiburg, Germany) \\
  \gls{gcms} & GC-MS-QP2010 Ultra (Shimadzu, Duisburg, Germany)\\
  \gls{hplc} & VWR-Hitachi LaChrom Elite (VWR, Darmstadt, Germany) \\
  \gls{itc} & MicroCal iTC200 (Malvern, Worcestershire, UK) \\
  \gls{mtp}-reader & SpectraMax M5 (Molecular Devices, Biberach, Germany) \\
  \gls{nmr}-spectrometer & Varian Unity 400 (Agilent, Böblingen, Germany) \newline Varian VNMRS 600 (Agilent, Böblingen, Germany) \\
  photospectrometer & Eppendorf Biophotometer Plus (Hamburg, Germany) \newline JASCO V-560 (Eaton, USA) \newline Colibri Microvolume Spectrometer (Biozym, Hess. Oldendorf, Germany) \\
  centrifuges & Eppendorf 5424 (Hamburg, Germany) \newline Hettich Mikro 120 (Kirchlengern, Germany) \newline Beckman Avanti J-E, Beckman Allegra X-30R (Krefeld, Germany)  \\
  centrifuge rotors & Beckman JA-10, JA-16.250, JS-4.3 (Krefeld, Germany)    
\end{longtable}
\end{small}

\subsection{Bacterial strains}

\subsubsection{\textit{E.coli}}
\begin{longtable}{p{0.3\textwidth}p{0.68\textwidth}}
  BL21(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Invitrogen, Karslruhe (Germany) \\
  C41(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Lucigen, Wisconsin (USA) \\
  C43(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3)\newline Lucigen, Wisconsin (USA) \\
  DH5$\alpha$ & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{lacZYA-argF}) U169 \textit{recA}1 \textit{endA}1 \textit{hsdR}17(r$_K^-$m$_K^+$) \textit{phoA supE}44 $\lambda^-$ \textit{thi}-1 \textit{gyrA}96 \textit{relA}1 \newline Invitrogen, Karlsruhe (Germany) \\
  JM110 & \textit{rpsL thr leu thi lacY galK galT ara tonA tsx dam dcm glnV}44 $\Delta$(\textit{lac-proAB}) e14- [F' \textit{traD}36 \textit{proAB}$^+$ \textit{lacI}$^q$ \textit{lacZ}$\Delta$M15] \textit{hsdR}17(r$_K^-$m$_K^+$) \newline Martin-Luther-University Halle-Wittenberg \\
  JW1593 \newline (BW25113 derivative) & \textit{rrnB} $\Delta$\textit{lacZ}4787 \textit{HsdR}514 $\Delta$(\textit{araBAD})568 \textit{rph}-1 $\Delta$\textit{ydgG} (Kan$^R$) \newline Keio Collection, National Institute of Genetics (Japan) \\
  MG1655 & F$^-$ $\lambda^-$ \textit{ilvG}$^-$ \textit{rfb}-50 \textit{rph}-1 \newline DSMZ, Hamburg (Germany) \\
  One Shot TOP10 & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{mrr-hsdRMS-mcrBC}) \textit{recA}1 \textit{endA}1 \textit{mcrA} $\Delta$\textit{lacX}74 \textit{araD}139 $\Delta$(\textit{ara-leu})7697 \textit{galU galK rpsL} (Str$^R$) $\lambda^-$ \textit{nupG} \newline Invitrogen, Karlsruhe (Germany) \\
  Origami(DE3) & $\Delta$(\textit{ara-leu})7697 $\Delta$\textit{lacX}74 $\Delta$\textit{phoA} \textit{Pvull phoR araD}139 \textit{ahpC galE galK rpsL} F'[\textit{lac + lacI q pro}] (DE3)\textit{gor}522::\textit{Tn}10 \textit{trxB} (Kan$^R$, Str$^R$, Tet$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) pLysS & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pLysSRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  T7 Express & \textit{fhuA}2 \textit{lacZ::T}7 \textit{gene}1 \textit{[lon] ompT gal sulA}11 \textit{R}(\textit{mcr-}73\textit{::miniTn}10--Tet$^S$)2 \textit{[dcm] R}(\textit{zgb}-210\textit{::Tn}10--Tet$^S$) \textit{endA}1 $\Delta$(\textit{mcrC-mrr})114\textit{::IS}10 \newline NEB, Massachusetts (USA) \\
\end{longtable}

\subsubsection{\textit{Agrobacterium tumefaciens}}
\begin{tabular}{p{0.2\textwidth}p{0.8\textwidth}}
GV3101 & chromosomal background: C58, marker gene: \textit{rif}, \gls{tip}: cured, opine: nopaline \newline Sylvestre Marillonet, IPB \\  
\end{tabular}


\subsection{Plasmids}
\ostTableEnv[label = tab:plasmids]{%
  \begin{tabularx}{\textwidth}{p{0.3\textwidth}X}
    \toprule
	name & supplier/source \\ 
    \midrule
    pACYCDuet-1 & Merck, Darmstadt (Germany)  \\
    pCDFDuet-1 & Merck, Darmstadt (Germany) \\
    pET-20b(+) & Merck, Darmstadt (Germany) \\
    pET-28a(+) & Merck, Darmstadt (Germany) \\
    pET-32a(+) & Merck, Darmstadt (Germany) \\
    pET-41a(+) & Merck, Darmstadt (Germany) \\
    pQE30 & QIAGEN, Hilden (Germany) \\
    pUC19 & Invitrogen, Karslruhe (Germany) \\
    pQU30-PFOMT & Thomas Vogt (\gls{ipb}, Halle, Germany) \\
    \bottomrule
  \end{tabularx}
}{%
Plasmids used in this work.
}

\subsection{Oligonucleotides and synthetic genes}
Oligonucleotides and primers were ordered from Eurofins Genomics (Ebersberg, Germany). The purity grade was \emph{high purity salt free} (HPSF).
Synthetic genes or gene fragments were obtained from GeneArt (Life Technologies, Darmstadt, Germany) or Eurofins Genomics (Ebersberg, Germany).

\ostTableEnv[label = tab:primers]{%
  \begin{tabularx}{\textwidth}{X >{\footnotesize}p{0.70\textwidth} X}
    \toprule
    	name & sequence (5'$\rightarrow$3') & cloning site \\ 
    \midrule
%    	T7 & TAATACGACTCACTATAGGGT & n/a \\
%       T7.term & GCTAGTTATTGCTCAGCGGT & n/a \\
        somt1 & TT\underline{G AAG AC}A AAA TGG CTT CTT CAT TAA ACA ATG GCC G & BpiI \\
        somt2 & TT\underline{G AAG AC}A AGG ACA CCC CAA ATA CTG TGA GAT CTT CC & BpiI \\
        somt3 & TT\underline{G AAG AC}A AGT CCT TAG GAA CAC CTT TCT GGG AC  & BpiI \\
        somt4 & TT\underline{G AAG AC}A AAA GCT CAA GGA TAG ATC TCA ATA AGA GAC & BpiI \\
        pfomt1.fw & CAG AGA GGC cTA TGA GAT TGG CTT GC & \\
        pfomt1.rv & GCA AGC CAA TCT CAT AgG CCT CTC TG & \\
        pfomt2.fw & \underline{CAT ATG} GAT TTT GCT GTG ATG AAG CAG GTC & NdeI \\
        pfomt2.rv & \underline{GAA TTC} AAT AAA GAC GCC TGC AGA AAG TG & EcoRI \\
        pRha1.fw & CTC TAG C\underline{AG ATC T}CG GTG AGC ATC ACA TCA CCA CAA TTC & BglII \\
        pRha1.rv & CAA TTG A\underline{GG ATC C}CC ATT TTA ACC TCC TTA GTG & BamHI \\
        pUC1.fw & GCG TAT TGG Gag aTC TTC CGC TTC CTC & \\
        pUC1.rv & GAG GAA GCG GAA GAt ctC CCA ATA CGC & \\
    \bottomrule
  \end{tabularx}
}{%
Primers used in this work. Recognition sites for endonucleases are underlined. Positions used for site directed mutagenesis are in lower case font.
}

\subsection{Software}
All mathematical and statistical computations and graphics were done with the \verb+R+ software (versions 3.1.X, \url{http://cran.r-project.org/}). Visualizations of macromolecules were arranged using the PyMol Molecular Graphics System, version 1.7.0.0 (Schrödinger, New York, USA).\\
Physicochemical calculations and calculations of different molecular descriptors were performed using Marvin Beans 15.4.13.0 (ChemAxon, Budapest, Hungary) and Molecular Operating Environment 2008.10 (Chemical Computing Group, Montreal, Canada).

%%%% Microbiologicyl Methods
\section{Microbiology}


%%%% Molecular Biology methods
\section{Molecular Biology}
Basic molecular biology methods like \gls{pcr}, DNA restriction/ligation, DNA gel electrophoresis, preparation of competent cells and transformation were performed based on the protocols summarized by Sambrook and Russell \citep{Sambrook2001}.\\
Plasmid DNA was isolated using the QIAprep Spin Miniprep Kit (QIAGEN, Hilden, Germany) according to the manufacturer's instructions. \\
In vitro site-directed mutatgenesis was set-up according to the protocol of the \emph{QuikChange Site-Directed Mutagenesis} kit \cite{Technologies} offered by Agilent Technologies (Santa Clara, USA). \\
Nucleotide fragments obtained by \gls{pcr}, restriction/ligation procedures or excision from electrophoresis gels were purified and concentrated using the \emph{Nucleospin Gel and PCR Clean-up} kit provided by Machery-Nagel (Düren, Germany) according to the instructions provided by the manufacturer.

\subsection{Golden Gate Cloning}

The Golden Gate cloning procedure is a one-pot method, meaning the restriction digestion and ligation are carried out in the same reaction vessel at the same time \cite{Kondou2011,Engler2008}. Consequently \gls{pcr}-fragments, destination vector, restriction endonuclease and ligase are added together in this reaction. The methodology employs type II restriction enzymes, which together with proper design of the fragments allow for a ligation product lacking the original restriction sites.  \\
For digestion/ligation reactions of fragments containing BpiI sites, 20\,fmol of each fragment or vector, together with 5\,U of BpiI and 5\,U of T4 ligase were combined in a total volume of \ul{15} 1$\times$ ligase buffer.
For fragments to be cloned via BsaI sites, BpiI in the above reaction was substituted by 5\,U BsaI. \\
The reaction mixture was placed in a thermocycler and inbcubated at \degC{37} for 2\,min and \degC{16} for 5\,min. These two first steps were repeated 50 times over. Finally, the temperature was raised to \degC{50} (5\,min) and \degC{80} (10\,min) to inactivate the enzymes.

\subsection{Subcloning of \textit{pfomt}}
The \textit{pfomt} gene was subcloned from the pQE-30 vector kindly provided by Thomas Vogt (\gls{ipb}, Halle, Germany) into the pET-28a(+) vector. 
First an endogenous NdeI site was removed by site directed mutagenesis using primers \emph{pfomt1.fw} and \emph{pfomt1.rv}.
The \textit{pfomt} gene was then amplified using oligonucleotides \emph{pfomt2.fw} and \emph{pfomt2.rv} and cloned into pET-28 employing the NdeI and EcoRI cloning sites. 

\subsection{Subcloning og \textit{somt-2}}


%%%% protein biochemical methods
\section{Protein biochemistry}
Stock solutions of antibiotics, IPTG or sugars were prepared according to the pET System Manual by Novagen \citep{Novagen2011}, unless otherwise noted.

\subsection{Protein production test (expression test)}
The heterologous production of proteins in \eco was assessed in a small scale protein production test, henceforth called expression test.
Single colonies of \eco transformed with the constructs to be studied were used to inoculate a 2\,ml starter culture in \gls{lb}-medium containing the appropriate antibiotics. 
The concentrations of antiobiotics used was as follows:
\ugperml{200} ampicillin, \ugperml{150} kanamycin, \ugperml{50} chloramphenicol, \ugperml{20} tetracycline. \\
The starter culture was allowed to grow at \degC{37} and 200\,rpm over night.
A 5\,ml sampling culture of the medium to be studied containing the appropriate antibiotics was prepared. The media tested included \gls{lb}, \gls{tb} and auto-induction media like ZYP-5052.
The sampling culture was inoculated to an \odx{600} of 0.075 using the starter culture and incubated at different temperatures and 200\,rpm in a shaking incubator.
1\,mM \gls{iptg} was added when the \odx{600} reached 0.6-0.8, if appropriate for the studied construct.
1\,ml samples were removed after different times of incubation (e.g. 4, 8, 12 hours), subfractionated (\ref{sec:subfractionation}) and analyzed via \gls{sds}-\gls{page} (\ref{sec:sdspage}).\\
Exact specifications of growth conditions (e.g. temperature, time, constructs) are discussed in the individual sections.

\subsection{Protein subfractionation}
\label{sec:subfractionation}
The protein subfractionation procedure described herein was adapted from the protocol described in the pET Manual \citep{Novagen2011}. 
Overall 5 protein subfractions can be obtained, including \emph{total cell protein, culture supernatant (medium) protein, periplasmic protein, solube cytoplasmic protein} and \emph{insoluble protein}. \\
The \odx{600} of the culture sample was measured and the cells harvested by centrifugation at \timesg{10000}, \degC{4} for 5\,minutes.
The protein in the supernatant medium was concentrated by precipitation with \gls{tca} (\ref{sec:tcaprecipitation}) for \sdspage analysis.
The periplasmic protein was prepared (\ref{sec:ppprep}) and also concentrated by \gls{tca} precipitation for \sdspage.
Cells were lysed by resuspending the cell pellet in (\odx{600}$\times~\mathrm{V}~\times~50$)\,\si{\micro\litre} of \gls{bper} and vortexing vigorously for 30\,s. 
The suspension was incubated at \gls{rt} for 30\,min to assure complete lysis. To separate insoluble protein and cell debris from the soluble cytosolic protein, the suspension was centrifuged at \timesg{10000} and \degC{4} for 10\,min.
Soluble cytoplasmic protein was contained in the supernatant, whereas insoluble protein remained in the pellet. For \sdspage analysis of the insoluble protein, the pellet was resuspended in the same volume of \gls{bper}.
To obtain only the total cell protein fraction, the preparation of periplasmic and soluble cytosolic protein was omitted. Sample volumes of \ul{10} of each fraction were used for \sdspage analysis.


\subsection{Protein sample concentration by \glsentrytext{tca} precipitation}
\label{sec:tcaprecipitation}
Diluted protein samples were concetrated by \gls{tca} precipitation in microcentrifuge tubes.
Therefore 0.1\,\gls{v} of 100\,\%~(w/v) \gls{tca} in water was added to the clarified sample, which was then vortexed for 15\,s and placed on ice for a minimum of 15\,min.
The sample was centrifuged at \timesg{14000}, \degC{4} for 15\,min. The supernatant was discarded and the pellet was washed twice with 0.2\,\gls{v} ice-cold acetone.
The acetone was removed and the pellet set to air-dry in an open tube.
After drying, the protein pellet was resuspended in 0.1\,\gls{v} \gls{pbs} containing 1\,$\times$ \gls{sds}-sample buffer by heating to \degC{85} and vigorous vortexing, to achieve a 10\,$\times$ concentration.
After resuspension the sample was analyzed by \sdspage or stored at \degC{-20} until use.

\subsection{Preparation of periplasmic protein}
\label{sec:ppprep}
Target proteins may be directed to the periplasmic space by N-terminal signal sequences like \textit{pelB} or \textit{DsbA/C} \citep{Makrides1996}. The periplasma is, other than the cytosol, an oxidizing environment and often used for the production of proteins containing dilsufide linkages.
The preparation of periplasmic protein was accomplished by an osmotic shock protocol modified from Current Protocols in Molecular Biology \citep{Ausubel2008}.\\
The cell pellet was resuspended in the same volume as the culture sample of 30\,mM tris-HCl, 20\,\% (w/v) sucrose, pH\,8 and 1\,mM \gls{edta} was added. The suspension was stirred for 10\,min at \gls{rt} and the cells were collected by centrifugation at \timesg{10000}, \degC{4} for 10\,min.
The supernatant was discarded and the cell pellet was resuspended in the same volume of ice-cold 5\,mM \cf{MgSO4}.
The suspension was stirred for 10\,min on ice, while the periplasmic proteins were released into the solution.
The cells were collected by centrifugation as before. Periplasmic proteins were contained in the supernatant.

\subsection{Discontinous SDS-polyacrylamide gel electrophoresis (\glsentrytext{sds}-\glsentrytext{page})}
\label{sec:sdspage}

The analysis of samples via \sdspage was realized via the discontinous system first described by Laemmli, which allows separation of proteins based on their electrophoretic mobility, which in turn depends on their size \citep{Laemmli1970}.\\
The \sdspage procedure was carried out according to standard protocols described by Sambrook and Russell \citep{Sambrook2001}.
Very dilute and/or samples with high ionic strength were concentrated and/or desalted by the \gls{tca} precipitation procedure described in \autoref{sec:tcaprecipitation}.
Generally a 10\,\% (acrylamide/bisacrylamide) running gel combined with a 4\,\% stacking gel was used. Reducing \sdspage sample buffer was added to the protein sample to be analyzed, whereafter the sample was heated to \degC{95} for 5\,min, to allow for total unfolding of the protein.
After cooling to \gls{rt} the samples were transferred into the gel pockets for analysis.
The \emph{PageRuler Prestained Protein Ladder} (Life Technologies GmbH, Darmstadt, Germany) was used as a \gls{mw} marker and run alongside every analysis as a reference. 
Gels were stained using a staining solution of 0.25\,\% Coomassie Brilliant Blue G-250 (w/v) in water:methanol:acetic acid (4:5:1) and destained by treatment with water:methanol:acetic acid (6:3:1). 


\subsection{Production of recombinant protein}
\subsubsection{Heterologous production of \glsentrytext{pfomt}}


%%%% analytical methods
\section{Analytics}
\subsection{In vitro determination of glucose}
The glucose concentration in clarified, aqueous samples was determined by a modified version of the glucose assay kit procedure provided by Sigma-Aldrich \citep{Sigma-Aldrich}. \Gls{god} oxidizes D-glucose to gluconic acid, whereby hydrogen peroxide is produced. The hydrogen peroxide can be detected and quantified by \gls{hrp}, which reduces the produced \cf{H2O2} and thereby oxidizes its chromogenic substrate \textit{o}-dianisidine. The oxidized form of \textit{o}-dianisidine can then be measured photospectrometrically \citep{Claiborne1979}.\\
The methodology employs a coupled photospectrometric assay using \gls{god} and \gls{hrp} with \textit{o}-dianisidine as reporter substrate. 
The assay was prepared in \gls{mtp}-format. A reaction solution containing \SI{12.5}{U\per\milli\litre} \gls{god}, \SI{2.5}{U\per\milli\litre} \gls{hrp} and \SI{0.125}{\milli\gram\per\milli\litre} \textit{o}-dianisidine dihydrochloride in \SI{50}{mM} sodium acetate pH\,5.1 was prepared. \\
Sample solutions from culture supernatants were typically diluted in 9\,volumes of water.
The reaction was started, by adding \ul{50} reaction solution to \ul{25} of sample and was incubated at \degC{37} and 200\,rpm for 30\,min in a shaking incubator.
\ul{50} 6\,M sulfuric acid was added to stop the reaction and achieve maximum color development (full oxidation of any \textit{o}-dianisidine charge transfer complexes). \\
The developed pink color was measured at 540\,nm in a \gls{mtp}-reader.
A calibration curve of a standard D-glucose solutions (0 to 100\,\si{\micro\gram\per\milli\litre}), that was always part of the experiments, was used to quantify the sample measurements.


\framedfigure[label = fig:godassay]{
  \begin{subfigure}{0.4\linewidth}\centering
    \begin{tikzpicture}[node distance=2.5cm]\tiny
     
     \node (0,0) (a) {D-Glc + \newline \cf{O2} + \cf{H2O}};
     \node [right of=a] (b) {gluconic acid + \cf{H2O2}};
     \draw[->] (a) -- node[midway, above] {GOD} (b);
		
     \begin{scope}[nodes = {shape=rectangle, draw=none, below=1cm}]
       \node 		at (a) (c) {\cf{H2O2} + \textit{o}-dianisidine};
       \node        at (b) (d) {CTC};
     \end{scope}
     \draw[->] (c) -- node[midway,above] {HRP} (d);
    
     \begin{scope}[->, nodes = {shape=rectangle, draw=none, below = 2cm}]
       \node 		at (a) (e) {CTC};
       \node        at (b) (f) {ox. dianisidine};      
     \end{scope}
     \draw[->] (e) -- node[midway, above] {\cf{H2SO4}} (f);

   \end{tikzpicture}
  \end{subfigure}
  \begin{subfigure}{0.4\linewidth}\centering
    \setatomsep{2em}
    \setarrowdefault{-90}
	\schemestart
    	\chemname{\chemfig[][scale=0.5]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}}{diamine}\arrow{->[*{0}{-2\cf{e^-},2\cf{H^+}}]}
	\chemname{\chemfig[][scale=0.5]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}}{charge transfer complex}\arrow{<=>}
	\chemname{\chemfig[][scale=0.5]{*6((=HN)-(-OMe)=-(-*6(-=-(=NH)-(-OMe)=-))-=--)}}{quinonediimine}
    \schemestop
      \end{subfigure}
  
}{GOD assay}

\subsection{\glsentrytext{hplc} measurements}
