%%%-------------------------------------------------%%%
%%% Sub document for material and methods %%%
%%%-------------------------------------------------%%%

\chapter{Material And Methods}
Within this section percentages refer to volume per volume (v/v) percentages unless otherwise specified.
%%%% materials (chemicals, inbstrumenmts, software etc)
\section{Materials}
\subsection{Chemicals}
Enzymes and buffers used for molecular cloning were obtained from Life Technologies (Darmstadt, Germany), unless otherwise noted. 
Flavonoid HPLC standards were purchased from Extrasynthese (Genay, France). 
Deuterated solvents were aquired from Deutero GmbH (Kastellaun, Germany). 
Solvents, purchased from VWR (Poole, England), were distilled in-house before use. 

All other chemicals were obtained from either Sigma-Aldrich (Steinheim, Germany), Applichem (Darmstadt, Germany), Carl Roth (Karlsruhe, Germany) or Merck (Darmstadt, Germany).

\subsection{Commonly used solutions and buffers}

\begin{tabularx}{\textwidth}{>{\small}p{0.25\textwidth}<{\small}X}
  50$\times$ 5052 & 25\,\% glycerol, 2.5\,\% (w/v) glucose, 10\,\% (w/v) \textalpha-lactose \\
  binding buffer & 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 2.5\,mM imidazole pH 7\\
  elution buffer & 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 250\,mM imidazole pH 7 \\
  lysis buffer & 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 2.5\,mM imidazole, 0.2\,\% Tween-20 pH 7\\
  1\,M \gls{mmt} pH 4 (10$\times$) & \SI{26.8}{\g\per\l} \textsc{l}-malic acid, \SI{78.1}{\g\per\l} \gls{mes}, \SI{26.8}{\g\per\l} \gls{tris}, 2.1\,\% 10\,M HCl \\
    1\,M \gls{mmt} pH 9 (10$\times$) & \SI{26.8}{\g\per\l} \textsc{l}-malic acid, \SI{78.1}{\g\per\l} \gls{mes}, \SI{26.8}{\g\per\l} \gls{tris}, 6.7\,\% 10\,M NaOH \\

  20$\times$ \gls{nps} & 1\,M \cf{Na2HPO4}, 1\,M \cf{KH2PO4}, 0.5\,M \cf{(NH4)2SO4} \\
    
  1\,M \gls{ssg} pH 4 (10$\times$) & \SI{14.8}{\g\per\l} succinic acid, \SI{60.4}{\g\per\l} \cf{NaH2PO4*H2O}, \SI{32.8}{\g\per\l} glycine, 0.4\,\% 10\,M NaOH \\
    1\,M \gls{ssg} pH 10 (10$\times$) & \SI{14.8}{\g\per\l} succinic acid, \SI{60.4}{\g\per\l} \cf{NaH2PO4*H2O}, \SI{32.8}{\g\per\l} glycine, 10.3\,\% 10\,M NaOH \\
    
  5$\times$ SDS sample buffer &  10\,\% (w/v) \gls{sds}, 10\,mM \textbeta-mercaptoethanol, 20\,\% glycerol, 0.2\,M Tris/HCl pH\,6.8, 0.05\,\% (w/v) bromophenolblue\\
  1000$\times$ trace elements & 50\,mM \cf{FeCl3}, 20\,mM \cf{CaCl2}, 10\,mM \cf{MnCl2}, 10\,mM \cf{ZnSO4}, 2\,mM \cf{CoCl2}, 2\,mM \cf{CuCl2}, 2\,mM \cf{NiCl2}, 2\,mM \cf{Na2MoO4}, 2\,mM \cf{Na2SeO3}, 2\,mM \cf{H3BO3} \\
\end{tabularx}
\addtocounter{table}{-1} 
  
\subsubsection{Preparation of \glsentryfirst{nades}}

\Gls{nades} were prepared by adding each component in a round-bottom flask with a stirrer and stirring the mixture at \degC{50} with intermittent sonication treatments until a clear solution was obtained.

\begin{table}[!htb]\centering\small
  \caption{\Glsentryfirst{nades}-mixtures used within this work.}
  \label{tab:des}
  \begin{tabular}{lp{0.3\textwidth}l>{\centering}p{0.3\textwidth}}
    \toprule
    name & composition & mole ratio & mass fraction (w/w) \tabularnewline
    \midrule
    PCH & propane-1,2-diol \newline choline chloride \newline water & 1:1:1 &  0.326 \newline 0.597 \newline 0.077 \tabularnewline
    GCH & \textsc{l}-glucose \newline choline chloride \newline water & 2:5:5 & 0.314 \newline 0.608 \newline 0.078 \tabularnewline
   \bottomrule
  \end{tabular}
\end{table}

%% \ostTableEnv[label=tab:des]{%
%%    \begin{tabularx}{\linewidth}{X p{0.3\textwidth} X p{0.3\textwidth}}
%%    \toprule
%%    name & composition & mole ratio & mass fraction (w/w) \\
%%    \midrule
%%    PCH & propane-1,2-diol \newline choline chloride \newline water & 1:1:1 &  0.326 \newline 0.597 \newline 0.077 \\
%%    GCH & \textsc{l}-glucose \newline choline chloride \newline water & 2:5:5 & 0.314 \newline 0.608 \newline 0.078 \\
%%    \bottomrule
%%  \end{tabularx}
%%  }{%
%%    \gls{nades}-mixtures used within this work.%
%%  }

\subsection{Culture media used to grow bacteria}

\begin{tabularx}{\linewidth}{>{\small}p{0.25\textwidth}<{\small}X}
  LB-medium & \SI{10}{\g\per\l} NaCl, \SI{10}{\g\per\l} tryptone, \SI{5}{\g\per\l} yeast extract, pH 7.5 \\
  LB-agar & LB + 1.5\,\% (w/v) agar-agar \\ 
  TB-medium & \SI{12}{\g\per\l} tryptone, \SI{24}{\g\per\l} yeaxst extract, 0.4\,\% glycerol, 72\,mM \cf{K2HPO4}, 17\,mM \cf{KH2PO4}\\
  ZY & \SI{10}{\g\per\l} tryptone, \SI{5}{\g\per\l} yeast extract \\ 
  ZYP-5052 & volume fraction (v/v): 0.928 ZY, 0.05 20$\times$ NPS, 0.02 50$\times$ 5052, 0.002 1\,M \cf{MgSO4}, 0.0002 1000$\times$ trace elements\\
\end{tabularx}
\addtocounter{table}{-1} 
  
\subsection{Bacterial strains}

\subsubsection{\textit{E.coli}}

\begin{tabularx}{\linewidth}{>{\small}p{0.25\textwidth}>{\small}X}
  BL21(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Invitrogen, Karslruhe (Germany) \\
  C41(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Lucigen, Wisconsin (USA) \\
  C43(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3)\newline Lucigen, Wisconsin (USA) \\
  DH5$\alpha$ & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{lacZYA-argF}) U169 \textit{recA}1 \textit{endA}1 \textit{hsdR}17(r$_K^-$m$_K^+$) \textit{phoA supE}44 $\lambda^-$ \textit{thi}-1 \textit{gyrA}96 \textit{relA}1 \newline Invitrogen, Karlsruhe (Germany) \\
  JM110 & \textit{rpsL thr leu thi lacY galK galT ara tonA tsx dam dcm glnV}44 $\Delta$(\textit{lac-proAB}) e14- [F' \textit{traD}36 \textit{proAB}$^+$ \textit{lacI}$^q$ \textit{lacZ}$\Delta$M15] \textit{hsdR}17(r$_K^-$m$_K^+$) \newline Martin-Luther-University Halle-Wittenberg \\
  JW1593 \newline (BW25113 derivative) & \textit{rrnB} $\Delta$\textit{lacZ}4787 \textit{HsdR}514 $\Delta$(\textit{araBAD})568 \textit{rph}-1 $\Delta$\textit{ydgG} (Kan$^R$) \newline Keio Collection, National Institute of Genetics (Japan) \\
  MG1655 & F$^-$ $\lambda^-$ \textit{ilvG}$^-$ \textit{rfb}-50 \textit{rph}-1 \newline DSMZ, Hamburg (Germany) \\
  One Shot TOP10 & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{mrr-hsdRMS-mcrBC}) \textit{recA}1 \textit{endA}1 \textit{mcrA} $\Delta$\textit{lacX}74 \textit{araD}139 $\Delta$(\textit{ara-leu})7697 \textit{galU galK rpsL} (Str$^R$) $\lambda^-$ \textit{nupG} \newline Invitrogen, Karlsruhe (Germany) \\
  Origami(DE3) & $\Delta$(\textit{ara-leu})7697 $\Delta$\textit{lacX}74 $\Delta$\textit{phoA} \textit{Pvull phoR araD}139 \textit{ahpC galE galK rpsL} F'[\textit{lac + lacI q pro}] (DE3)\textit{gor}522::\textit{Tn}10 \textit{trxB} (Kan$^R$, Str$^R$, Tet$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) pLysS & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pLysSRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  T7 Express & \textit{fhuA}2 \textit{lacZ::T}7 \textit{gene}1 \textit{[lon] ompT gal sulA}11 \textit{R}(\textit{mcr-}73\textit{::miniTn}10--Tet$^S$)2 \textit{[dcm] R}(\textit{zgb}-210\textit{::Tn}10--Tet$^S$) \textit{endA}1 $\Delta$(\textit{mcrC-mrr})114\textit{::IS}10 \newline NEB, Massachusetts (USA) \\
\end{tabularx}
\addtocounter{table}{-1} 
 
\subsubsection{\textit{Agrobacterium tumefaciens}}
%\begin{table}[htb]\centering\small
  \begin{tabularx}{\linewidth}{>{\small}p{0.25\textwidth}>{\small}X}
    GV3101 & chromosomal background: C58, marker gene: \textit{rif}, \gls{tip}: cured, opine: nopaline \newline Sylvestre Marillonet, IPB \\
  \end{tabularx}
%\end{table}
\addtocounter{table}{-1} 

\subsection{Plasmids}
%% \ostTableEnv[label = tab:plasmids]{%
%%   \begin{tabularx}{\textwidth}{p{0.3\textwidth}X}
%%     \toprule
%% 	name & supplier/source \\ 
%%     \midrule
%%     pACYCDuet-1 & Merck, Darmstadt (Germany)  \\
%%     pCDFDuet-1 & Merck, Darmstadt (Germany) \\
%%     pET-20b(+) & Merck, Darmstadt (Germany) \\
%%     pET-28a(+) & Merck, Darmstadt (Germany) \\
%%     pET-32a(+) & Merck, Darmstadt (Germany) \\
%%     pET-41a(+) & Merck, Darmstadt (Germany) \\
%%     pQE30 & QIAGEN, Hilden (Germany) \\
%%     pUC19 & Invitrogen, Karslruhe (Germany) \\
%%     pQE30-PFOMT & Thomas Vogt (\gls{ipb}, Halle, Germany) \\
%%     \bottomrule
%%   \end{tabularx}
%% }{%
%% Plasmids used in this work.
%% }

\begin{table}[htb]\centering\small
  \caption{Plasmids used in this work.}
  \label{tab:plasmids}
  \begin{tabular}{ll}
    \toprule
    name & supplier/source \\ 
    \midrule
    pACYCDuet-1 & Merck, Darmstadt (Germany)  \\
    pCDFDuet-1 & Merck, Darmstadt (Germany) \\
    pET-20b(+) & Merck, Darmstadt (Germany) \\
    pET-28a(+) & Merck, Darmstadt (Germany) \\
    pET-32a(+) & Merck, Darmstadt (Germany) \\
    pET-41a(+) & Merck, Darmstadt (Germany) \\
    pQE30 & QIAGEN, Hilden (Germany) \\
    pUC19 & Invitrogen, Karslruhe (Germany) \\
    \bottomrule
  \end{tabular}
\end{table}

\subsection{Oligonucleotides and synthetic genes}
Oligonucleotides and primers were ordered from Eurofins Genomics (Ebersberg, Germany). The purity grade was \emph{high purity salt free} (HPSF).
Synthetic genes or gene fragments were obtained from GeneArt\textsuperscript{\textregistered} (Life Technologies, Darmstadt, Germany) or Eurofins Genomics (Ebersberg, Germany).

\bweigel{needs completion}
\begin{table}[!htb]\centering\small
  \caption{Primers used in this work. Recognition sites for endonucleases are underlined. Positions used for site directed mutagenesis are in lower case font.}
  \label{tab:primers}
  \begin{tabular}{l>{\scriptsize}p{0.65\textwidth}l}
    \toprule
    	name & \small  sequence (5'$\rightarrow$3') & cloning site \\ 
    \midrule
%    	T7 & TAATACGACTCACTATAGGGT & n/a \\
%       T7.term & GCTAGTTATTGCTCAGCGGT & n/a \\
        somt1 & TT\underline{G AAG AC}A AAA TGG CTT CTT CAT TAA ACA ATG GCC G & BpiI \\
        somt2 & TT\underline{G AAG AC}A AGG ACA CCC CAA ATA CTG TGA GAT CTT CC & BpiI \\
        somt3 & TT\underline{G AAG AC}A AGT CCT TAG GAA CAC CTT TCT GGG AC  & BpiI \\
        somt4 & TT\underline{G AAG AC}A AAA GCT CAA GGA TAG ATC TCA ATA AGA GAC & BpiI \\
        pfomt1.fw & CAG AGA GGC cTA TGA GAT TGG CTT GC & \\
        pfomt1.rv & GCA AGC CAA TCT CAT AgG CCT CTC TG & \\
        pfomt2.fw & \underline{CAT ATG} GAT TTT GCT GTG ATG AAG CAG GTC & NdeI \\
        pfomt2.rv & \underline{GAA TTC} AAT AAA GAC GCC TGC AGA AAG TG & EcoRI \\
        pRha1.fw & CTC TAG C\underline{AG ATC T}CG GTG AGC ATC ACA TCA CCA CAA TTC & BglII \\
        pRha1.rv & CAA TTG A\underline{GG ATC C}CC ATT TTA ACC TCC TTA GTG & BamHI \\
        pUC1.fw & GCG TAT TGG Gag aTC TTC CGC TTC CTC & \\
        pUC1.rv & GAG GAA GCG GAA GAt ctC CCA ATA CGC & \\
    \bottomrule
  \end{tabular}
\end{table}


\subsection{Instruments}

%\begin{small}
\begin{tabularx}{\linewidth}{>{\small}p{0.3\textwidth}>{\small}X}
  \glssymbol{cd}-spectrometer & Jasco J-815 (Eaton, USA) \\
  electrophoresis (horizontal) & Biometra Compact XS/S (Göttingen, Germany)\\
  electrophoresis (vertical) & Biometra Compact M (Göttingen, Germany) \newline Biometra Minigel-Twin (Göttingen, Germany) \\
  \glsentrytext{fplc} & ÄKTA purifier (GE Healthcare, Freiburg, Germany) \\
  \glsentrytext{gcms} & GC-MS-QP2010 Ultra (Shimadzu, Duisburg, Germany)\\
  \glsentrytext{hplc} & VWR-Hitachi LaChrom Elite (VWR, Darmstadt, Germany) \\
  \glsentrytext{itc} & MicroCal iTC200 (Malvern, Worcestershire, UK) \\
  plate-reader & SpectraMax M5 (Molecular Devices, Biberach, Germany) \\
  \glsentrytext{nmr}-spectrometer & Varian Unity 400 (Agilent, Böblingen, Germany) \newline Varian VNMRS 600 (Agilent, Böblingen, Germany) \\
  photospectrometer & Eppendorf Biophotometer Plus (Hamburg, Germany) \newline JASCO V-560 (Eaton, USA) \newline Colibri Microvolume Spectrometer (Biozym, Hess. Oldendorf, Germany) \\
  centrifuges & Eppendorf 5424 (Hamburg, Germany) \newline Hettich Mikro 120 (Kirchlengern, Germany) \newline Beckman Avanti J-E, Beckman Allegra X-30R (Krefeld, Germany)  \\
  centrifuge rotors & Beckman JA-10, JA-16.250, JS-4.3 (Krefeld, Germany)    
\end{tabularx}
\addtocounter{table}{-1} 

\subsection{Software}
All mathematical and statistical computations and graphics were done with the \verb+R+ software (versions 3.1.X, \url{http://cran.r-project.org/}) \citep{RCoreTeam2015}. Visualizations of macromolecules were arranged using the PyMol Molecular Graphics System, version 1.7.0.0 (Schrödinger, New York, USA) or UCSF Chimera version 1.9 (\url{http://www.cgl.ucsf.edu/chimera})\cite{Pettersen2004}.
Physicochemical calculations and calculations of different molecular descriptors were performed using Marvin Beans 15.4.13.0 (ChemAxon, Budapest, Hungary) and Molecular Operating Environment 2008.10 (Chemical Computing Group, Montreal, Canada).
Special software used for X-ray crystal structure solution is discussed seperately in the corresponding section (\ref{sec:xraymethods}).

%%%% Molecular Biology methods
\section{Molecular Biology}
\label{sec:molbio}
Basic molecular biology methods like \gls{pcr}, DNA restriction/ligation, DNA gel electrophoresis, preparation of competent cells and transformation were performed based on the protocols summarized by Sambrook and Russell \citep{Sambrook2001}.
Plasmid DNA was isolated using the QIAprep\textsuperscript{\textregistered} Spin Miniprep Kit (QIAGEN, Hilden, Germany) according to the manufacturer's instructions. 
In vitro site-directed mutatgenesis was set-up according to the protocol of the \emph{QuikChange\textsuperscript{\texttrademark} Site-Directed Mutagenesis} kit \cite{Technologies} offered by Agilent Technologies (Santa Clara, USA). 
Nucleotide fragments obtained by \gls{pcr}, restriction/ligation procedures or excision from electrophoresis gels were purified and concentrated using the \emph{Nucleospin Gel and PCR Clean-up} kit provided by Machery-Nagel (Düren, Germany) according to the instructions provided by the manufacturer.

\subsection{Golden Gate Cloning}
\label{sec:GGcloning}
The Golden Gate cloning procedure is a one-pot method, meaning the restriction digestion and ligation are carried out in the same reaction vessel at the same time \cite{Kondou2011,Engler2008}. Consequently \gls{pcr}-fragments, destination vector, restriction endonuclease and ligase are added together in this reaction. The methodology employs type II restriction enzymes, which together with proper design of the fragments allow for a ligation product lacking the original restriction sites.  
For digestion/ligation reactions of fragments containing BpiI sites, 20\,fmol of each fragment or vector, together with 5\,U of BpiI and 5\,U of T4 ligase were combined in a total volume of \ul{15} 1$\times$ ligase buffer.
For fragments to be cloned via BsaI sites, BpiI in the above reaction was substituted by 5\,U BsaI. 
The reaction mixture was placed in a thermocycler and inbcubated at \degC{37} for 2\,min and \degC{16} for 5\,min. These two first steps were repeated 50 times over. Finally, the temperature was raised to \degC{50} (5\,min) and \degC{80} (10\,min) to inactivate the enzymes.

The \textit{SOMT2} gene was amplified from pET28a-SOMT using primers \textit{somt1--somt4} (\autoref{tab:primers}), cloned into vector pICH41308 (level 0) using BpiI and consequently subcloned into the level 1 module pICH75044, alongside 35S promoter and \gls{nos}-terminator, using BsaI (\autoref{fig:cloning}).
The resulting construct was denoted as pBEW107.

\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \includegraphics[width=\textwidth]{usr/graphics/static/cloning_pBEW107}
  \end{figureframe}
  \caption{Golden Gate cloning scheme for \somt}
  \label{fig:cloning}
\end{figure}

\subsection{Subcloning of genes}
All subcloning procedures were performed according to \autoref{sec:molbio} and specifically \autoref{sec:GGcloning}.
Specific steps for the subcloning of any genes discussed can be found in the appendix (p.\pageref{tab:molecularcloning}).
The \textit{pfomt} gene was subcloned from the pQE-30 vector kindly provided by Thomas Vogt (\gls{ipb}, Halle, Germany) into the pET-28a(+) vector.
The \textit{somt-2} gene was subcloned from the pQE-30 vector kindly provided by Martin Dippe (\gls{ipb}, Halle, Germany) into the pET-28-MC vector.

\subsection{Transformation of electrocompetent \textit{Agrobacterium tumefaciens} cells}
A \ul{50} aliquot of electrocompetent \agro cells was thawed on ice.
\SIrange{50}{100}{\ng} of plasmid were added, the solution was mixed gently and transferred to a pre-cooled electroporation cuvette.
After pulsing (2.5\,kV, \SI{200}{\ohm}) 1\,ml of \gls{lb}-medium was added, the mixture transferred to a 1.5\,ml tube and incubated for (3 to 4)\,hours at \degC{28}.
The culture was centrifuged (\timesg{10000}, 1\,min) and \ul{900} supernatant were discarded.
The pellet was resuspended in the remaining liquid, plated onto \gls{lb}-agar plates supplemented with \ugperml{40} rifampicin and \ugperml{50} carbencillin and incubated at \degC{28} for (2 to 3)\,days.

\section{Treatment of plant material}
\label{sec:agronico}
\subsection{Infiltration of \textit{Nicotiana benthamiana}}
Before infiltration \nico plants were pruned, such that only leaves to be infiltrated remained with the plant (\autoref{fig:c.nico}).
5\,ml cultures of transformed \agro in \gls{lb}-medium (with \ugperml{40} rifampicin and \ugperml{50} carbencillin) were grown over night at \degC{28} and 220\,rpm.
\odx{600} of the culture was measured and adjusted to 0.2 by dilution with infiltration buffer (10\,mM \gls{mes}/\cf{NaOH}, 10\,mM \cf{MgSO4} pH 5.5).
When multiple \agro transformed with different contructs/plasmids were used for infiltration, the cultures were mixed and diluted using infiltration buffer, such that \odx{600} of each culture in the mix was 0.2.
The solution was infiltrated into the abaxial side of \nico leaves using a plastic syringe.
The leaf material was harvested after 7 days.

\subsubsection{Infiltration of \nico for \textit{in vivo} biotransformation using SOMT-2}
Both sides of \nico leaves were infiltrated with different samples (\autoref{fig:c.nico}).
The left side was infiltrated with \agro cultures transformed with pAGM10733 (\gls{pal}), pAGM10406 (\gls{chs}) and pBEW107 (\somt).
For the right side the \agro culture containing pBEW107 was replaced by a control: \agro transformed with the empty vector pICH75044. 


\begin{figure}[htb]
  \begin{figureframe}
    \includegraphics[width=0.5\textwidth]{usr/graphics/static/nico2.eps}
  \end{figureframe}
  \caption{Pruned \nico plant, with two bottom and one top leaf, ready to be infiltrated.}
  \label{fig:c.nico}
\end{figure}

\subsection{Plant material harvest}
Infiltrated/Infected areas of \nico leaf material were cut out and grouped by plant number, leaf position (top/bottom) and leaf side (right/left).
The grouped clippings were weighed, frozen in liquid nitrogen, ground to a powder, freeze-dried and stored at \degC{-80}.

\subsection{Extraction of flavonoids from \nico leaves}
Two tips of a small spatula of freeze-dried material ($\approx$6\,mg), were weighed exactly and extracted with \ul{500} 75\,\% aqueous methanol containing 1\,mM ascorbic acid, 0.2\,\% formic acid and 0.1\,mM flavone (internal standard).
Therefore the suspension was vortexed for 30\,s, rotated on an orbital shaker for 10\,min and vortexed again for 30\,s.
The suspension was centrifuged (\timesg{20000}, \degC{4}, 10\,min) and the supernatant transferrerd to a new tube, to remove the insoluble plant material.
The supernatant was centrifuged again (\timesg{20000}, \degC{4}, 10\,min) and the resulting supernatant was transferred to a \glssymbol{hplc}-vial and stored at \degC{-20} until analysis.

%% \subsection{Cloning of \textit{p-lsr} reporter constructs} --> evtl Ergebnisse



%%%% protein biochemical methods
\section{Protein biochemistry}
Stock solutions of antibiotics, IPTG or sugars were prepared according to the pET System Manual by Novagen \citep{Novagen2010}, unless otherwise noted.

\subsection{Determination of protein concentration}
Protein concentrations were estimated using the absorption of protein solutions at 280\,nm, which is mainly dependent on the amino acid composition of the protein studied \citep{Gill1989}.
Extinction coefficients of proteins were calculated from the amino acid sequence using the ExpPASy servers's ProtParam tool \citep{Gasteiger2005}.

%\begin{lstTabularx}[1]{\linewidth}{p{0.5\textwidth}X}
\begin{table}\small\centering
   \caption{Calculated extinction coefficients of proteins used in this work.} 
  \label{tab:extinctioncoefficients}
  \begin{tabular}{lc}
     \toprule
	protein/enzyme & $\varepsilon^{\mathrm{\SI{1}{\g\per\l}}}_{280\mathrm{nm}}$ in \si[per-mode=reciprocal]{\ml\per\mg\per\cm}\\ 
     \midrule
     \glsentrytext{pfomt} (reduced) & 0.714 \\
     \glsentrytext{pfomt} Y51K N202W (reduced) & 0.852 \\
     \glsentrytext{somt} (oxidized) & 1.263 \\
     \glsentrytext{somt} (reduced) & 1.247 \\
 %    \glsentrytext{comt} & \\
     \bottomrule
  \end{tabular}
\end{table}
%\end{lstTabularx}


\subsection{Protein production test (expression test)}
The heterologous production of proteins in \eco was assessed in a small scale protein production test, henceforth called expression test.
Single colonies of \eco transformed with the constructs to be studied were used to inoculate a 2\,ml starter culture in \gls{lb}-medium containing the appropriate antibiotics. 
The working concentrations of antiobiotics used was as follows:
\ugperml{200} ampicillin, \ugperml{150} kanamycin, \ugperml{50} chloramphenicol, \ugperml{20} tetracycline. 
The starter culture was allowed to grow at \degC{37} and 200\,rpm over night.
A 5\,ml sampling culture of the medium to be studied containing the appropriate antibiotics was prepared. The media tested included \gls{lb}, \gls{tb} and auto-induction media like ZYP-5052.
The sampling culture was inoculated to an \odx{600} of 0.075 using the starter culture and incubated at different temperatures and 200\,rpm in a shaking incubator.
1\,mM \gls{iptg} was added when the \odx{600} reached 0.6-0.8, if appropriate for the studied construct.
1\,ml samples were removed after different times of incubation (e.g. 4, 8, 12 hours), subfractionated (\ref{sec:subfractionation}) and analyzed via \gls{sds}-\gls{page} (\ref{sec:sdspage}).

%Exact specifications of growth conditions (e.g. temperature, time, constructs) are discussed in the individual sections.

\subsection{Protein subfractionation}
\label{sec:subfractionation}
The protein subfractionation procedure described herein was adapted from the protocol described in the pET Manual \citep{Novagen2010}. 
Overall 5 protein subfractions can be obtained, including \emph{total cell protein, culture supernatant (medium) protein, periplasmic protein, solube cytoplasmic protein} and \emph{insoluble protein}. 
The \odx{600} of the culture sample was measured and the cells harvested by centrifugation at \timesg{10000}, \degC{4} for 5\,minutes.
The protein in the supernatant medium was concentrated by precipitation with \gls{tca} (\ref{sec:tcaprecipitation}) for \sdspage analysis.
The periplasmic protein was prepared (\ref{sec:ppprep}) and also concentrated by \gls{tca} precipitation for \sdspage.
Cells were lysed by resuspending the cell pellet in (\odx{600}$\times~\mathrm{V}~\times~50$)\,\si{\micro\litre} of \gls{bper} and vortexing vigorously for 30\,s. 
The suspension was incubated at \gls{rt} for 30\,min to assure complete lysis. To separate insoluble protein and cell debris from the soluble cytosolic protein, the suspension was centrifuged at \timesg{10000} and \degC{4} for 10\,min.
Soluble cytoplasmic protein was contained in the supernatant, whereas insoluble protein remained in the pellet. For \sdspage analysis of the insoluble protein, the pellet was resuspended in the same volume of \gls{bper}.
To obtain only the total cell protein fraction, the preparation of periplasmic and soluble cytosolic protein was omitted. Sample volumes of \ul{10} of each fraction were used for \sdspage analysis.


\subsection{Protein sample concentration by \glsentrytext{tca} precipitation}
\label{sec:tcaprecipitation}
Diluted protein samples were concetrated by \gls{tca} precipitation in microcentrifuge tubes.
Therefore 0.1\,\gls{v} of 100\,\%~(w/v) \gls{tca} in water was added to the clarified sample, which was then vortexed for 15\,s and placed on ice for a minimum of 15\,min.
The sample was centrifuged at \timesg{14000}, \degC{4} for 15\,min. The supernatant was discarded and the pellet was washed twice with 0.2\,\gls{v} ice-cold acetone.
The acetone was removed and the pellet set to air-dry in an open tube.
After drying, the protein pellet was resuspended in 0.1\,\gls{v} \gls{pbs} containing 1\,$\times$ \gls{sds}-sample buffer by heating to \degC{85} and vigorous vortexing, to achieve a 10\,$\times$ concentration.
After resuspension the sample was analyzed by \sdspage or stored at \degC{-20} until use.

\subsection{Preparation of periplasmic protein}
\label{sec:ppprep}
Target proteins may be directed to the periplasmic space by N-terminal signal sequences like \textit{pelB} or \textit{DsbA/C} \citep{Makrides1996}. The periplasma is, other than the cytosol, an oxidizing environment and often used for the production of proteins containing dilsufide linkages.
The preparation of periplasmic protein was accomplished by an osmotic shock protocol modified from Current Protocols in Molecular Biology \citep{Ausubel2008}.
The cell pellet was resuspended in the same volume as the culture sample of 30\,mM tris-HCl, 20\,\% (w/v) sucrose, pH\,8 and 1\,mM \gls{edta} was added. The suspension was stirred for 10\,min at \gls{rt} and the cells were collected by centrifugation at \timesg{10000}, \degC{4} for 10\,min.
The supernatant was discarded and the cell pellet was resuspended in the same volume of ice-cold 5\,mM \cf{MgSO4}.
The suspension was stirred for 10\,min on ice, while the periplasmic proteins were released into the solution.
The cells were collected by centrifugation as before. Periplasmic proteins were contained in the supernatant.

\subsection{Discontinous SDS-polyacrylamide gel electrophoresis (\glsentrytext{sds}-\glsentrytext{page})}
\label{sec:sdspage}

The analysis of samples via \sdspage was realized via the discontinous system first described by Laemmli, which allows separation of proteins based on their electrophoretic mobility, which in turn depends on their size \citep{Laemmli1970}.
The \sdspage procedure was carried out according to standard protocols described by Sambrook and Russell \citep{Sambrook2001}.
Very dilute and/or samples with high ionic strength were concentrated and/or desalted by the \gls{tca} precipitation procedure described in \autoref{sec:tcaprecipitation}.
Generally a 10\,\% (acrylamide/bisacrylamide) running gel combined with a 4\,\% stacking gel was used. Reducing \sdspage sample buffer was added to the protein sample to be analyzed, whereafter the sample was heated to \degC{95} for 5\,min, to allow for total unfolding of the protein.
After cooling to \gls{rt} the samples were transferred into the gel pockets for analysis.
The \emph{PageRuler\textsuperscript{\texttrademark} Prestained Protein Ladder} (Life Technologies GmbH, Darmstadt, Germany) was used as a \gls{mw} marker and run alongside every analysis as a reference. 
Gels were stained using a staining solution of 0.25\,\% Coomassie Brilliant Blue G-250 (w/v) in water:methanol:acetic acid (4:5:1) and destained by treatment with water:methanol:acetic acid (6:3:1). 

\subsection{Buffer change of protein samples}
\label{sec:rebuffer}
The buffer in protein samples was exchanged either by dialysis, or by centrifugal filter concentrators (Amicon\textsuperscript{\textregistered} Ultra Centrifugal Filter; Merck, Darmstadt, Germany).
Large volumes of highly concentrated protein solutions were preferably dialyzed.
Respectively, very dilute samples were concentrated and rebuffered using centrifugal concentrators. 
Dialysis was carried out at least twice against a minimum of 100 times the sample volume.
Dialysis steps were carried out at \gls{rt} for 2 hours, or over-night at \degC{4}.
Centrifugal concentrators were used according to the manufacturers instructions.

\subsection{Production of recombinant protein}
\subsubsection{Heterologous production of \glsentrytext{pfomt}}
\Gls{pfomt} was produced as a N-terminally (His)$_6$-tagged fusion protein.
A 2\,ml starter culture of \gls{lb} containing \ugperml{100} kanamycin was inoculated with a single colony of \eco BL21(DE3) transformed with pET28-pfomt and incubated at \degC{37}, 220\,rpm for 6\,hours.
The main culture (\gls{zyp} containing \ugperml{200} kanamycin) was inoculated with the starter culture such that \odx{600} was 0.05.
The culture was incubated in a shaking incubator at \degC{37}, 220\,rpm over night ($\approx$16\,h).
Due to the autoinducing nature of the \gls{zyp} medium, addition of \gls{iptg} was not neccesary.
Cells were harvested by centrifugation at \timesg{10000}, \degC{4} for 10\,min and the supernatant discarded.
The pellet was resuspended in 50\,mM \gls{tris}/HCl, 500\,mM NaCl, 2.5\,mM imidazole, 10\,\% glycerol pH\,7 using a volume of $\approx$\SI{10}{\milli\litre\per\gram} of cell pellet.
The cells were lysed by sonication (70\,\% amplitude, 1\,s on-off-cycle) for 30\,seconds, which was repeated twice.
The crude lysate was clarified by centrifugation at \timesg{15000}, \degC{4} for 15\,minutes followed by filtration through a \SI{0.45}{\micro\metre} filter.
Consequently, the His-tagged \gls{pfomt} was purified by \gls{imac} (\ref{sec:imac}).
The eluted \gls{pfomt} protein was dialyzed (\ref{sec:rebuffer}) against 25\,mM \glslink{hepes}{HEPES}, 100\,mM NaCl, 5\,\% glycerol pH 7 and stored at \degC{-20} until use.

\subsubsection{Heterologous production of \glsentrytext{somt}}
\Gls{somt} was produced as a fusion protein with an N-terminal His-tag.
A starter \gls{lb}-culture ($\approx$\,2\,ml) containing \ugperml{100} kanamycin was inoculated with a single colony of \eco BL21(DE3) transformed with pET28MC-somt \bweigel{Überprüfen!} and incubated at \degC{37}, 220\,rpm for 6\,hours.
The starter culture was used to inoculate the main culture (\gls{lb}-medium containing \ugperml{100} kanamycin), such that \odx{600}$\approx$\,0.05.
The culture was incubated at \degC{37}, 220\,rpm in a shaking incubator until \odx{600}$\approx$\,0.6.
Expression was induced by addition of 1\,mM \gls{iptg}.
Incubation continued at \degC{37}, 220\,rpm for 6 hours.
Cells were harvested by centrifugation (\timesg{10000}, \degC{4}, 10\,min) and used, or stored at \degC{-20} until use.
\Gls{somt} was produced in \glspl{ib}, which were prepared as laid out in \autoref{sec:ibprep}.

\subsection{Preparation of \glsentryfirst{ib}}
\label{sec:ibprep}
Often, when recombinant protein is produced in high levels in \eco it is accumulated in so-called \glsentryfirst{ib} \citep{Rudolph1996}. 
The accumulating \glspl{ib} consist mainly of the overproduced target protein, which is inherently quite pure already.
\Glspl{ib} can be selectively recovered from \eco cell lysates and can consequently be refolded. 
\Glspl{ib} were prepared according to a modified protocol by Palmer \citep{Palmer2012}. 
The cells were resusupended in 5\,ml/$\mathrm{g_{cells}}$ IB lysis buffer (100\,mM \gls{tris}/HCl, 1\,mM \gls{edta} pH\,7), 0.5\,mM \gls{pmsf} was added as protease inhibitor.
The solution was homogenized using a tissue grinder homogenizer (Ultra Turrax\textsuperscript\textregistered; IKA\textsuperscript\textregistered-Werke GmbH \& Co. KG, Staufen, Germany).
\ugperml{200} lysozyme was added to aid in the breakage of cells and the cells were lysed by sonicating thrice at 70\,\% amplitude (1\,s on-off-cycle) for 30 seconds.
DNase~I (\ugperml{10}) was added and the solution was incubated on ice for 10\,min.
The lysate was clarified by centrifuging for 1\,h at \timesg{20000}, \degC{4}.
The supernatant was discarded and the pellet was resuspended in 5\,ml/$\mathrm{g_{cells}}$ IB wash buffer I (20\,mM \gls{edta}, 500\,mM NaCl, 2\,\% (w/v) Triton X-100 pH), followed by thorough homogenization.
The solution was centrifuged (30\,min at \timesg{20000}, \degC{4}), the supernatant discarded and the pellet was washed twice more.
To remove detergent, the pellet was washed twice again with IB washing buffer II (20\,mM \gls{edta}, 100\,mM Tris/HCl pH\,7).
The \glspl{ib} were resuspended in IB solubilization buffer (100\,mM \gls{tris}/HCl, 5\,mM \gls{dtt}, 6\,M \gls{gua} pH\,7), such that the protein concentration was about \SI{25}{\mg\per\ml} and stored at \degC{-20} until use.


\subsection{Purification of His-tagged proteins using \glsentryfirst{imac}}
\label{sec:imac}
N- or C-terminal oligo-histidine tags (His-tags) are a common tool to ease purification of recombinantly produced proteins.
The free electron pairs of the imidazol nitrogens of histidines can complex divalent cations such as \cf{Mg^2+} or \cf{Ni^2+}, which are usually immobilized on a matrix of \gls{nta}-derivatives.
The affinity of the His-tag is correlated with its length and tagged proteins can simply be eluted by increasing the concentration of competing molecules (e.g. imidazole). 
His-tagged protein was purified by \gls{fplc} via \cf{Ni^2+}- (HisTrap FF crude)  or \cf{Co^2+}-NTA (HiTrap Talon FF crude) columns, obtained from GE Healthcare (Freiburg, Germany), following modified suppliers instructions.
First the column was equilibrated with 5\,\gls{cv} of binding buffer (50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 2.5\,mM imidazole pH 7).
The sample (generally clarified lysate) was applied to the column using a flow of \SI{0.75}{\milli\litre\per\minute}. Unound protein was removed by washing with 3\,\gls{cv} binding buffer.
Unspecifically bound proteins were washed away by increasing the amount of elution buffer (50\,mM \gls{tris}/HCl, 500\,mM NaCl, 10\,\% glycerol, 250\,mM imidazole pH 7) to 10\,\% (constant for 3 to 5\,\gls{cv}). 
Highly enriched and purified target protein was eluted with 6 to 10\,\gls{cv} of 100\,\% elution buffer. 

\subsection{Refolding of \glsentrytext{somt} on a micro scale using \glsentryfirst{doe}}
\label{subsec:doerefolding}
\Gls{doe} and \glsentryfirst{ffd} have been successfully used to optimize the refolding conditions of several proteins \citep{Willis2005,Akbari2010,Benoit2007}.
Thus, an approach using \gls{ffd} was used to find optimal refolding conditions for \gls{somt}. 
Factors studied were pH (buffer), arginine, glycerol, divalent cations, ionic strength, redox system, cyclodextrin and co-factor addition.
The experimental matrix was constructed using the \verb+FrF2+ package (\url{http://cran.r-project.org/web/packages/FrF2/index.html}) in the R software.

%% \begin{lstTabularx}[1]{\linewidth}{Xc>{\centering\arraybackslash}p{0.25\textwidth}>{\centering\arraybackslash}p{0.25\textwidth}c}
%%   \caption{Factors used in the construction of the \gls{ffd}.} \\
%%   \label{tab:ffd_factors}
%%   \small
%%   \hfill \\
%%      \toprule
%% 	factor & symbol & \multicolumn{2}{c}{setting (level)} & unit \\ 
%%         & & - & + & \\
%%      \midrule
%%      \endfirsthead
%%      \toprule
%% 	factor & symbol & \multicolumn{2}{c}{setting (level)} & unit \\ 
%%         & & - & + & \\
%%      \midrule
%%      \endhead
%%      pH & A & 5.5 & 9.5 & - \\
%%      arginine & B & 0 & 0.5 & M \\
%%      glycerol & C & 0 & 10 & \% (v/v) \\
%%      divalent cations\footnote{no: 1\,mM \gls{edta}; yes: 2\,mM \cf{CaCl2}, \cf{MgCl2}} & D & no & yes & - \\
%%      ionic strength\footnote{low: 10\,mM NaCl, 0.5\,mM KCl; high: 250\,mM NaCl, 10\,mM KCl} & E & low & high & -  \\
%%      redox state\footnote{reducing: 5\,mM \gls{dtt}; redox-shuffling: 1\,mM \gls{gsh}, 0.2\,mM \gls{gssg}} & F & reducing & redox-shuffling & - \\
%%      \textalpha -cyclodextrin & G & 0 & 30 & mM \\
%%      \gls{sah} & H & 0 & 0.5 & mM \\
%%      \bottomrule
%% \end{lstTabularx}

%\begin{lstTabularx}[1]{\linewidth}{Xc>{\centering\arraybackslash}p{0.25\textwidth}>{\centering\arraybackslash}p{0.25\textwidth}c}
\begin{table}[htb]\small\centering
  \caption{Factors and their high and low levels (+/-) used in the construction of the \glsentryfirst{ffd}.} 
  \label{tab:ffd_factors}
  \begin{tabular}{lcccc}
    \toprule
    factor & symbol & \multicolumn{2}{c}{setting (level)} & unit \\ 
    & & - & + & \\
    \midrule
    pH & A & 5.5 & 9.5 & - \\
    arginine & B & 0 & 0.5 & M \\
    glycerol & C & 0 & 10 & \% (v/v) \\
    divalent cations\textsuperscript{\textdagger} & D & no & yes & - \\
    ionic strength\textsuperscript{\textdaggerdbl} & E & low & high & -  \\
    redox state\textsuperscript{$\star$} & F & reducing & redox-shuffling & - \\
    \textalpha -cyclodextrin & G & 0 & 30 & mM \\
    \glsentrytext{sah} & H & 0 & 0.5 & mM \\
    \bottomrule
    \multicolumn{5}{p{0.95\textwidth}}{\scriptsize \textsuperscript{\textdagger}no: 1\,mM \gls{edta}; yes: 2\,mM \cf{CaCl2}, \cf{MgCl2} \newline \textsuperscript{\textdaggerdbl}low: 10\,mM NaCl, 0.5\,mM KCl; high: 250\,mM NaCl, 10\,mM KCl \newline \textsuperscript{$\star$}reducing: 5\,mM \gls{dtt}; redox-shuffling: 1\,mM \gls{gsh}, 0.2\,mM \gls{gssg}}
  \end{tabular}
\end{table}

\begin{table}[htb]\small\centering
  \caption{Experimental design matrix for the \glsentrytext{ffd}.} 
  \label{tab:ffd_matrix}
  \begin{tabular}{ccccccccccccc}
%\begin{lstTabularx}[1]{\linewidth}{ccccccccccccc}
     \toprule
	Experiment & A & B & C & D & E & F & G & H \\ 
     \midrule
     %% \endfirsthead
     %% \toprule
     %% Experiment & A & B & C & D & E & F & G & H \\ 
     %% \midrule
     %% \endhead
     1 & +&+&+&-&-&-&-&+\\
     2 & -&-&-&-&-&-&-&-\\
     3 &+&-&+&+&-&+&+&-\\
     4 &-&+&+&-&+&+&+&-\\
     5 &+&+&-&-&+&+&-&-\\
     6 &-&+&-&+&+&-&+&+\\
     7 &+&+&-&+&-&-&+&-\\
     8 &-&-&+&-&+&-&+&+\\
     9 &+&-&+&+&+&-&-&-\\
     10 &-&-&-&-&-&+&+&+\\
     11 &+&-&-&+&+&+&-&+\\
     12 &-&+&+&+&-&+&-&+\\
     \bottomrule
%\end{lstTabularx}
  \end{tabular}
\end{table}

The buffers were mixed from stock solutions and prepared in 1.5\,ml microcentrifuge tubes immediately prior to the experiment.
\ul{50} of solubilized \gls{somt} (\SI{1}{\mg\per\ml}) in \gls{ib} solubilization buffer was added to \SI{1}{\milli\litre} of each buffer followed by a short vortex boost for rapid mixing.
The final protein concentration in the refolding reaction was \ugperml{50}, whereas the remaining \gls{gua} concentration was $\approx$286\,mM.
The refolding reactions were incubated at \gls{rt} for 1 hour, followed by an over night incubation at \degC{4}.
After incubation the refolding reactions were centrifuged (\timesg{10000}, \degC{4}, 10\,min) to separate insoluble and soluble protein fractions.
The supernatant was transferred to a new tube, whereas the pellet was washed twice with \ul{200} acetone and once with \ul{400} methanol/acetone (1:1). 
The pellet was resuspended in \ul{100} \gls{pbs} with \ul{20} \sdspage sample buffer and \ul{10} were used for \sdspage analysis. 
\ul{100} of the supernatant were concentrated using \gls{tca} precipitation (\ref{sec:tcaprecipitation}) and analyzed by \sdspage.
The remaining supernatant was rebuffered into 50\,mM \gls{bistris} pH\,7.5 using Amicon\textsuperscript{\textregistered} Ultra 0.5 ml centrifugal filters (Merck, Darmstadt, Germany) according to the manufacturers instructions.
The pre-weighed collection tubes were re-weighed after recovery and the volume of recovered liquid calculated ($\rho \approx \mathrm{\SI{1}{\g\per\cm\cubed}}$).
The sample was filled up to \ul{100} using 50\,mM \gls{bistris} pH 7.5 and the protein concentration was assessed using the Roti\textsuperscript{\textregistered}-Quant protein quantification solution (Carl Roth, Karlsruhe, Germany) according to the manufacturers description.
\ul{50} of each refolded sample was used for an activity test using naringenin as substrate (\ref{sec:omtassay}). 
The reactions were incubated over night and stopped by the extraction method.
However, before the actual extraction \ul{1} of \gls{ac9} was added as internal standard.
The samples were analyzed by \gls{hplc}. 

\subsubsection{Assessment of refolding performance}
The performance of each buffer on the refolding of \gls{somt} was examined by comparing the \sdspage results, as well as the amount of soluble protein and the conversion of naringenin over night (see \autoref{sec:omtassay}).
Main effects were analyzed qualitatively using main effects plots \citep{Box2005}.
  
\subsubsection{Upscaling of refolding reactions}
Refolding reactions were scaled up to 50\,ml.
Therefore 2.5\,ml solubilized \gls{somt} (\SI{1}{\mg\per\ml}) were added over 10\,minutes to 50\,ml of refolding buffer while stirring at \gls{rt}.
The refolding reaction was allowed to complete over night at \degC{4}.


\subsection{Enzymatic production of \glsentrytext{sam} and \glsentrytext{sae}}
\label{sec:samprod}
\Gls{sam} and \gls{sae} were prepared according to the method described by Dippe, et. al \citep{Dippe2015}. 

Preparative reactions (20\,ml) were performed in  0.1\,M Tris/HCl, 20\,mM \cf{MgCl2}, 200\,mM \cf{KCl} pH\,8.0 and contained 7.5\,mM \gls{atp}, 10\,mM \textsc{d,l}-methionine or \textsc{d,l}-ethionine, for the production of \gls{sam} or \gls{sae} respectively, and 0.2\,\gls{u} \gls{sams} variant I317V.
The reaction was stopped by lowering the pH to 4 using 10\,M acetic acid after 18\,h of incubation at \degC{30}, 60\,rpm.
After 10\,min incubation on ice the solution was centrifuged (\timesg{15000}, 10\,min) to remove insoluble matter. 
The supernatant was transferred to a round bottom flask, frozen in liquid nitrogen and lyophilized. 
Crude products were extracted from the pellet using 73\,\% ethanol and purified using \gls{iex}.
\Gls{iex} was performed on a sulfopropyl sepharose matrix (25\,ml) via isocratic elution (500\,mM \cf{HCl}).
%Therefore the column was equilibrated using 500\,mM \cf{HCl} and the crude extract injected.
Before injection, the crude extract was acidified to 0.5\,M \cf{HCl} using concentrated hydrochloric acid.
After elution, the product containing fractions were dried via lyophilization. 
The amount of product was determined by \gls{uvvis}-spectroscopy at 260\,nm using the published extinction coefficient of \gls{sam} (\textepsilon\textsubscript{0}\,=\,\SI[per-mode=reciprocal]{15400}{\per\molar\per\cm}) after resuspension in water \citep{Shapiro1966}.



\section{Crystallographic Procedures}
\label{sec:xraymethods}

\subsection{Crystallization of proteins}

Commercially available crystallization screens were used to find initial crystallization conditions.
The tested screens included kits available from Hampton Research (Aliso Viejo, USA) and Jena Bioscience (Jena, Germany).
Crystallization screens were processed in 96-well \glspl{mtp}, where each well possessed 4 subwells aligned in a 2$\times$2 matrix. The subwells were divided into 3 shallow wells for sitting drop vapour diffusion experimental setups and a fourth subwell, which was deep enough to act as buffer reservoir.
This way the performance of each crystallization buffer could be assessed using three different protein solutions with varying concentrations, effectors etc.
A pipetting robot (Cartesian Microsys, Zinsser-Analystik; Frankfurt, Germany) was used to mix \SI{200}{\nano\litre} of each, protein and buffer solution, for a final volume of \SI{400}{\nano\litre}.
The crystallization preparations were incubated at \degC{16} and the progress of the experiment was documented by an automated imaging-system (Desktop Minstrel UV, Rigaku Europe, Kent, UK).
Furthermore, fine screens (e.g. for refinement of crystallization conditions) were set up by hand in 24-well \glspl{mtp} using the hanging drop vapour diffusion method.

\subsubsection{\glsentrytext{pfomt}}
\Gls{pfomt} protein was concentrated to \SIrange{6}{8}{\mg\per\ml} and rebuffered to 10\,mM \gls{tris}/HCl pH 7.5 using Amicon\textsuperscript{\textregistered} Ultracel centrifugal concentrators (10\,kDa \gls{mwco}).
The concentrated protein solution was centrifuged at \timesg{14000}, \degC{4} for 10\,min to remove any insoluble material or aggregates.
Flavonoids and phenylpropanoid substrates were added to the protein solution from 10\,mM stock solution in \gls{dmso}. 
Crystallization screens were set up as described above. 

\textit{apo}-\Gls{pfomt} was crystallized using the following conditions -- 2\,M \cf{(NH4)2SO4}, 20\,\%glycerol. The protein solution contained 0.25\,mM \gls{sae}, 0.25\,mM \cf{MgCl2}, 0.25\,mM eriodictyol and \SI{7.53}{\mg\per\ml} (\SI{0.262}{\milli\molar}) \gls{pfomt} \bweigel{ist das korrekt?}.

\subsubsection{Crystallization of proteins using \glsentrytext{nades}}
\Gls{nades} have the potential to be excellent solvents for hydrophobic compounds such as flavonoids or cinnamic acids \citep{Dai2013} and in addition they are able to stabilize and activate enzymes \citep{Huang2014}.
\bweigel{To introduction}

Four different model proteins (bovine trypsin, hen-egg white lysozyme, proteinase K and \textit{Candida cylindrica} lipase B) were used to assess the capability of \gls{nades} for protein crystallization.
\Gls{pch} was tested in a full factorial grid layout using \gls{pch} concentrations of \SIlist{20;30;40;50}{\percent} combined with buffers of different pH.
The buffers included 0.1\,M sodium acetate pH (4.5 and 5.5), 0.1\,M sodium citrate pH 6.5, 0.1\,M \gls{hepes}/NaOH pH (7 and 7.5) and 0.1\,M \gls{tris}/HCl pH 8.5.
Thus, the full factorial design had a size of $4 \times 6 = 24$ different conditions.
Protein solutions were prepared from lyophilized protein and were as follows: \SI{90}{\mg\per\ml} trypsin in \SI{10}{\mg\per\ml} benzamidine, 3\,mM \cf{CaCl2}; \SI{75}{\mg\per\ml} lysozyme in 0.1\,M sodium acetate pH 4.6; \SI{24}{\mg\per\ml} proteinase K in 25\,mM \gls{tris}/HCl pH 7.5 and \SI{6}{\mg\per\ml} lipase B in water.
For crystallization \ul{2} enzyme solution and \ul{1} reservoir buffer were mixed and set up in a hanging drop experiment on a 24-well \gls{mtp}.
The experiments were set up at \degC{4}.


\subsection{Data collection and processing}
Crystallographic data were collected at the beamline of the group of Professor Stubbs (\gls{mlu}, Halle, Germany). 
The beamline was equipped with a rotating anode X-ray source MicroMax007 (Rigaku/MSC, Tokio, Japan), which had a maximum power of \SI{0.8}{\kilo\watt} (\SI{40}{\kilo\volt}, \SI{20}{\milli\ampere}) and supplied monochromatic \cf{Cu}-K\textsubscript{\textalpha}-radiation with a wavelength of 1.5418\,\gls{a}.
Diffraction patterns were detected with a Saturn 944+ detector (CCD++, Rigaku/MSC, Tokio, Japan). 

Indexing and integration of the reflexes via \gls{ft} was accomplished using \emph{XDS} \citep{Kabsch2010,Kabsch1993,Kabsch2010a} or \emph{MOSFLM} \citep{Powell1999}.
\emph{Scala} \citep{Evans2006}, which is integrated in the \gls{ccp4}-Suite, was used for scaling of the intensities.


\subsection{Structure solution}
For the determination of the electron density $\rho(\textrm{\textbf{r}})$, where \textbf{r} is the positional vector, from the diffraction images by \gls{ft} two terms are neccessary as coefficients; the \emph{structure factor amplitudes}, $F_{\mathrm{obs}}(\mathrm{\textbf{h}}$) and the \emph{phase angles} or \emph{phases}, $\alpha\mathrm{(\textbf{h})}$, where \textbf{h} is the reciprocal index vector.
The structure factor amplitudes can be directly determined from the measured and corrected diffraction intensities of each spot.
However, the phase information is lost during the detection of the diffracted photons and there is no direct way to determine the phases.
This constitutes the so-called \emph{phase problem}.
Thus, additional phasing experiments are necessary in order to obtain the phases.
A variety of phasing experiments are available, which include \emph{marker atom substructure methods}, \emph{density modification} and \emph{\gls{mr}} techniques \citep{Rupp2009}.
Phases of the structures herein were exclusively determined by \gls{mr} \citep{Rossmann1962,Rossmann2001}. 
\Gls{mr} was performed using the software \emph{Phaser} \citep{McCoy2007,McCoy2007a}, which is included in the \gls{ccp4}-Suite \citep{Winn2011}. 
A previously published \gls{pfomt} structure (\gls{pdb}-code: 3C3Y \citep{Kopycki2008b}) was used as a template during \gls{mr} procedure for the \gls{pfomt} structure solution.
%%
%\bweigel{Lysozym struktur}
%For the \gls{mr} of the lysozyme structure the \gls{pdb}-entry 4NHI was used.


\subsection{Model building, refinement and validation}
Macromolecular model building and manipulation, as well as real space refinement and Ramachandran idealization were performed using the \gls{coot} software \citep{emsley2010:coot}.
Structure refinement was done using the software REFMAC5 \citep{Murshudov1997,Vagin2004} as part of the \gls{ccp4}-suite or the \gls{phenix} \citep{Adams2010}.
Validation of the structures was carried out using the web service MolProbity (\url{http://molprobity.biochem.duke.edu/}) \citep{Chen2010}.
Structure visualization and the preparation of figures was performed using PyMOL (Schrödinger, New York, USA) and UCSF Chimera (\url{http://www.cgl.ucsf.edu/chimera})\cite{Pettersen2004}.

\subsection{\textit{In silico} substrate docking}
\textit{In silico} molecular docking studies were performed using the AutoDock Vina 1.1.2 or AutoDock 4.2.6 software in combination with the AutoDockTools-Suite (\url{http://autodock.scripps.edu/}) \citep{Huey2007,Morris2009,Trott2010}.
Substrates were docked into the \gls{pfomt} structure with the \gls{pdb}-code 3C3Y.
The grid box, which determines the search space, was manually assigned to center at \numlist{1.581;5.196;25.718} (x, y, z) and had size of \SIlist{22;20;25}{\angstrom} (x, y, z).
The exhaustiveness of the global search for AutoDock Vina was set to 25, whereas the rest of the input parameters were kept at their defaults.

%%%% analytical methods
\section{Analytics}

\subsection{Recording of growth curves}
Starter cultures ($\approx$2\,ml) of the transformed \eco cells were prepared in the medium to be studied, containing the appropriate antibiotics.
The cultures were incubated at \degC{37}, 200\,rpm over night and harvested by centrifugation (\timesg{5000}, \degC{4}, 5\,min).
The pellet was resuspended in 15\,ml \gls{pbs} and the suspension centrifuged (\timesg{5000}, \degC{4}, 5\,min).
The supernatant was discarded and the washing step repeated once more.
The washed pellet was resuspended in 2\,ml of the medium to be studied with the appropriate antibiotics and the \odx{600} was measured.
Three independent 50\,ml cultures of the medium containing the appropriate antibiotics were inoculated such that $\mathrm{OD}^{600}\,\approx\,0.05$ using the washed cell suspension.
The cultures were incubated at the conditions to be studied and sampled at appropriate intervals of time ($\approx$1\,h).
One\,ml samples were kept on ice until all samples were aquired.
\SI{100}{\micro\litre} aliquots of the samples were transferred into a clear \gls{mtp} and the \odx{600} was measured. \\
\Gls{gfp} fluorescence was measured accordingly, but the \gls{mtp} used was opaque.
Exitation ($\lambda^{\mathrm{ex}}$) and emission ($\lambda^{\mathrm{em}}$) wavelengths were \SIlist{470;510}{\nm} respectively.


\subsection{\textit{In vitro} determination of glucose}
The glucose concentration in clarified, aqueous samples was determined by a modified version of the glucose assay kit procedure provided by Sigma-Aldrich \citep{Sigma-Aldrich}. \Gls{god} oxidizes \textsc{d}-glucose to gluconic acid, whereby hydrogen peroxide is produced. 
The hydrogen peroxide can be detected and quantified by \gls{hrp}, which reduces the produced \cf{H2O2} and thereby oxidizes its chromogenic substrate \textit{o}-dianisidine via consecutive one-electron transfers. 
The oxidized diimine form of \textit{o}-dianisidine can then be measured photospectrometrically \citep{Claiborne1979}.

The methodology employs a coupled photospectrometric assay using \gls{god} and \gls{hrp} with \textit{o}-dianisidine as reporter substrate. 
The assay was prepared in \gls{mtp}-format. A reaction solution containing \SI{12.5}{U\per\milli\litre} \gls{god}, \SI{2.5}{U\per\milli\litre} \gls{hrp} and \SI{0.125}{\milli\gram\per\milli\litre} \textit{o}-dianisidine dihydrochloride in \SI{50}{mM} sodium acetate pH\,5.1 was prepared. 
Sample solutions from culture supernatants were typically diluted in 9\,volumes of water.
The reaction was started, by adding \ul{50} reaction solution to \ul{25} of sample and was incubated at \degC{37} and 200\,rpm for 30\,min in a shaking incubator.
\ul{50} 6\,M sulfuric acid was added to stop the reaction and achieve maximum color development (full oxidation of any \textit{o}-dianisidine charge transfer complexes) (\autoref{fig:godassay}). 
The developed pink color was measured at 540\,nm in a \gls{mtp}-reader.
A calibration curve of a standard \textsc{d}-glucose solutions (0 to 100\,\si{\micro\gram\per\milli\litre}), that was always part of the experiments, was used to quantify the sample measurements.


\framedfigure[label = fig:godassay]{
%  \begin{subfigure}{0.2\linewidth}\centering
%    \begin{chemEnv}[0.5]
%      \schemestart
%      D-Glc\arrow{-y>[\chemfig{O_2}, \chemfig{H_2O}][\chemfig{H_2O_2}][GOD]}[,1.5]gluconic acid 
%      \arrow(@c1.south east--.north east){0}[-90,.7]
%      o-dianisidine\arrow{-y>[\chemfig{H_2O_2}][\chemfig{2 H_2O}][HRP]}[,1.5]CTC 
%      \arrow(@c3.south east--.north east){0}[-90,.7]
%      CTC\arrow{->[][\chemfig{H_2SO_4}]}[,1.5]diimine 
%      \schemestop
% \end{chemEnv}
%  \end{subfigure}
%  \begin{subfigure}{0.7\linewidth}\centering
    \begin{chemEnv}[0.75]
      \setarrowdefault{-90}
      \schemestart
    	\Mchemname{\chemfig[][scale=1]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}}{\textbf{diamine}}\arrow{->[*{0}{-\cf{e^-},\cf{H^+}}]}
    	\subscheme{
          1/2 \arrow{0}[0,0.2] \Mchemname{\chemleft[
	      \subscheme{
	        \chemfig[]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}\arrow{0}[0,0.1]
                \chemfig[]{*6((=HN)-(-OMe)=-(=*6(-=-(=NH)-(-OMe)=-))-=--)}    
              }\chemright]}{charge transfer complex}
          %+ \chemfig{H^+}
          }
          \arrow{<=>[*{0}{-\cf{e^-},\cf{H^+}}]}
	\Mchemname{\chemfig[]{*6((=HN)-(-OMe)=-(=*6(-=-(=NH)-(-OMe)=-))-=--)}}{quinonediimine}
    \schemestop
    \end{chemEnv}
%  \end{subfigure}
  
}{Oxidation of the reporter substrate \textit{o}-dianisidine. Consecutive one-electron transfers lead to the fully oxidized diimine form of \textit{o}-dianisidine. The first electron transfer is believed to produce a charge transfer complex intermediate. \citep{Josephy1982a,Claiborne1979}}

\subsection{\textit{In vitro} \glsentryfirst{omt} assay}
\label{sec:omtassay}
\bweigel{nochmal auseinanderklamüsern wegen den konzentrationen und eingesetzten enzymmengen...}
\gls{omt} assays were conducted in a total volume of \SIrange{50}{100}{\micro\litre}. 
The standard assay buffer was 100\,mM Tris/HCl, \uM{2.5} \gls{gsh} pH\,7.5.
\SI{1}{\milli\molar} \cf{MgCl2}, which was otherwise omitted, was added for reactions using cation dependent \glspl{omt} (e.g. \gls{pfomt}).
Reactions contained \SI{0.5}{\milli\molar} alkyl donor (e.g.~(\textit{S},\textit{S})-\glslink{sam}{SAM}) and \SI{0.4}{\milli\molar} flavonoid or cinnamic acid substrate.
Enzymatic reactions were started by addition of enzyme (usually \SI{0.2}{\milli\gram\per\milli\litre}) and incubated at \degC{30}.
%%%%%%%%%%%%%%%%%%%%%%% Platzhalter
Reactions were stopped by addition of \ul{500} ethyl acetate containing 2\,\% formic acid and vortexed for 15\,s to extract the hydrophobic phenylpropanoids and flavonoids.
After centrifugation (\timesg{10000}, \degC{4}, 10\,min) the organic phase was transferred into a new tube.
The reaction was extraced once more with \ul{500} ethyl acetate, 0.2\,\% formic acid and the pooled organic phases were evaporated using a vacuum concentrator (Concentrator 5301; eppendorf, Hamburg, Germany).
The residue was dissolved in methanol and centrifuged at \timesg{10000} for 10\,min to remove unsoluble matter. 
The supernatant was transferred into a \glsentrytext{hplc} vial and analyzed by \gls{hplc} (\ref{sec:hplc}).
When detection of hydrophobic (e.g. flavonoids) and hydrophilic compounds (e.g. \gls{sam}, \gls{sah}) was performed simultaneously reactions were stopped by addition of 0.3 volumes 10\,\% (w/v) \gls{tca} in 50\,\% acetonitrile.
The mixture was vortexed for complete mixing and incubated on ice for at least 30\,min.
After centrifugation (\timesg{10000}, \degC{4}, 10\,min) the supernatant was transferred into \glsentrytext{hplc}-sample vials and analyzed (see \ref{sec:hplc}).

\subsubsection{Measurement of activity/pH profiles}\label{sec:pHassay}
Assays to measure activity over larger pH ranges were set up in 50\,mM \glsentryfirst{mmt}- (pH 4 to 9) or \glsentryfirst{ssg}-buffer (pH 4 to 10) to keep the concentrations of buffer salts constant for each pH \citep{Newman2004}. 
The protein of interest was first extensively dialyzed against the reaction buffer (e.g. \glsentrytext{mmt}, \glsentrytext{ssg}) at pH 7 with added \gls{edta} (\mM{5}) and then against the same buffer without \gls{edta}.
Standard reaction conditions were 50\,mM buffer, 0.4\,mM alkyl acceptor (e.g. caffeic acid), 0.5\,mM \gls{sam}, \uM{2.5} \gls{gsh} and \SI{0.2}{\mg\per\ml} enyzme.
\cf{MgCl2} was either omitted or added at \SI{10}{\milli\molar} to assess influences of divalent cations.
Assays were stopped as described in \ref{sec:omtassay} and analyzed accordingly.

\subsubsection{Estimation of product concentration and enzymatic activities}
\label{ref:concestimation}

Product concentrations were estimated from \gls{hplc} runs. 
The automatically integrated peaks of \gls{sam} and \gls{sah} provided the \gls{auc}.
From the \gls{auc} of both peaks the concentrations were estimated as follows.
%
\begin{align*}
  \intertext{Under the assumption, that}
  AUC^\mathrm{SAH} + AUC^\mathrm{SAM} &= 1 \sim c_0^\mathrm{SAM}, \\
  \intertext{the fraction and concentration of one (e.g. SAH) can be estimated by}
  x^\mathrm{SAH} &= \frac{AUC^\mathrm{SAH}}{AUC^\mathrm{SAH} + AUC^\mathrm{SAM}} \\
  \intertext{and}
  c^\mathrm{SAH} &= x^\mathrm{SAH} \times  c_0^\mathrm{SAM}.
\end{align*}
%
The amount $n$ is obtained by multiplying the concentration $c$ by the injection volume $V$.
Enzymatic activities (i.e. initial rates) can be calculated from the concentrations by standard procedures.

\subsection{Photospectrometric assay for the methylation of catecholic moieties} 
\label{sec:photoassay}
Catecholic moieties can form stable complexes in the presence of heavy metals such as copper or iron \citep{Schweigert2001,Mentasti1973}.
Hence, caffeic acid can complex ferric (\cf{Fe^3+}) ions and form a colored complex with \textlambda\textsubscript{max} = \SI{595}{nm} \citep{Dippe}. 
Since the complex formation is specific for caffeic acid and methylated derivatives (i.e. ferulic and iso-ferulic acid) cannot complex \cf{Fe^3+}, this can be used as a measure for methylation reactions.
\Gls{omt} assays were prepared as before (\autoref{sec:omtassay}). 
However, the reactions were stopped by addition of 0.1 volumes 1\,M Tris/HCl pH 8, immediately followed by 0.5 volumes catechol reagent (\mM{2} \cf{FeCl3} in \mM{10} \cf{HCl}).
The complex formation reaction was allowed to equilibrate for 5\,min at \gls{rt} and the absorbance at \SI{595}{\nm} was measured.

\subsection{Concentration of \glsentrytext{somt} using \glsentryfirst{hic}}
After refolding using rapid dilution protein samples are very dilute and a concentration step is required.
Refolded \gls{somt} was concentrated directly from the refolding buffer using \gls{hic}. 
The ammonium sulfate concentration of the protein sample was brought to 1\,M using a 2\,M \cf{(NH4)2SO4} solution and the pH was adjusted to 7 using 5\,M NaOH.
The sample was centrifuged (\timesg{20000}, \degC{4}, 30\,min) to remove insoluble material and the clarified supernatant was applied to a 1\,ml HiTrap Phenyl FF (Low Sub) (GE Healthcare, Freiburg, Germany), which had been equilibrated with high salt buffer (1\,M \cf{(NH4)2SO4}, 50\,mM \gls{hepes} pH 7).
The target protein was eluted using a stepwise gradient (\SIlist{1;0.8;0.6;0.4;0.2;0}{\molar} \cf{(NH4)2SO4}, 50\,mM \gls{hepes} pH 7; 5\,\gls{cv} each) to remove the ammonium sulfate.
The column was washed using 20\,\% ethanol.
Before \sdspage analysis the eluted high salt fractions were desalted using \gls{tca} precipitation (\ref{sec:tcaprecipitation}).

\subsection{Analytical gel filtration}
Analytical gel filtration was done using a Superdex 200 10/300 GL column (GE Healthcare, Freiburg, Germany) in combination with a \gls{fplc} system according to the manufacturers instructions.
The column was equilibrated using an appropriate buffer (e.g. 0.1\,M \gls{tris}/HCl pH 7.5) and \ul{100} of sufficiently concentrated ($\geq$\SI{1}{\mg\per\ml}) protein sample were injected.
The Gel Filtration Standard by Bio-Rad (München, Germany) was run seperately to assess the size of the proteins in the analyzed sample.


\subsection{Binding experiments using \Glsentryfirst{itc}}
\Gls{itc} can be used to directly characterize the thermodynamics of an observed process, be this a binding interaction or an enzymatic reaction \citep{Freyer2008}.   
\Gls{itc} measurements to describe the interaction between \gls{pfomt} and its substrates/effector where performed using a MicroCal iTC200 device (Malvern, Worcestershire, UK).
\Gls{pfomt} protein was extensively dialyzed against 50\,mM \gls{mmt}-buffer pH 7 prior to \gls{itc} experiments.
The solution was susequently centrifuged (\timesg{14000}, \degC{4}, 10\,min), to remove insoluble matter and aggregates.
The dialysate was stored at \degC{4} and used to prepare substrate and effector solutions.
Generally \uM{50} protein was provided in the \gls{itc} cell and the effectors/substrates to be titrated were loaded into the syringe.
The substance concentration in the syringe was ten times higher than the protein solution.
Experiments were carried out at \degC{20} unless otherwise stated. The stirring speed was set to 500\,rpm.
The injection volume was set to \SIrange{2}{4}{\micro\litre}, amounting to a total of \numrange{10}{19} injections.


\subsection{\Glsentryfirst{hplc} analytics}
\label{sec:hplc}
Due to their aromaticity, methanolic extracts of flavonoids exhibit two major absorption peaks in the \gls{uvvis} region of the light spectrum in the range of \SIrange{240}{400}{\nm} \citep{Mabry1970}.
However, even the more simple phenyl propanoids (e.g. cinnamic acids) show absorption of light in the \gls{uvvis}-region. 
Methanolic extracts of flavonoids and phenyl propanoids were analyzed by \gls{hplc} using a \gls{pda}-detector, which was set to record in the range of \SIrange{200}{400}{\nm}.
\Gls{hplc} runs were performed on a reverse-phase C-18 end-capped column (YMC-Pack ODS-A; YMC Europe, Dinslaken, Germany) with a pore size of 120\,\gls{a}.
The mobile phase was aqueous acetonitrile supplemented with 0.2\,\% formic acid. The flow was kept constant at \SI{0.8}{\milli\litre\per\minute}.
\SI{10}{\micro\litre} \gls{omt} enzyme assay extract (\ref{sec:omtassay}) were injected and analyzed using an acetonitrile gradient starting with 5\,\% acetonitrile (4\,min). 
The acetonitrile content was increased to 100\,\% in 21\,min and was kept at 100\,\% for 5\,min.
%Beside the \gls{pda} data, four distinct traces (200, 220, 280 and 325\,nm) were also recorded.
Peaks were integrated from the 280\,nm trace using the software provided by the manufacturer of the device.

\subsection{\Glsentryfirst{lcmsms} measurements}

The positive and negative ion high resolution \gls{esi} and \gls{cid} MSn spectra as well as \gls{hcd} MS/MS spectra were obtained from an Orbitrap Elite mass spectrometer (Thermo Fisher Scientific, Bremen, Germany) equipped with an \gls{hesi} ion source (positive spray voltage \SI{4.5}{\kilo\volt}, negative spray voltage \SI{3.5}{\kilo\volt}, capillary temperature \degC{275}, source heater temperature \degC{250}, \gls{ftms} \gls{rp} \num{30000}). 
Nitrogen was used as sheath and auxiliary gas. 
The MS system was coupled with an \gls{uhplc} system (Dionex UltiMate 3000, Thermo Fisher Scientific), equipped with a RP-C18 column (particle size \SI{1.9}{\micro\metre}, pore size \SI{175}{\angstrom}, 50\,x\,\SI{2.1}{\milli\metre} inner diameter, Hypersil GOLD, Thermo Fisher Scientific, column temperature \degC{30}) and a photodiode array detector (\SIrange{190}{400}{\nano\metre}, Thermofisher Scientific). 
For the \gls{uhplc} a gradient system was used starting from \cf{H2O}:\cf{CH3CN} 95:5 (each containing \SI{0.2}{\percent} formic acid) raised to 0:100 within 10\,min and held at 0:100 for further 3\,min. 
The flow rate was \SI{150}{\micro\litre\per\minute}. 

The mass spectra (buffer gas: helium) were recorded using \glspl{nce} of \SIrange{30}{45}{\percent} and \SIrange{75}{100}{\percent} for \gls{cid} and \gls{hcd} mass spectra respectively (see Appendix).
The instrument was externally calibrated using the Pierce\textsuperscript{\textregistered} LTQ Velos ESI positive ion calibration solution (product number 88323, Thermofisher Scientific, Rockford, IL, 61105 USA) and the Pierce\textsuperscript{\textregistered} LTQ Velos ESI negative ion calibration solution (product number 88324, Thermofisher Scientific, Rockford, IL, 61105 USA) for positive and negative ionization mode respectively.
%The data were evaluated by the Xcalibur software 2.7 (Thermo Fisher Scientific). 
