%%%-------------------------------------------------%%%
%%% Sub document for material and methods %%%
%%%-------------------------------------------------%%%

\chapter{Material And Methods}

%%%% materials (chemicals, inbstrumenmts, software etc)
\section{Materials}
\subsection{Chemicals}
Enzymes and buffers used for molecular cloning were obtained from Thermo Scientific (Darmstadt, Germany), unless otherwise noted. 
Flavonoid HPLC standards were purchased from Extrasynthese (Genay, France). 
Deuterated solvents were aquired from Deutero GmbH (Kastellaun, Germany). 
Solvents, purchased from VWR (Poole, England), were distilled in-house before use. \\
All other chemicals were obtained from either Sigma-Aldrich (Steinheim, Germany), Applichem (Darmstadt, Germany), Carl Roth (Karlsruhe, Germany) or Merck (Darmstadt, Germany).

\subsection{Instruments}

\begin{small}
\begin{longtable}{p{0.35\textwidth}p{0.65\textwidth}}
  \gls{cd}-spectrometer & Jasco J-815 (Eaton, USA) \\
  electrophoresis (horizontal) & Biometra Compact XS/S (Göttingen, Germany)\\
  electrophoresis (vertical) & Biometra Compact M (Göttingen, Germany) \newline Biometra Minigel-Twin (Göttingen, Germany) \\
  \gls{fplc} & ÄKTA purifier (GE Healthcare, Freiburg, Germany) \\
  \gls{gcms} & GC-MS-QP2010 Ultra (Shimadzu, Duisburg, Germany)\\
  \gls{hplc} & VWR-Hitachi LaChrom Elite (VWR, Darmstadt, Germany) \\
  \gls{itc} & MicroCal iTC200 (Malvern, Worcestershire, UK) \\
  \gls{mtp}-reader & SpectraMax M5 (Molecular Devices, Biberach, Germany) \\
  \gls{nmr}-spectrometer & Varian Unity 400 (Agilent, Böblingen, Germany) \newline Varian VNMRS 600 (Agilent, Böblingen, Germany) \\
  photospectrometer & Eppendorf Biophotometer Plus (Hamburg, Germany) \newline JASCO V-560 (Eaton, USA) \newline Colibri Microvolume Spectrometer (Biozym, Hess. Oldendorf, Germany) \\
  centrifuges & Eppendorf 5424 (Hamburg, Germany) \newline Hettich Mikro 120 (Kirchlengern, Germany) \newline Beckman Avanti J-E, Beckman Allegra X-30R (Krefeld, Germany)  \\
  centrifuge rotors & Beckman JA-10, JA-16.250, JS-4.3 (Krefeld, Germany)    
\end{longtable}
\end{small}

\subsection{Bacterial strains}

\subsubsection{\textit{E.coli}}
\begin{longtable}{p{0.3\textwidth}p{0.68\textwidth}}
  BL21(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Invitrogen, Karslruhe (Germany) \\
  C41(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) \newline Lucigen, Wisconsin (USA) \\
  C43(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3)\newline Lucigen, Wisconsin (USA) \\
  DH5$\alpha$ & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{lacZYA-argF}) U169 \textit{recA}1 \textit{endA}1 \textit{hsdR}17(r$_K^-$m$_K^+$) \textit{phoA supE}44 $\lambda^-$ \textit{thi}-1 \textit{gyrA}96 \textit{relA}1 \newline Invitrogen, Karlsruhe (Germany) \\
  JM110 & \textit{rpsL thr leu thi lacY galK galT ara tonA tsx dam dcm glnV}44 $\Delta$(\textit{lac-proAB}) e14- [F' \textit{traD}36 \textit{proAB}$^+$ \textit{lacI}$^q$ \textit{lacZ}$\Delta$M15] \textit{hsdR}17(r$_K^-$m$_K^+$) \newline Martin-Luther-University Halle-Wittenberg \\
  JW1593 \newline (BW25113 derivative) & \textit{rrnB} $\Delta$\textit{lacZ}4787 \textit{HsdR}514 $\Delta$(\textit{araBAD})568 \textit{rph}-1 $\Delta$\textit{ydgG} (Kan$^R$) \newline Keio Collection, National Institute of Genetics (Japan) \\
  MG1655 & F$^-$ $\lambda^-$ \textit{ilvG}$^-$ \textit{rfb}-50 \textit{rph}-1 \newline DSMZ, Hamburg (Germany) \\
  One Shot TOP10 & F$^-$ $\Phi$80\textit{lacZ}$\Delta$M15 $\Delta$(\textit{mrr-hsdRMS-mcrBC}) \textit{recA}1 \textit{endA}1 \textit{mcrA} $\Delta$\textit{lacX}74 \textit{araD}139 $\Delta$(\textit{ara-leu})7697 \textit{galU galK rpsL} (Str$^R$) $\lambda^-$ \textit{nupG} \newline Invitrogen, Karlsruhe (Germany) \\
  Origami(DE3) & $\Delta$(\textit{ara-leu})7697 $\Delta$\textit{lacX}74 $\Delta$\textit{phoA} \textit{Pvull phoR araD}139 \textit{ahpC galE galK rpsL} F'[\textit{lac + lacI q pro}] (DE3)\textit{gor}522::\textit{Tn}10 \textit{trxB} (Kan$^R$, Str$^R$, Tet$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  Rosetta(DE3) pLysS & F$^-$ \textit{ompT} \textit{hsdSB}(r$_B^-$,m$_B^-$) \textit{gal dcm} $\lambda$(DE3) pLysSRARE (Cam$^R$) \newline Novagen, Wisconsin (USA) \\
  T7 Express & \textit{fhuA}2 \textit{lacZ::T}7 \textit{gene}1 \textit{[lon] ompT gal sulA}11 \textit{R}(\textit{mcr-}73\textit{::miniTn}10--Tet$^S$)2 \textit{[dcm] R}(\textit{zgb}-210\textit{::Tn}10--Tet$^S$) \textit{endA}1 $\Delta$(\textit{mcrC-mrr})114\textit{::IS}10 \newline NEB, Massachusetts (USA) \\
\end{longtable}

\subsubsection{\textit{Agrobacterium tumefaciens}}
\begin{tabular}{p{0.2\textwidth}p{0.8\textwidth}}
GV3101 & chromosomal background: C58, marker gene: \textit{rif}, \gls{tip}: cured, opine: nopaline \newline Sylvestre Marillonet, IPB \\  
\end{tabular}


\subsection{Plasmids}
\ostTableEnv[label = tab:plasmids]{%
  \begin{tabular}{p{0.2\textwidth}p{0.5\textwidth}}
    \toprule
	name & description \\ 
    \midrule
    pACYC Duet-1 & .. \\
    pCDF Duet-1 & .. \\
    pET20b(+) & ... \\
    pET28a(+) & .. \\
    pET32a(+) & .. \\
    pET41a(+) & .. \\
    pQE30 & .. \\
    pUC19 & .. \\
    \bottomrule
  \end{tabular}
}{%
Plasmids used in this work.
}

\subsection{Oligonucleotides}
Oligonucleotides and primers were ordered from Eurofins Genomics (Ebersberg, Germany). The purity grade was \emph{high purity salt free} (HPSF).

\ostTableEnv[label = tab:primers]{%
  \begin{tabular}{p{0.15\textwidth}p{0.7\textwidth}p{0.15\textwidth}}
    \toprule
    	name & sequence (5'$\rightarrow$3') & cloning site \\ 
    \midrule
%    	T7 & TAATACGACTCACTATAGGGT & n/a \\
%       T7.term & GCTAGTTATTGCTCAGCGGT & n/a \\
        somt1 & TT\underline{GAAGAC}AAAATGGCTTCTTCATTAAACAATGGCCG & BpiI \\
        somt2 & TT\underline{GAAGAC}AAGGACACCCCAAATACTGTGAGATCTTCC & BpiI \\
        somt3 & TT\underline{GAAGAC}AAGTCCTTAGGAACACCTTTCTGGGAC & BpiI \\
        somt4 & TT\underline{GAAGAC}AAAAGCTCAAGGATAGATCTCAATAAGAGAC & BpiI \\
    \bottomrule
  \end{tabular}
}{%
Primers used in this work. Recognition sites for endonucleases are underlined. 
}

\subsection{Software}
All mathematical and statistical computations and graphics were done with the \verb+R+ software (versions 3.1.X, \url{http://cran.r-project.org/}). Visualizations of macromolecules were arranged using the PyMol Molecular Graphics System, version 1.7.0.0 (Schrödinger, New York (USA)).\\
Physicochemical calculations and calculations of different molecular descriptors were performed using Marvin Beans 15.4.13.0 (ChemAxon, Budapest (Hungary)) and Molecular Operating Environment 2008.10 (Chemical Computing Group, Montreal (Canada)).

%%%% Microbiologicyl Methods
\section{Microbiology}


%%%% Molecular Biology methods
\section{Molecular Biology}
Basic molecular biology methods like \gls{pcr}, DNA restriction/ligation, DNA gel electrophoresis, preparation of competent cells and transformation were performed based on the protocols summarized by Sambrook and Russell \citep{Sambrook2001}.\\
Plasmid DNA was isolated using the QIAprep Spin Miniprep Kit (QIAGEN, Hilden (Germany)) according to the manufacturer's instructions. \\
In vitro site-directed mutatgenesis was set-up according to the protocol of the \emph{QuikChange Site-Directed Mutagenesis} kit \cite{Technologies} offered by Agilent Technologies (Santa Clara, USA). \\
Nucleotide fragments obtained by \gls{pcr}, restriction/ligation procedures or excision from electrophoresis gels were purified and concentrated using the \emph{Nucleospin Gel and PCR Clean-up} kit provided by Machery-Nagel (Düren, Germany) according to the instructions provided by the manufacturer.

\subsection{Golden Gate Cloning}

The Golden Gate cloning procedure is a one-pot method, meaning the restriction digestion and ligation are carried out in the same reaction vessel at the same time \cite{Kondou2011,Engler2008}. Consequently \gls{pcr}-fragments, destination vector, restriction endonuclease and ligase are added together in this reaction. The methodology employs type II restriction enzymes, which together with proper design of the fragments allow for a ligation product lacking the original restriction sites.  \\
For digestion/ligation reactions of fragments containing BpiI sites, 20\,fmol of each fragment or vector, together with 5\,U of BpiI and 5\,U of T4 ligase were combined in a total volume of \ul{15} 1$\times$ ligase buffer.
For fragments to be cloned via BsaI sites, BpiI in the above reaction was substituted by 5\,U BsaI. \\
The reaction mixture was placed in a thermocycler and inbcubated at \degC{37} for 2\,min and \degC{16} for 5\,min. These two first steps were repeated 50 times over. Finally, the temperature was raised to \degC{50} (5\,min) and \degC{80} (10\,min) to inactivate the enzymes.

%\subsection{Polymerase chain reaction (\protect{\Gls{pcr}})}
%\subsection{\Glsentryfirst{pcr}}
%\Glssymbol{pcr} was performed using the \emph{KOD Hot Start DNA Polymerase} kit (Novagen, Wisconsin (USA)).
%\cite{Sambrook2001}



%%%% protein biochemical methods
\section{Protein biochemistry}
Stock solutions of antibiotics, IPTG or sugars were generally prepared according to the pET System Manual by Novagen \citep{Novagen2011}.

\subsection{Protein production test (Expression test)}
The heterologous production of proteins in \eco was assessed in a small scale protein production test, henceforth called expression test.
Single colonies of \eco transformed with the constructs to be studied were used to inoculate a 2\,ml starter culture in \gls{lb}-medium containing the appropriate antibiotics. 
The concentrations of antiobiotics used was as follows:
\ugperml{200} ampicillin, \ugperml{150} kanamycin, \ugperml{50} chloramphenicol, \ugperml{20} tetracycline. \\
The starter culture was allowed to grow at \degC{37} and 200\,rpm over night.
A 5\,ml sampling culture in the medium to be studied containing the appropriate antibiotics was prepared. The media tested included \gls{lb}, \gls{tb} and auto-induction media like \gls{zyp}.
The sampling culture was inoculated to an \odx{600} of 0.075 using the starter culture and incubated at different temperatures and 200\,rpm in a shaking incubator.
1\,mM \gls{iptg} was added when the \odx{600} reached 0.6-0.8, if appropriate for the studied construct.
1\,ml samples were removed after different times of incubation (e.g. 4, 8, 12 hours), subfractionated and analyzed via \gls{sds}-\gls{page}.


\subsection{Production of recombinant protein}
\subsubsection{Heterologous production of \glsentrytext{pfomt}}


%%%% analytical methods
\section{Analytics}
\subsection{In vitro determination of glucose}
The glucose concentration in clarified, aqueous samples was determined by a modified version of the glucose assay kit procedure provided by Sigma-Aldrich \citep{Sigma-Aldrich}. \Gls{god} oxidizes D-glucose to gluconic acid, whereby hydrogen peroxide is produced. The hydrogen peroxide can be detected and quantified by \gls{hrp}, which reduces the produced \cf{H2O2} and thereby oxidizes its chromogenic substrate \textit{o}-dianisidine. The oxidized form of \textit{o}-dianisidine can then be measured photospectrometrically \citep{Claiborne1979}.\\
The methodology employs a coupled photospectrometric assay using \gls{god} and \gls{hrp} with \textit{o}-dianisidine as reporter substrate. 
The assay was prepared in \gls{mtp}-format. A reaction solution containing \SI{12.5}{U\per\milli\litre} \gls{god}, \SI{2.5}{U\per\milli\litre} \gls{hrp} and \SI{0.125}{\milli\gram\per\milli\litre} \textit{o}-dianisidine dihydrochloride in \SI{50}{mM} sodium acetate pH\,5.1 was prepared. \\
Sample solutions from culture supernatants were typically diluted in 9\,volumes of water.
The reaction was started, by adding \ul{50} reaction solution to \ul{25} of sample and was incubated at \degC{37} and 200\,rpm for 30\,min in a shaking incubator.
\ul{50} 6\,M sulfuric acid was added to stop the reaction and achieve maximum color development (full oxidation of any \textit{o}-dianisidine charge transfer complexes). \\
The developed pink color was measured at 540\,nm in a \gls{mtp}-reader.
A calibration curve of a standard D-glucose solutions (0 to 100\,\si{\micro\gram\per\milli\litre}), that was always part of the experiments, was used to quantify the sample measurements.


\framedfigure[label = fig:godassay]{
  \begin{subfigure}{0.4\linewidth}\centering
    \begin{tikzpicture}[node distance=2.5cm]\tiny
     
     \node (0,0) (a) {D-Glc + \newline \cf{O2} + \cf{H2O}};
     \node [right of=a] (b) {gluconic acid + \cf{H2O2}};
     \draw[->] (a) -- node[midway, above] {GOD} (b);
		
     \begin{scope}[nodes = {shape=rectangle, draw=none, below=1cm}]
       \node 		at (a) (c) {\cf{H2O2} + \textit{o}-dianisidine};
       \node        at (b) (d) {CTC};
     \end{scope}
     \draw[->] (c) -- node[midway,above] {HRP} (d);
    
     \begin{scope}[->, nodes = {shape=rectangle, draw=none, below = 2cm}]
       \node 		at (a) (e) {CTC};
       \node        at (b) (f) {ox. dianisidine};      
     \end{scope}
     \draw[->] (e) -- node[midway, above] {\cf{H2SO4}} (f);

   \end{tikzpicture}
  \end{subfigure}
  \begin{subfigure}{0.4\linewidth}\centering
    \setatomsep{2em}
    \setarrowdefault{-90}
	\schemestart
    	\chemname{\chemfig[][scale=0.5]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}}{diamine}\arrow{->[*{0}{-2\cf{e^-},2\cf{H^+}}]}
	\chemname{\chemfig[][scale=0.5]{*6((-H_2N)-(-OMe)=-(-*6(-=-(-NH_2)=(-OMe)-=))=-=)}}{charge transfer complex}\arrow{<=>}
	\chemname{\chemfig[][scale=0.5]{*6((=HN)-(-OMe)=-(-*6(-=-(=NH)-(-OMe)=-))-=--)}}{quinonediimine}
    \schemestop
      \end{subfigure}
  
}{GOD assay}

\subsection{\glsentrytext{hplc} measurements}
