%%%-------------------------------------------------%%%
%%% Sub document for Evaluation of PFOMT towards homologous alkylation %%%
%%%-------------------------------------------------%%%

\chapter{Engineering of \glsentryfirst{pfomt}}
\newpage
%\phantomsection
\label{ch:pfomt}
%\addcontentsline{toc}{chapter}{Evaluation of PFOMT towards the acceptance of long-chain SAM analogues}

  
\ospSetTitle{Evaluation of PFOMT towards the acceptance of long-chain SAM analogues}
\ospSetAuthors{\underline{Benjamin Weigel}\textsuperscript{1,a}, %
  Martin Dippe, %
  Ludger A. Wessjohann\textsuperscript{1,c}}
\ospSetContacts{bweigel@ipb-halle.de\textsuperscript{a}, law@ipb-halle.de\textsuperscript{c}}
\ospSetAffiliations{Leibniz-Institute of Plant Biochemistry, Department of Bioorganic Chemistry\textsuperscript{1}}
\ospSetKeywords{methyl transferase, pfomt, SAM}

\WEBmakeTitle{%

\textbf{Abstract}

The cation dependent \pfomt from the ice plant, \textit{Mesembryanthemum crystallinum}, methylates a number of flavonoids and phenyl propanoids.
A newly solved crystal structure of the protein without any bound ligand shows the fully resolved N-terminus, which acts as a lid to close off the active site.
Binding of co-substrates (analogues) (e.g. \gls{sah}, \gls{sam}, \gls{sae}) is more entropically driven as the chain length increases.
However, even though the ethyl-analogue of \gls{sam} -- \gls{sae} -- was shown to bind to the enzyme, no conversion of the model substrate caffeic acid was observed for the wild-type and several engineered variants.
}

\section{Introduction}

Small changes to molecules can have profound influences on their chemical, physical and biological properties.
For example, butyric acid esters differing only by a few methylene groups already exhibit quite divergent smells. 
However, not only the macroscopically qualitative properties can differ.
The quantifyable psychotomimetic effect of methylated and ethylated lysergic acid amids differ by at least an order of magnitude \citep{Shulgin1997,Hofmann2000}.
There are many more of these so-called \gls{sar} and \gls{qsar} studies on any number of compounds \citep{Santos2014,Alexander1966,Mani1991}.

Methylation reactions are one of the key tailoring steps during natural product biosynthesis and can in consequence greatly affect a molecules bio- and physicochemical behavoir \citep{Struck2012,Ley2005}.
\Glspl{mt} catalyze the transfer of a methyl group from the co-substrate \gls{sam} to an activated atom of the acceptor molecule \cite{Struck2012}.
%The cation-dependent \gls{mt} from the ice plant, \mesem, \gls{pfomt} was used as a model systemmethylates various flavonoids and phenyl propanoids and is a well behaved 
 
Between the highly complex core structures of natural products, which are produced by a plethora of enzymes (e.g. \glspl{pks}, \glspl{nrps}, terpene cyclases), and the rather simple alkyl-modification introduced by methylation, nature is missing some medium-sized modifaction options that proceed as elegantly as the methylation by \glspl{mt}.
Thus, natural products containing longer chain alkyl modifications like ethyl or propyl moieties on O, N or S-centers have rarely, if ever been observed.\footnote{Reaxys searches for natural product isolates with a molecular mass between (150 and 1500) containing the substructures methyl, ethyl or propyl connected to a heteroatom return 66759, 2797 and 52 results respectively. However, it stands to note that 70\,\% of the propyl results were either esters or otherwise activated moieties. \citep{Elsevier}}

It has recently been shown however, that a wide array of \gls{sam} analogues are used as co-substrates by a variety of \glspl{mt} \citep{Struck2012}.  
The majority of the work so far has been done on \glspl{pmt} and \glspl{dmt}, since epi-genetics and finding regions of gene-regulation is of great interest. 
However, \glspl{smmt} have also been shown to accept different \gls{sam} analogues \cite{Stecher2009,Lee2010,Law2015,Singh2014,Zhang2006}.
There have been a great many of \gls{sam} analogues synthesized, both chemically and enzymatically, that were consequently studied with the help of \glspl{mt} \citep{Struck2012,Dalhoff2006,Singh2014}.

The \gls{omt} \gls{pfomt} is a highly promiscuous enzyme with regards to its flavonoid substrates and has extensively been characterized \citep{Vogt2004,Ibdah2003,Kopycki2008b,Brandt2015}.
However, the promiscuity towards different \gls{sam} analogues has net yet been described.
Combination of both, substrate and co-substrate promiscuity in the small molecule \gls{mt} \gls{pfomt} could provide a powerful tool towards the biosynthestic production of novel small molecules with potentially new and promising biological activities.
Functionalization/Detection of substrates could furthermore provide a means of finding new compounds/substrates in complex (e.g. biological) samples analogous to \gls{abpp} approaches.

In this work we show, that \gls{pfomt} binds the co-substrate analogues \gls{sah}, \gls{sam} and \gls{sae} with similar affinities. 
A newly developed crystal structure of the \textit{apo}-enzyme shows the fully resolved N-terminus is lodged in a cleft atop the active site, closing it off. 
Although semi-rationally designed enzyme variants could not afford enzymatic ethylation of substrates, the regio-selectivity of the methylation reaction was altered.

%The already extensively studied \gls{pfomt} was the prime candidate, since the preparation and crystallizability were established and lots of substrates had already been described \citep{Vogt2004,Ibdah2003,Kopycki2008b,Brandt2015}.

%\section{\textit{In silico} docking studies using computational tools}
% \bweigel{entweder noch machen oder raus damit}
 
%Prior to any experimental techniques different known and potential substrates were docked to  using %molecular modelling.


\section{Crystallization of \glsentrytext{pfomt}}

The crystal structure of \pfomt was published in 2008, however binding of substrates could not be accomplished \citep{Kopycki2008b}.
Nonetheless, the demethylated co-substrate \gls{sah} was cocrystallized.
The first goal of this study was to crystallize the \textit{apo}-form of the enzyme, to obtain a system that allows for the soaking of substrates.
At the same time, \pfomt was to be cocrystallized along with an acceptor substrate and the co-substrate analogs \gls{sae} and \gls{sah}.

%% The binding of the non-natural substrate \gls{sae} to \gls{pfomt} could be shown.
%% However, a transethylation reactivity was not observed.
%% The question was to the chemical reasons behind these observations.
%% Previous work on the crystal structure of \gls{pfomt} had been done, but only \gls{sah} could be co-crystallized \citep{Kopycki2008b}.
%% But since the crystallizability of \gls{pfomt} had already been shown, this method was chosen to answer the aforementioned question.

At first the already available crystallization procedures were evaluated \citep{Kopycki2008b}. 
However, reproduction of these results could not be accomplished and new crystallization conditions had to be found.
%This was done using commercially crystallization screening kits and a semi-automated pipetting robot along with an automated imaging system for the observation of the crystallization plates.

Several commercially available buffer solutions (see \autoref{sec:xraymethods}) were screened in combination with different protein solutions (e.g. solutions containing co-substrates and acceptor substrates or not) to obtain protein crystals co-crystallized with substrates or of the \textit{apo}-form.
%During the preparation of the protein solutions it was noted, that upon addition of the flavonoids or phenyl propanoids from \gls{dmso} stocks these tended to precipitate.
Crystals were obtained in various wells after a few days.
The crystal shape varied from very smooth and almost cubic (high ammonium sulfate) over sphreulites and intergrown crystals (\cf{CaCl2}, PEG-4000) to brittle and ragged needles (\cf{LiCl}, PEG-6000)  (\autoref{fig:d_xtals}).

\framedfigure[label = fig:d_xtals]{%
  \begin{subfigure}{0.18\textwidth}
    \includegraphics[trim=0 0 0 0, clip]{usr/graphics/static/MP000715_C8-3_64d}
    \caption{}
  \end{subfigure}
  \begin{subfigure}{0.18\textwidth}
    \includegraphics[trim=0 0 0 0, clip]{usr/graphics/static/MC001414_D4-2_92d}
    \caption{}
  \end{subfigure}
  \begin{subfigure}{0.18\textwidth}
    \includegraphics[trim=0 0 0 0, clip]{usr/graphics/static/MC001414_C12-1_3d}
    \caption{}
  \end{subfigure}
  \begin{subfigure}{0.18\textwidth}
    \includegraphics[trim=0 0 0 0, clip]{usr/graphics/static/MC001414_D4-1_92d}
    \caption{}
  \end{subfigure}
  \begin{subfigure}{0.18\textwidth}
    \includegraphics[trim=0 0 0 0, clip]{usr/graphics/static/MC001414_G10-1_64d}
    \caption{}
  \end{subfigure}
}{%
  Some crystal and pseudo-crystal shapes that were observed during the crytsallization screen.
  a -- high \cf{(NH4)2SO4}, b-c -- \cf{CaCl2}, PEG-4000, e -- \cf{LiCl}, PEG-6000
}

Crystals that were large enough ($\geq$ \SI{50}{\micro\metre} \bweigel{wie gro√ü?}), where screened for diffraction at the home-source after cryoprotection.
A rough estimate of the resolution, cell parameters and the space group was aquired, if the diffraction images could be indexed.
The screened crystals all had similar cell parameters and belonged to the same space group, $P2_12_12_1$, as the previously published structure (pdb: \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y})\citep{Kopycki2008b}.
However, the unit cell of crystals that grew out of high ammonium sulfate concentrations ($\geq$\SI{1.8}{\molar}) was approximately four times as large as that of the published structure. % and crystrals that developed under different crystallization conditions.
%Consequently the assymetric unit consisted of 4, instead of 2 \gls{pfomt} monomers.
Several datasets were collected of crystals from high \cf{(NH4)2SO4}, since these seemed to be promising candidates to find differences in the bound substrates.
Datasets of crystals that grew from other conditions were insufficient for structure solution.


\subsubsection{The crystal structure of \textit{apo}-PFOMT}

%Most of the collected datasets were partly solved.
%As it turned out however the substrates were not co-crystallized.
\Gls{pfomt} crystallized without any bound substrates under conditions of high \cf{(NH4)2SO4}.
One dataset was solved to completion to obtain a complete structure of this novel \textit{apo}-\gls{pfomt} at a resolution of \SI{1.95}{\angstrom} (\autoref{tab:xtalstat}).
The assymetric unit of \emph{apo}-\Gls{pfomt} contained two homodimers (4 monomers) (\autoref{fig:d_struct_a}), rather than just one homodimer (\href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y}).
The active site of each monomer was found to be empty except for a sole sulfate ion, which was positioned where the amino- and carboxylate groups of the \gls{sah} residue in the \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y} structure (\autoref{fig:d_struct_b}).
Shifts in the structure of some loops were observed and contrary to the previously published structure the entire N-terminus was resolved up to and including the His-tag.

The resolved N-terminus contained another N-terminal \textalpha-helix, which was positioned in a cleft on the surface, where substrates may be bound \citep{Kopycki2008b}.
This interaction extends up to the His-tag.
Considerable movement was observed in different parts of the protein, when no substrate was bound, some of which can be attributed to \gls{sam} and metal ion binding residues (\autoref{fig:d_struc.compare} and \autoref{fig:app_struc.compare}) as is obvious for the loop region between \textbeta-sheet 1 and \textalpha-helix 4.
Nonetheless, most of the movement seemed to be restricted to areas, which are not directly involved in the binding of either \gls{sam} or metal ions.
However, all of the regions that moved are located at or near the active site.


\framedfigure[label = fig:d_struct]{%
  \begin{subfigure}{\textwidth}\centering
%    \includegraphics[scale=0.225, trim=0 0 0 0, clip]{usr/graphics/static/tetramer.png}
    \includegraphics[width=0.9\textwidth, trim=0 0 0 0, clip]{usr/graphics/static/tetramer.png}
    \caption{}
    \label{fig:d_struct_a}
  \end{subfigure}
  
  \begin{subfigure}{\textwidth}\centering
%    \includegraphics[scale=0.175, trim=0 0 0 0, clip]{usr/graphics/static/pfomt_monomer.png}
    \includegraphics[width=0.9\textwidth, trim=0 0 0 0, clip]{usr/graphics/static/pfomt_monomer.png}
    \caption{}
    \label{fig:d_struct_b}
  \end{subfigure}
    
}{%
An overview of the features in the \emph{apo}-\gls{pfomt} structure. \textbf{\subref{fig:d_struct_a}} -- The assymetric unit of \textit{apo}-\gls{pfomt} consists of two homodimers (4 monomers). Individual monomers are rainbow colored from N- (blue) to C-terminus (red). %
 \textbf{\subref{fig:d_struct_b}} -- Comparison of \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y} (steelblue) and \textit{apo}-\gls{pfomt} (green). The N-terminus of \textit{apo}-\gls{pfomt} was resolved up to the N-terminus (red) and even the His-tag (red, transparent) was partly resolved.
      The N-terminus fits into a cleft on the surface of the \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y} structure, shown as a surface model on the right.
      \gls{sah} (white ball-and-sticks) and \cf{Ca^2+} (green sphere) are featured in the published structure, whereas a sulphate ion (red/yellow spheres) was bound in the newly solved structure.
}

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    <<struct.compare,  eval=F, echo=F, tidy=TRUE, fig.height=2, fig.width=5.6, out.width='\\textwidth', dev='tikz', error=T>>=
    @
    \includegraphics[width=\textwidth, trim=0 0 0 0, clip]{usr/graphics/static/struct_compare-1}
  \end{figureframe}
    \caption{Positional differences between the individual residues of the solved \textit{apo}-PFOMT and the structure with bound \gls{sah} (pdb: \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y}). The \glsentryfirst{dpi} of the structures was \SIlist{0.137;0.064}{\angstrom} respectively.
      The overall \gls{rmsd} amounted to \SI{0.9034}{\angstrom}.
  The secondary structure of apo-\gls{pfomt} is displayed at the top. Helices are displayed as rectangles and sheets are shown as arrows.
  Graphical background annotations are used to display the binding sites of \gls{sah} (green) and the metal ion (plum).
  The orange bars indicate regions, where much movement seems to happen upon binding or release of the co-substrate.
  The blue bar shows the region that was annotated as "insertion loop" in previous studies \citep{Kopycki2008b}.}
  \label{fig:d_struc.compare}
\end{figure}

Unfortunately soaking of these ``\textit{apo}''-crystals did not afford binding of substrates.
%\clearpage

\section{Substrate binding studies using \glsentrytext{itc}}

The binding of different substrates to \gls{pfomt} was examined by \gls{itc}, to determine whether the enzyme can bind non-natural \gls{sam} analogues.
The homologues \gls{sah}, \gls{sam} and \gls{sae} were selected to also study the influence of the alkyl chain length on binding (\autoref{fig:d_itc_substances}).
Furthermore the binding of the substrate caffeic acid and the influence of \cf{Mg^2+} addition on substrate binding was investigated.

\begin{figure}[htb]
  \begin{figureframe}
    %\framedfigure[label = fig:d_itc_substances]{%
    \includegraphics[width=\textwidth]{usr/graphics/static/itc-sam}
    \caption{The binding of different \glsentrytext{sam} analogues was measured via \glsentrytext{itc}}
    \label{fig:d_itc_substances}
  \end{figureframe}
\end{figure}
%}{%
   %    
%    }

The $K_D$ values of \gls{sah}, \gls{sam} and \gls{sae} were all in the low micromolar range, around \SI{2}{\micro\molar}.
However, the binding enthalpy clearly decreased with the length of the aliphatic chain connected to the sulfur atom (\autoref{fig:itc.a}).
The binding of \gls{sah}, gave off more heat than the binding of \gls{sam}, which in turn gave off more heat than the binding of \gls{sae} (\autoref{tab:ditc}).
Thus, the entropic influence must get larger with increasing chain length in order for equations \eqref{eq:gibbs1} and \eqref{eq:gibbs2} to still hold true.

\begin{equation} \label{eq:gibbs1}
  \Delta G = \Delta H - T \Delta S 
\end{equation}
\begin{equation} \label{eq:gibbs2}
  \Delta G = \Delta G^0 - RT~\mathrm{ln}\,K 
\end{equation}

Indeed, the value for $\Delta S$ was negative for binding of \gls{sah}, but positive for the binding of \gls{sam} and \gls{sae} (\autoref{tab:ditc}). 
This relationship between the change of entropy and the change of enthalpy has been found for many biological systems and is called \gls{eec} \citep{Searle1995,Gilli1994,Dunitz1995}.
The stoichiometry for the binding process is given by the parameter $N$.
For all the ligands \gls{sah}, \gls{sam} and \gls{sae} this value was found to be about 0.5, which corresponds to one bound molecule ligand per dimer of \pfomt (\autoref{tab:ditc}).

Upon titration of caffeic acid to \pfomt small amounts of released heat were detected for the system (\autoref{fig:itc.c}). 
When the enzyme was incubated with \gls{sah} prior to addition of caffeic acid the released heat was slightly increased.
The slope of the \gls{itc} profile also got steeper.
However, the data obtained could not be fitted to afford a sensible solution.
When caffeic acid and \cf{Mg^2+} were incubated with \pfomt prior to addition of \gls{sah}, the process of heat production as observed by \gls{itc} had a steeper slope (\autoref{fig:itc.b}). 
Nonetheless, the thermodynamic parameters did not differ significantly.
\cf{Mg^2+}, in the form of an \cf{MgCl2} solution, titrated to the enzyme solution did not cause signals during the \gls{itc} experiments.



\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \begin{subfigure}{0.45\textwidth}
      \includegraphics[scale=0.25]{usr/graphics/static/itc_sah-sam-sae}
      \caption{}
      \label{fig:itc.a}
    \end{subfigure}
    \begin{subfigure}{0.45\textwidth}
      \begin{subfigure}{\textwidth}
        \includegraphics[width=\textwidth]{usr/graphics/static/itc_sah-1}
        <<itc.sah,  eval=F, echo=F, tidy=TRUE, fig.height=4.5, fig.width=8, out.width='\\textwidth', dev='tikz', error=T>>=
        @
        \caption{}
        \label{fig:itc.b}
      \end{subfigure}
     
      \begin{subfigure}{\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/itc_ca-1}
        <<itc.ca, eval=F,  echo=F, tidy=TRUE, fig.height=4.5, fig.width=8, dev='tikz', error=T>>=
        @
        \caption{}
        \label{fig:itc.c}
      \end{subfigure}
    \end{subfigure}
  \end{figureframe}
    \caption{ITC measurements of \pfomt:effector binding. \textbf{\subref{fig:itc.a}} -- Binding of \gls{sah}, \gls{sam} and \gls{sae} to \gls{pfomt}. 
      \textbf{\subref{fig:itc.b}} -- SAH is injected into a \gls{pfomt} solution, with (red) or without (black) addition of \cf{Mg^2+} and caffeic acid. 
          When \cf{Mg^2+} and caffeic acid were already present, the binding process seems to happen quicker, but is less enthalpic.
          \textbf{\subref{fig:itc.c}} -- Upon addition of caffeic acid to the protein heat is produced, however no sensible binding curve could be obtained.}
  \label{fig:d_itc.sah}
\end{figure}

%\ostTableEnv[label=tab:ditc]{%
%\begin{lstTabularx}[1]{\textwidth}{Xrrrr}%
\begin{table}[htb]\centering\small
  \caption{Results of fitting a simple one-site binding model to the data obtained from \gls{itc} experiments.} %\\
  \label{tab:ditc}
  \begin{tabular}{lrrrr}
  %\hfill \\
    \toprule
    & $K_D$ [\si{\micro\molar}]  & $\Delta H$ [\si[per-mode=reciprocal]{\cal\per\mole}] & $\Delta S$ [\si[per-mode=reciprocal]{\cal\per\mole\per\kelvin}] & N  \\
    \midrule
    SAH & \num[separate-uncertainty]{2.06(427)} &  \num[separate-uncertainty]{-10380(1025)} & \num[separate-uncertainty]{-9.41} & \num[separate-uncertainty]{0.505(38)} \\
    SAM & \num[separate-uncertainty]{1.08(350)} &  \num[separate-uncertainty]{-4606(242)} & \num[separate-uncertainty]{11.6} & \num[separate-uncertainty]{0.492(18)} \\
    SAE & \num[separate-uncertainty]{2.22(379)} &  \num[separate-uncertainty]{-1338(190)} & \num[separate-uncertainty]{21.3} & \num[separate-uncertainty]{0.513(50)} \\
    \bottomrule
\end{tabular}
\end{table}
%\end{lstTabularx}
%}{%
%Results of fitting a simple one-site binding model to the data obtained from \gls{itc} experiments.
%}

\section{Study of variants for long-chain alkylations}

Since the ability to bind the enlongated analogue \gls{sae} was present in wild-type \gls{pfomt}, the activity of the \gls{pfomt} protein towards \gls{sae} was tested.
Activity tests were performed with caffeic acid as substrate under standard reaction conditions.
Unfortunately no ethylation of the substrate by \pfomt was observed, even after extended incubation times.
%The wild-type of \gls{pfomt} was able to use \gls{sae} as a co-substrate for the ethylation of caffeic acid, albeit the amount of detected product was very minute. \bweigel{literaturvergleich der ums√§zte?}
%The site of ethylation was determined by \gls{lcms} measurements. \bweigel{LCMS messungen?}
%It was found that ethylation occurs on the catecholic group, however it could not be determined whether at the 3- or 4-position.
%Nonetheless it is highly likely that ethylation occurs at the same position as methylation and thus the product was annotated as 3-ethoxy-4-hydroxy cinnamic acid. 
%Enzyme variants were prepared to further test the ethylation reactivity of \gls{pfomt}, since a number of groups were able to accomplish transalkylation with larger substrates by expanding the available space in the active site \citep{Wang2011}.

Consequently enzyme variants were prepared to achieve a \pfomt variant with an ethylation activity, since a number of groups were able to accomplish transalkylation with larger substrates by expanding the available space in the active site \citep{Wang2011}.
The available crystal structures of \pfomt were consulted to select suitable residues.
Residues that were exchanged were selected based upon their position in the active site and in relation to the substrate(s) (\autoref{fig:d_activesite_resis}).
The residues were exchanged to the non-spaceous alanine, as well as amino acids frequently observed at homologous positions in other class I \glspl{omt}.
%Fortunately a crystal structure of \gls{pfomt} was available to help with the selection. \bweigel{Genauer zu den Resten...}

\framedfigure[label = fig:d_activesite_resis]{%
\includegraphics[width=0.5\textwidth, trim=300 100 300 100, clip]{usr/graphics/static/d_activesite_resis}
}{%
   The active site of \gls{pfomt} (pdb: \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y}). 
   The outline of the protein backbone is displayed, with active site residues portrayed as colored sticks (cyan -- F103, red -- F80, turqouise -- M52, yellow -- Y51, white -- F198, blue -- W184, orange -- N202, grey -- as labelled).
   The co-substrate \gls{sam} (ball-and-stick model) was docked into the structure. %    
    }

Over 20 enzyme variants were prepared to assess, whether \gls{pfomt} ethylation activity would improve over the wild-type.
However, no ethylation activity was observed for either variant.
%Be that as is may, an improved ethylation activity was not observed. 
Some of the new variants however displayed an increased methylation activity with the substrates caffeic acid and \gls{sam} (\autoref{fig:pfomt_DIM}).
The methylation activity of some of the variants increased by over 4-fold.
Interestingly most amino acid substitutions proved as beneficial.

Methylation activity benifited greatly from the replacement of bulky hydrophobic residues by smaller and/or charged residues in the vicinity of the acceptor substrates (Tyr51, Trp184 and Phe198).
However, this was not a general trend since the substitutions N202W and Y51W also improved methylation activity.
Looking more closely at residue Tyr51, the activity enhancing effect was greatest, when the tyrosine was substituted by the basic amino acids lysine or arginine.
In addition to an enhanced activity the selectivity for the hydroxyl position to be methylated was also altered in these variants.
This was not apparent, when caffeic acid was used as a substrate. 
However when a flavonoid, especially eriodictyol, was used not only the 3' hydroxyl, but to some extent the 4' hydroxyl was methylated (\autoref{fig:app_pfomt_3to4}).
This effect was improved in some double variants, where also position 202 was altered. 
For example the variant Y51R N202W almost exclusively methylated flavonoid substrates at the 4' position\bweigel{grafik? \& paper}.
A detailed discussion of the results was published in a peer reviewed journal.

\begin{figure}[htpb]
  \begin{figureframe}
    \includegraphics[width=\textwidth]{usr/graphics/static/pfomt_DIM}
  \end{figureframe}
  \caption{Activities of different \pfomt variants towards caffeic acid methylation. Colorations correspond to the ones used in \autoref{fig:d_activesite_resis}.}
  \label{fig:pfomt_DIM}
\end{figure}
  

\section{Conclusion/Discussion}

\bweigel{mehr auf die aktuelle literatur eingehen...auch wenn sie sich i grenzen h√§lt...}
Whereas the binding of \gls{sah} was solely dependent on the large negative enthalpy, the binding of \gls{sae} was almost entirely driven by entropy, since $\Delta H$ was close to 0 (\autoref{tab:ditc}).
Entropy gain can be a major driving force for ligand-protein interactions and in some cases ligand binding can be entirely attributed this gain in entropy \citep{Li2008}.
Displacement of protein-bound water molecules contributes strongly to the entropic gain.
There were some waters present in the active site of \pfomt in the crystal structure developed herein.
However, no metal ion was present in the active site in the \textit{apo}-\pfomt structure. 
Furthermore \cf{Mg^2+} titration via \gls{itc} did not afford significant signals, suggesting the notion, that the metal is only bound along with the co-substrate (\autoref{fig:asite_waters})\bweigel{im buch nochmal bei comt nachlesen}. 
It has been suggested, that the entropy cost to transfer one water molecule from bulk to the protein-bound state can be up to \SI[per-mode=reciprocal]{7}{\cal\per\mole\per\kelvin} \citep{Dunitz1994}.
The replacement of ordered waters from the active site or from a hydrated metal ion by a growing aliphatic chain could therefore explain the gain in entropy, and \gls{sah} is positioned in a way to warrant exactly that (\autoref{fig:asite_waters}). 
Also, the hydrogen and metal complexing bonds consequently lost could explain the less negative enthalpy.
However, this is purely hypothetical since more evident data is missing.
Additional insight might be gained by expanding the \gls{itc} experiments to even longer \gls{sam} analogues.
The limited space in the active site, which forces the growing side chain to expel water and possibly the metal ion might also be the reason for the inactivity of \pfomt towards \gls{sae}. 
If the metal ion is blocked from its complexing moieties, activation of the substrate hydroxyl would be hindered.

Comparison of the novel \textit{apo}-\pfomt and the published structure (pdb: \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y}) suggests that the movement (upon ligand binding) along multiple parts of the backbone promixal to the active site pocket is a main contributer to the overall \gls{rmsd} of \SI{0.9}{\angstrom} (\autoref{fig:d_struc.compare}).


\begin{figure}[htpb]
  \begin{figureframe}
    \begin{subfigure}{0.48\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/apo_asite.png}
      \caption{}
      \label{fig:asite_waters_a}
    \end{subfigure}
    \begin{subfigure}{0.48\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/3c3y_asite.png}
      \caption{}
      \label{fig:asite_waters_b}
    \end{subfigure}
  \end{figureframe}
\caption{Comparison of the active sites of \textbf{\subref{fig:asite_waters_a}} -- the solved \textit{apo}-structure (green) and \textbf{\subref{fig:asite_waters_b}} -- the ligand-bound structure (steelblue; pdb: \href{http://www.rcsb.org/pdb/explore/explore.do?structureId=3c3y}{3C3Y}).
Waters are represented as small red spheres, calcium as a green sphere (complexing bonds are dashed) and \gls{sah} is displayed as a white ball-and-stick model. 
A possible hydrogen bond network (blue lines) for the ligand-bound state is displayed.}
\label{fig:asite_waters}
\end{figure}


The N-terminus of \pfomt seems to act as a lid, which is closed in the \textit{apo}-form, but highly flexible and therefore unresolved in the ligand bound form.
Furthermore, the native enzyme has been shown to be truncated, starting only at residue 12 and being less catalytically efficient than the full length protein \cite{Vogt2004,Kopycki2008b}.
The work presented here consequently supports the notion that the N-terminus plays an important role on the regulation of the enzymatic activity.

During our studies, transethylation activities could not be observed for any of the prepared \pfomt variants.
However, some of the variants showed higher methylation activities towards caffeic acid and even different regioselectivities (3'\textrightarrow 4') than the wild-type.

Given the fact that only residues in the active site and therefore in direct contact with the substrates were prepared, the laid out findings provide novel hints for indirect proximal regions in the \pfomt structure that might be studied using site-directed mutagenesis, gene-shuffling or similar approaches in order to work towards a variant that can in fact employ \gls{sae} for transalkylation reactions.
Furthermore variation of these regions might provide variants with altered substrate specificities which are of high interest.




%The existing crystal structure of \gls{pfomt} (pdb: 3c3y) shows 3 water molecules that could be disp%laced by a growing aliphatic chain (\autoref{fig:asite_waters}). 
%All of these waters are complexing a \cf{Ca^2+} atom.

%% \framedfigure[label = fig:pfomt_vogt]{%
%% \includegraphics[width=0.5\textwidth]{usr/graphics/static/pfomt_vogt.png}
%% }{%
%% from: Kopycki, J. G., Rauh, D., Chumanevich, A. a., Neumann, P., Vogt, T., \& Stubbs, M. T. (2008). Biochemical and Structural Analysis of Substrate Promiscuity in Plant Mg2+-Dependent O-Methyltransferases. Journal of Molecular Biology, 378(1), 154‚Äì164. http://doi.org/10.1016/j.jmb.2008.02.019
%%}

\section{Contributions}

Benjamin Weigel wrote the manuscript, prepared figures, sub-cloned, produced and crystallized \pfomt, solved the \textit{apo}-structure and conducted the \gls{itc} experiments. 
Dr. Martin Dippe prepared most of the \gls{pfomt} variants and ethylation activity tests.
Dr. Christoph Partier (group of Prof. Dr. Milton T. Stubbs, \gls{mlu} Halle-Wittenberg) helped collect X-ray datasets.
