%%%-------------------------------------------------%%%
%%% Sub document for methylation of non-catechols %%%
%%%-------------------------------------------------%%%

\chapter{Enzymatic methylation of Non-catechols}

\section{Introduction}
Non-catechols in nature (biosynthesis, mode of action?), chemical methylation???

phenol vs methoxyphenols vs catechols \\

The 4'-hydroxyl of naringenin is non-catecholic in nature.
However it is much more acidic than the 4'-hydroxyl of eriodictyol (pKa 9.8 vs 12.7) and thus about equally as acidic as the 3'-hydroxyl of eriodictyol (pKa 9.7) \footnote{pKa values were calculated using ChemAxon's MarvinBeans 15.2.16.0} \citep{Schweigert2001a}.

\section{\glsentrytext{somt}}

\somt has been described in the literature to methylate multiple flavonoids at the 4'-position of the B-ring \citep{Kim2005c,Kim2005b}.
There it also showed the highest activity towards naringenin, to produce ponciretin (also known as isosakuranetin).
%Non-chemical methylations of this nature are of great scientific and economic interest.
Furthermore it has been the only characterized enzyme of this nature described in literature. \bweigel{is that true?}
Thus, \somt was selected as a model candidate for enzymes that can methylate 4'-hydroxyls of non-catecholic flavonoids.



\subsection{\textit{In vivo} biotransformation in \nico}
\bweigel{eventuell weglassen? wo ist der mehrwert fÃ¼r die arbeit???}
The group of Sylvestre Marillonet (IPB) established an efficient system to clone and assemble multi enzyme pathways in \nico, using a modular cloning toolbox, which has already been used to produce flavonoids \cite{Konig2014}.
The group had already cloned all the enzymes required to establish the pathway up to naringenin in \nico (\autoref{fig:somtpathway}).
However, the \textit{SOMT2} gene needed to be cloned into suitable vectors to be transiently expressed by \nico.
The gene was first cloned into a level 0 module using BpiI and consequently subcloned into a level 1 module flanked by a promoter and a terminator using BsaI (\autoref{fig:cloning}).
The 35S-promoter was used alonside the \gls{nos}-terminator to achieve the highest possible transcription rates. 

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \begin{tikzpicture}[scale=0.75, every node/.style={transform shape}]
      \tikzstyle{cpd}=[rectangle, fill=blue!30]
      \path 	(0,0) node[cpd] (a) {phenyl alanine} 
      (3.5,0) node[cpd] (b) {cinnamic acid}
      (7,0) node[cpd] (c) {coumaric acid}
      (10.5, 0) node[cpd] (d) {coumaroyl CoA}
      (7.5, -0.5) node (da) {3 x malonyl CoA}
      (10.5, -2) node[cpd] (e) {naringenin chalcone}
      (10.5, -4) node[cpd] (f) {naringenin}
      (10.5, -6) node [draw=red, cpd] (g) {4'-O-methyl-naringenin};
		
      \draw[->] (a) -- node[above, gray] {PAL} (b);
      \draw[->] (b) -- node[above] {C4H} (c);
      \draw[->] (c) -- node[above] {4CL} (d);
      \draw[->] (d) -- node[right, gray] {CHS} (e);
      \draw[-] (da.east) .. controls +(right:1cm) and +(up:0.5cm) .. (e.north) ;
      \draw[->] (e) -- node[right] {CHI} (f);
      \draw[->] (f) -- node[right, red] {SOMT2} (g);
    \end{tikzpicture}
    \end{figureframe}
    \caption{Semi-synthetic pathway to naringenin and 4'-O-methyl naringenin in \textit{N. benthamiana}. Enzymes not endogenous to \textit{N. benthamiana} are in gray. PAL - phenylalanine ammonia lyase, C4H - cinnamic acid 4-hydroxylase, 4CL - 4-coumaric acid:CoA ligase, CHS - chalcone synthase, CHI - chalcone isomerase, SOMT2 - soy O-methyl transferase 2}
    \label{fig:somtpathway}
\end{figure}

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=\textwidth]{usr/graphics/static/tmp_cloning}
  \end{figureframe}
  \caption{Cloning of \somt}
  \label{fig:cloning}
\end{figure}

Both sides of \nico leaves were infiltrated with different samples.
The left side was infiltrated with \agro cultures transformed with pAGM10733 (\gls{pal}), pAGM10406 (\gls{chs}) and pBEW107 (\somt).
For the right side the \agro culture containing pBEW107 was replaced by a control: \agro transformed with the empty vector pICH75044. 
After 7 days, the plant material was harvested.
The average weight loss after freeze drying was \SI{87.5}{\percent}.

The dried material was extracted and analyzed via \gls{hplc} to determine whether ponciretin or related compounds were produced (\autoref{tab:naringenin}).
However, through comparison with authentic standards it was apparent, that none of the expected compounds were detected.
This finding suggest, that neither naringenin, nor any resulting flavonoids (ponciretin, poncirin, didymin) were present in detectable amounts in the plant tissue at the time of harvest.
Although unlikely, it cannot be exluded that higher amounts of the compounds of interest were present at some point in the tissue.
Numerous more experiments were required to address the issue of non-detection, which however was outside the scope of this work.


\begin{lstTabularx}[1]{\linewidth}{clll}
  %\ostTableEnv[label=tab:naringenin]{%
 %  \begin{tabularx}{\textwidth}{clll}%
  \caption{Naringenin and 4'-methylated derivatives.} \\
  \label{tab:naringenin}
  \hfill \\
    \toprule
    \multirow{4}{*}{%
      \includegraphics{usr/graphics/static/naringenin}%
    }%
    & R\textsuperscript{1} & R\textsuperscript{2} & name \\
    \cmidrule{2-4}
    & H & H & naringenin \\
    & \cf{CH3} & H & ponciretin \\
    & \cf{CH3} & rutinose \footnote{\textalpha -\textsc{l}-rhamnopyrasonyl-(1\ra6)-\textbeta -\textsc{d}-glucopyranose} & poncirin \\
    & \cf{CH3} & neohesperidose \footnote{\textalpha -\textsc{l}-rhamnopyrasonyl-(1\ra2)-\textbeta -\textsc{d}-glucopyranose} & didymine \\
    \bottomrule
 % \end{tabularx}
\end{lstTabularx}
%}{%
%Naringenin derivatives.
%}

The \gls{hplc} chromatograms were analyzed by \glsentryfirst{pca} after the data were aligned, centered and scaled, to assess whether the collected plant material samples were different from one another (\autoref{fig:e_pca} and \ref{fig:x_pca}).
The \gls{pca}-plot shows that the samples of the different leaf sides do not separate, indicating no difference between infiltration with the \textit{SOMT} gene and vector control between the first two principal components, which account for \SI{80}{\percent} of the variance.
However, there is a slight separation between top and bottom leaves in the second principal component and between plant 3 and plants 1/2 in the first principal component.
This suggest, that the chemical composition as detected by \gls{hplc} is slightly different in the top and botton leaves, as well as between plants.


\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.5\textwidth]{usr/graphics/static/tmp_pca2}
  \end{figureframe}
  \caption{\gls{pca} of leaf material. The samples are colored by leaf side.}
  \label{fig:e_pca}
\end{figure}



\subsection{\textit{In vivo} biotransformation in \eco}

Kim \etal already showed, that \somt could be used for the biotransformation of different flavonoids in \eco live cultures \citep{Kim2005b,Kim2005c}.
Their technique was to be studied and expanded upon to gain additional insight into the enzyme.
The \textit{\somt} gene was cloned into the pET28a(+) and pET41a(+) vectors, to obtain constructs for the production of \somt without and with a N-terminal \gls{gst}-tag, respectively since both have been used successfully by Kim \etal.
However, methylated flavonoids were not detected  when biotransformations were prepared according to the methods of the aforementioned authors (\autoref{fig:e_invivomethod}). 

Thus, the biotransformation medium was changed to auto-induction medium (\gls{zyp}) \citep{Studier2005}.
\SI{0.05}{\percent} glucose were present in the medium.
Growth curves were measured, alongside glucose depletion, to establish the time at which substrate addition would be most beneficial.
Furthermore, \sdspage samples were prepared to observe protein accumulation.
The glucose present in the medium was depleted after about 5\,hours into growth.
At that time expression of the \textit{\somt} gene is expected to begin, because the catabolite repression on the \textit{lac}~promoter is relieved.

Thus, \SI{0.1}{\milli\molar} flavonoid were added at 4\,hours to minimize its influence on growth and possible degradation.
Although \sdspage samples were prepared throughout the course of the experiment, accumulating \somt could not be clearly distinguished from endogenous \eco protein in the \sdspage gels (\autoref{fig:x_somtgel}).
Nonetheless, methylation of some of the tested substrates was observed over a course of 30\,hours (\autoref{tab:invivosomt}).


\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/tmp_somt_invivo_method}
  \end{figureframe}
  \caption{Biotransformation methods as described by Kim \etal (left) and developed in this work (right).}
  \label{fig:e_invivomethod}
\end{figure}

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/tmp_e_growthcurve}
  \end{figureframe}
  \caption{Growth curve of \eco BL21(DE3) expressing \textit{\somt} at \degC{37}. 
  Glucose is depleted about 5\,hours into growth, at which point the start of \textit{\somt} expression is expected.
  The \odx{600} after inoculation was about \num{0.1}.}
  \label{fig:e_growthcurve}
\end{figure}


\gls{lcms} was employed to determine the site of methylation, since this method is highly sensitive and numerous structural studies on flavonoids using tandem-mass spectrometry experiments have highlighted the feasability of this approach  \citep{Fabre2001,Lee2005}.
\Gls{cid} was used to obtain structural information about the target molecules, since soft ionization techniques (e.g. \gls{esi}) used in \gls{lcms} instruments primarily produce protonated and deprotonated molecular ions, but rarely yield fragments \citep{Sleno2004}.
The \gls{cid} method collides the precursor ions with a neutral target gas while increasing the energy to induce fragmentation.
The produced fragments vary depending on the energy chosen for fragmentation.
Flavonoids follow certain different fragmentation pathways \citep{Fabre2001,Lee2005}. 
The fragmentation of interest in this work, was the one along the C-ring, which produces two fragments (A- and B-ring) (\autoref{fig:e_MSscheme}).
The mass of the A- and B-ring fragments gives strong evidence for the position (ring) at which methylation occured.
Using the \gls{cid} technique, an energy of \SI{30}{\electronvolt} proved sufficient to fragment most flavonoids along the C-ring as is shown here for the methylated naringenin (\autoref{fig:e_LCMS_ponciretin}).
The molecular ion [M+H]\textsuperscript{+} of the methylated naringenin has a \gls{mz} of 287.092.
The fragments helping to derive structural information are \mz{133} and \mz{153}, which can only be explained if the B-ring was methylated (\autoref{fig:e_MSscheme}).
If the A-ring was methylated, the expected fragment ions of A and B-ring would have \gls{mz}-values of 167 and 119 respectively.


\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \begin{subfigure}{\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/MS2_naringenin}
      \caption{CID spectrum (\SI{30}{\electronvolt}) of methylated naringenin.}
    \end{subfigure}
    
    \begin{subfigure}{\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/MS2_naringenin_scheme}
      \caption{Scheme displaying the fragmentation of ponciretin.}
      \label{fig:e_MSscheme}
    \end{subfigure}
%    
  \end{figureframe}
  \caption{The masses resulting from the fragmentation into A- and B-ring along the C-ring (dashed line, \textbf{b}) are evidence, that the 4'-hydroxyl on the B-ring is methylated by \somt}
  \label{fig:e_LCMS_ponciretin}
\end{figure}


The \gls{lcms} results suggest, that methylation occured exclusively at the 4'-hydroxyl.
There was no conversion detected, when a 4'-hydroxyl was absent (\autoref{tab:invivosomt}).
A free 4'-hydroxyl seems therefore necessary for a substance to be a substrate for \somt, which confirms the previous results by Kim \etal \citep{Kim2005b}. 
Conversion was observed for flavonoids and the stilbene resveratrol, although conversion rates of the isoflavone genistein were very low.
No conversion of anthraquinones, cinnamic acid derivates or chalcones was detected, which is also in accordance with previously published data.
\somt acts on phenolic, catecholic as well as (4-hydroxy-3-methoxy-phenyl)-moeities, as is suggested by the assay results that showed methylation of naringenin, eriodictyol and homoeriodictyol respectively.
The methylation of (4-hydroxy-3-methoxy-phenyl)-moiteties and of stilbenes are properties of \somt that have not been described before.

The conversion ratios were assessed, but are beset with large errors due to the nature of \eco rich medium extracts.
The highest conversions were observed for flavanones and flavones (up to $\geq$\SI{55}{\percent}). The tested isoflavones and flavonols showed much lesser conversion ratios (less then \SI{10}{\percent}).
The conversion ratios of apigenin ($\geq$\SI{54}{\percent}) and naringenin ($\geq$\SI{55}{\percent}) are comparable to the ones published by Kim \etal.
However, genistein only showed minute conversions, which is converse to the data published by their group \citep{Kim2005b}.
Conversion of eriodictyol, homoeriodictyol and kaempferol were not reported before.

The biotransformation of resveratrol to 3,5-dihydroxy-4'-methoxy-stilbene showed a conversion ratio of $\geq$\SI{86}{\percent} in 30 hours.
This is roughly double the conversion which was recently reported for \invivo biotransformations using the specific resveratrol \gls{omt} sbCOM1, which only achieved \SI{42}{\percent} conversion in 36 hours \citep{Kang2014}.

\begin{lstTabularx}[1]{\linewidth}{XlllX}
  %\ostTableEnv[label=tab:naringenin]{%
 %  \begin{tabularx}{\textwidth}{clll}%
  \caption{\textit{In vivo} biotransformation of different flavonoids, phenylpropanoids and anthraquinones by \somt in \eco} \\
  \label{tab:invivosomt}
  \small
  \hfill \\
    \toprule
    substrate & class & 4'-OH & conversion\footnote{Calculation of conversion percentages are only rough estimates, because of the nature of crude medium extracts.} & product\footnote{as determined by \gls{lcms}} \\
    \midrule
    alizarin & anthraquinone & no & no & -- \\
    purpurin & anthraquinone & no & no & -- \\
    apigenin & flavone & yes & yes ($\geq$\SI{54}{\percent}) & 4'-\textit{O}-methyl apigenin \\  
    chrysin & flavone & no & no & -- \\
    genistein & isoflavone & yes & yes ($<$\SI{1}{\percent}) & Biochanin A\footnote{4'-\textit{O}-methyl genistein} \\
    galangin & flavonol & no & no & -- \\ 
    kaempferol & flavonol & yes & yes ($\geq$\SI{6}{\percent}) & kaempferide\footnote{4'-\textit{O}-methyl kaempferol} \\
    naringenin & flavanone & yes & yes ($\geq$\SI{55}{\percent}) & ponciretin \\
    eriodictyol & flavanone & yes & yes ($\geq$\SI{40}{\percent}) & hesperetin \\
    homoeriodictyol & flavanone & yes & yes ($>$\SI{6}{\percent})& 3',4'-(\textit{O,O})-dimethyl eriodictyol\\
    hesperetin & flavanone & no & no & -- \\
    phloretin & chalcone & yes & no & -- \\
    resveratrol & stilbene & yes & yes ($\geq$\SI{86}{\percent})& 4'-\textit{O}-methyl resveratrol \\
    \textit{p}-coumaric acid & cinnamic acid & yes & no & -- \\ 
    caffeic acid & cinnamic acid & yes & no & -- \\
    reosmin & cinnamic acid\footnote{homo-dihydro cinnamic ketone} & yes & no & -- \\ 
    \bottomrule
 % \end{tabularx}
\end{lstTabularx}


\subsection{\textit{In vitro} studies using recombinantly produced \glsentrytext{somt}}

\textit{In vivo} biotransformations are an important tool for the primary characterization of enzymes.
However, because live organisms are used and lots of variables are unknown, these systems can cause large errors and are not fit to thouroughly characterize an enzyme.
Initially \somt was to be purified to homogeneity to be later thourougly characterized \invitro, since Kim \etal had showed the recombinant production of \somt in \eco as a fusion protein with an N-terminal \gls{T7tag}, but had missed to characterize the recombinant enzyme.

\subsubsection{Protein production test}
Initial protein production tests were carried out using \textit{\somt} cloned into pET28a(+) with an N-terminal \gls{histag}.
However, \somt was not produced in soluble form (\autoref{fig:e_somtgel_exp}).
Numerous systems were tested for the expression of \textit{\somt}. 
\eco strains used for the trials included BL21(DE3), Rosetta(DE3), Origami(DE3), C41(DE3), C43(DE3), C41(DE3) pLys, C43(DE3) pLys and DH5\textalpha. 
The \textit{\somt} gene was cloned into multiple other vectors, inluding pET20b for periplasmic protein proiduction, pET32 for expression with an \gls{trx} and vectors that carry promoters for induction by rhamnose.
Multiple media, including \gls{tb}, \gls{lb} and autoinductiuon media were used along with different inducers (e.g. lactose, rhamnose, \gls{iptg}) at different temperatures.
Nonetheless \somt could not be produced in a soluble form and expression only resulted in \glspl{ib}.

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[scale=0.25]{usr/graphics/static/WEB214_SDSPAGE}
  \end{figureframe}
  \caption{\sdspage of pET28a(+) \textit{\somt} expressed in \eco BL21(DE3) in autoinduction medium at different temperatures (shown above). The insoluble fractions show a protein band the same height as the \SI{40}{\kilo\dalton}) marker band, which corresponds to the \somt protein (\SI{40425}{\dalton}). M -- protein size marker, S -- soluble fraction, I -- insoluble fraction}
  \label{fig:e_somtgel_exp}
\end{figure}

\subsubsection{\textit{In vitro} protein refolding}

Since the \somt protein could not be obtained in soluble formen, when recombinantly expressed in \eco, the \glspl{ib} were prepared and used for \invitro refolding studies.
Protein refolding is no trivial task. 
The refolding process competes with misfolding and aggregation processes and refolding buffers have to be optimized in order to obtain an efficient refolding system with the best possible results \cite{Willis2005,Yamaguchi2014,Rudolph1996}.
Refolding efficiency is best measured via biological activity, but even with adequate assays refolding studies are a time-consuming process of trial-and-error.
The number of experiments required to even test only four variables, for example pH, salt, temperature and protein concentration with 3 states each (e.g. low, medium, high) in all possible combinations results in $3^4=81$.
An experimental setup, which accounts for all possible variable (factor) combinations is also called a \emph{full factorial design}.
These setups capture main effects, as well as higher level interaction effects \citep{Montgomery2001,Box2005}.
However, for screening purposes only a fraction of the experiments can be run.
The objective of these \glsentryfirst{ffd} experiments is to identify the variables, which have large effects and are worth expanding the experimental investigation upon.
\Glspl{ffd} have been successfully used for a number of protein refolding trials \citep{Willis2005,Vincentelli2004,Tobbell2002}.

The following factors were studied for the \invitro refolding of \somt: pH, arginine addition, glycerol addition, addition of divalent cations, ionic strength, redox system, cyclodextrin addition and effector (\gls{sah}) addition (see tables \ref{tab:ffd_factors} and \ref{tab:ffd_matrix}).
Two factor levels were used in a twelve-run design. This is sufficient to find some main effects, however no statement about interaction effects can be made.

The \sdspage gels that were prepared of the soluble and insoluble fraction already show big differences between folding buffers.
Refolding buffers 2,3 and 8--11 mainly produced insoluble protein, whereas the majority of the protein in refolding buffers 1, 4--7 and 12 was in soluble form in these buffers after an overnight refolding reaction (\autoref{fig:e_refold_gel}). 
After rebuffering the cleared refolding reactions into a unified buffer the protein concentrations were estimated by Bradford \citep{Bradford1976}.
The protein concentrations are consistent with the \sdspage gels.
Soluble protein was obtained for buffers 1, 4--7 and 12.
The highest amount of soluble protein was present, when the refolding reaction took place in buffers 5 or 7.


\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/refold_sdspage}
  \end{figureframe}
  \caption{\sdspage of the insoluble and soluble fractions of the refolding reactions.}
  \label{fig:e_refold_gel}
\end{figure}


The common denominator of those buffers is that all of them contained arginine, whose addition has proven beneficial for many refolding applications \citep{Chen2008a,Tsumoto2004,Fujimoto2010}.
\Glspl{meplot} illustrate the difference between level means for each factor.
The \meplot for protein concentration shows, that arginine seems to be an important factor for the refolding of \somt (\autoref{fig:MEplot_BF}).
Furthermore, the addition of \gls{sah} or glycerin and the pH seem to have an influence, whereas the other factors seem to play only a minor role for high protein concentrations after refolding.
However, the \aov test suggests that only arginine addition has a significant influence on refolding, when the significance level is set to \SI{5}{\percent} ($p$-value \num{0.0158}).
The other $p$-values are all higher, which suggests no influence. 
Only the $p$-value for \gls{sah} addition (\num{0.0897}) would show significance, if the significance level was raised to \SI{10}{\percent}.


%\begin{figure}[htpb]
%  \centering
%  \begin{figureframe}
%    
%    \begin{subfigure}{0.8\textwidth}\centering
%      \includegraphics[width=\textwidth]{usr/graphics/static/e_refold_BF}
%      \caption{Protein concentration after rebuffering every refolding reaction.}
%    \end{subfigure}
%        
%    \begin{subfigure}{0.8\textwidth}\centering
%      \includegraphics[width=\textwidth]{usr/graphics/static/MEplot_BF}
%      \caption{Main effects plot.}
%      \label{fig:MEplot}
%    \end{subfigure}
%    
%  \end{figureframe}
%  \caption{}
%  \label{fig:refold_BF}
%\end{figure}

\begin{lstTabularx}[1]{\linewidth}{lrrrrrl}
%
  \caption{Results for the \aov of the main effects model describing soluble protein. Significance codes: 0.05 (\textbf{~*~}), 0.1 (\textbf{~.~})} \\
  \label{tab:aov_BF}
  \small
  \hfill \\
    \toprule
    & df & Sum Sq & Mean Sq & F value & Pr($>$F) & \\ 
    \hline
    Arginine      & 1 & 3595.63 & 3595.63 & 24.56 & 0.0158 & * \\ 
    pH            & 1 & 529.87 & 529.87 & 3.62 & 0.1533 & \\ 
    Glycerin      & 1 & 752.08 & 752.08 & 5.14 & 0.1083 & \\ 
    ionicStrength & 1 & 82.37 & 82.37 & 0.56 & 0.5077 & \\ 
    divCations    & 1 & 5.49 & 5.49 & 0.04 & 0.8588 & \\ 
    redox         & 1 & 5.52 & 5.52 & 0.04 & 0.8584 & \\ 
    CycloDex      & 1 & 134.67 & 134.67 & 0.92 & 0.4083 & \\ 
    SAH           & 1 & 896.83 & 896.83 & 6.13 & 0.0897 & .\\ 
    Residuals     & 3 & 439.26 & 146.42 &  &  & \\ 
    \bottomrule
\end{lstTabularx}


Activity tests were conducted with the refolded protein samples to check for naringenin conversion.
The amount of produced ponciretin was determined relative to the internal standard \glsentryfirst{ac9}.
The protein activity in the refolded samples was generally very low.
The maximum conversion of about \SI{8.7}{\percent} was observed for the refolded protein sample 7.
The activity of the samples did not correlate well with the amount of soluble protein.
This becomes clear from the samples 4 and 5, where the amount of soluble protein was high but the observed activity was at a baseline level.
The \meplot also suggests that the main effects have changed. 
Most notably the redox state of the refolding reaction seemed to have a big influence on the protein activity.
Indeed, the \aov test suggests that using reducing refolding conditions (\gls{dtt}) over a redox-shuffling system (GSH:GSSG, oxidizing) has a significant influence on methylation activity ($p-\mathrm{value} = 0.0210$). 
However, there is the possibility for \somt to form intramolecular disulfide bridges, as the modelled structure suggests (\autoref{fig:x_somtmodel}).
There are also reports, which showed that intermolecular disulfide bridges can contribute to the stability of, mainly archeal, \glspl{mt} and have no influence on the enzymatic activity \cite{Guelorget2010,Jorda2011,Gee2005}.
Nevertheless most \glspl{mt} are only active under reducing conditions and literature suggests, that sometimes assays of \glspl{mt} are explicitly conducted under reducing conditions \cite{Ibdah2003,Zubieta2001a}.

The $p$-value for arginine addition is 0.0649, which also suggests a significant contribution when using a significance level of \SI{10}{\percent}.
This is plausible, since there cannot be any activity when no soluble protein is present.
Judging from the \aov test, the remaining factors have no significant impact on the protein activity after refolding.

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    
    \begin{subfigure}{0.48\textwidth}\centering
      \includegraphics[width=\textwidth]{usr/graphics/static/e_refold_BF}
      \caption{Protein concentration after rebuffering every refolding reaction.}
    \end{subfigure}%       
    \begin{subfigure}{0.48\textwidth}\centering
      \includegraphics[width=\textwidth]{usr/graphics/static/MEplot_BF}
      \caption{\Glspl{meplot} for soluble protein.}
      \label{fig:MEplot_BF}
    \end{subfigure}
    
    \begin{subfigure}{0.48\textwidth}\centering
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_refold_hplc}
      \caption{Relative conversion of naringenin to ponciretin by refolded \somt samples.}
    \end{subfigure}%
    \begin{subfigure}{0.48\textwidth}\centering
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_MEP_activity}
      \caption{\Glspl{meplot} for \somt activity.}
    \end{subfigure}
    
  \end{figureframe}
  \caption{Results of \invitro protein refolding trials.}
  \label{fig:e_refolding_analysis}
\end{figure}


\begin{lstTabularx}[1]{\linewidth}{lrrrrrl}
%
  \caption{Results for the \aov of the main effects model describing protein activity. Significance codes: 0.05 (\textbf{~*~}), 0.1 (\textbf{~.~})} \\
  \label{tab:aov_activity}
  \small
  \hfill \\
    \toprule
    & df & Sum Sq & Mean Sq & F value & Pr($>$F) &   \\ 
    \midrule
    Arginine      & 1 & 0.03 & 0.03 & 8.14 & 0.0649 & .  \\ 
    pH            & 1 & 0.02 & 0.02 & 4.83 & 0.1153 &   \\ 
    Glycerin      & 1 & 0.00 & 0.00 & 0.55 & 0.5122 &   \\ 
    ionicStrength & 1 & 0.01 & 0.01 & 3.27 & 0.1682 &   \\ 
    divCations    & 1 & 0.00 & 0.00 & 0.57 & 0.5047 &   \\ 
    redox         & 1 & 0.06 & 0.06 & 19.88 & 0.0210 & *  \\ 
    CycloDex      & 1 & 0.00 & 0.00 & 0.78 & 0.4428 &   \\ 
    SAH           & 1 & 0.00 & 0.00 & 0.13 & 0.7439 &   \\ 
    Residuals     & 3 & 0.01 & 0.00 &  &  &   \\ 
    \bottomrule
\end{lstTabularx}

\subsubsection{Scaleup of \invitro refolding reactions}

Buffer 7 (\SI{50}{\milli\molar} borate/NaOH, \SI{0.5}{\molar} arginine, \SI{2}{\milli\molar} \cf{CaCl2}, \SI{2}{\milli\molar} \cf{MgCl2}, \SI{10}{\milli\molar} \cf{NaCl}, \SI{0.5}{\milli\molar} \cf{KCl}, \SI{30}{\milli\molar} \textalpha-cyclodextrin, \SI{5}{\milli\molar} \gls{dtt} pH 8.5) was used to scale up the refolding reaction from a total volume of \SI{1.05}{\milli\litre} to a volume of \SI{50}{\milli\litre}.
After refolding and concentration the protein solution was analyzed by gel filtration, \gls{cd} spectrometry and activity tests were conducted.

The retention time for \somt during the gel filtration run was 14.5\,min \bweigel{genau raussuchen} (\autoref{fig:x_somt_gefi}).
From the gel filtration standard measurements, this corresponds to a molecular weight of approximatly \SI{57853}{\dalton}, which is roughly the weight of one \somt monomer (\SI{40}{\kilo\dalton}).
The \gls{cd} spectrum  (\autoref{fig:x_somt_cd}) suggest, that the refolded \somt possesses a secondary structure and is not present as an unfolded random coil.
The secondary structure was estimated from the measured \gls{cd}-spectrum by the K2D3 web service \cite{Louis-Jeune2012}.
According to the calculations, the secondary structure elements consist of \SI{12.39}{\percent} \textalpha-helix and \SI{32.51}{\percent} \textbeta-sheet.
However, the calculated protein model (\ref{fig:x_somtmodel}) suggests the helix content is much higher (\SI{52.3}{\percent}), whereas the sheet content is accordingly lower (\SI{15.4}{\percent}).
This finding suggests, that the refolded protein is not in a native state.
Also, the enzyme showed no activity for naringenin methylation, which was further evidence that the scaled-up refolding was unsuccessful.
Even over the course of many trials a successful large scale refolding of \somt yielding active protein could not be achieved.

These results display that \gls{doe} combined with \gls{ffd} can be a valuable tool for the identification of main factors during protein refolding.
However, there still exists a discrepancy between small scale refolding reactions and the process of upscaling, which might not be trivial.

\section{\glsentrytext{pfomt}}
\subsection{Phenolic hydroxyls}

Phenolic hydroxyl groups have \pka-values of around 10 as demonstrated by four \textit{p}-cresole derivatives (\autoref{tab:pkaphenols}). 
Catecholic systems have two \pka s. The 3-hydroxyl in this example has a much smaller \pka than the 4-hydroxyl. 
This is in part due to the mesomeric (+M) and inductive (+I/-I) properties the substituents display.
The M and I--effects let the 3-OH be deprotonated first, which in turn really lowers the acidity and thus inceases the \pka of the 4-OH.
(4-hydroxy-3-methoxy)- and (3-hydroxy-4-methoxy)-derivatives again have a similar \pka, with the \textit{meta}-position slightly more acidic due to the +I--effect of the methyl substituent.
The nucleophilicity of these phenolic groups happens to coincide with their \textsc{Br\o nsted} acidity.
Chemically speaking the hydroxyl with the lower \pka always reacts first with an electrophile.
      
However, different enzymes are able to regioselectivly methylate the 3- or the 4-OH of such catecholic systems. \bweigel{cite dippe et al 2015}
Enzyme's active sites create a ``microclimate'', which can selectively raise or lower the effective \pka of functional groups and allows for the efficient manipulation of the macroscopically observed regioselectivity.


\begin{table}
    \caption{\pka-values of phenolic hydroxyl groups examplified by \textit{p}-cresole derivatives. Substituent positions on the aromatic ring are arbitrary and do not reflect conventions of the \glsentryfirst{iupac}.} 
  \label{tab:pkaphenols}
  \centering\small
  \begin{tabularx}{\linewidth}{cllll}
    \toprule
    \multirow{5}{*}{%
      \includegraphics{usr/graphics/static/phenols}%
    }%
    & R\textsuperscript{1} & R\textsuperscript{2} & \pka\textsuperscript{-R\textsuperscript{1}} &  \pka\textsuperscript{-R\textsuperscript{2}} \\
    \cmidrule{2-5}
    & OH & H & 10.36 & -- \\
    & OH & OH & 13.1 & 9.55 \\
    & OH & O-Me & 10.34 & -- \\
    & O-Me & OH & -- & 10.08 \\
    \bottomrule
  \end{tabularx}
\end{table}

Previous studies have established that \pfomt is a 3'-\textit{O}-methyl transferase, which is not able to methylate substrates that bear either phenolic (e.g. naringenin), (3'-hydroxy-4-methoxy)- (e.g. hesperetin) or (4'-hydroxy-3-methoxy)-moieties (e.g. homo-eriodictyol) \citep{Ibdah2003}.
The reactions were all run under the same ``standard''  conditions.
However, the reaction buffer/medium can have a tremendous impact on enzymes and their reactions. 
Therefore reaction conditions require optimization, just as the enzymes themselves, to augment an enzymatic process \citep{Bornscheuer2012,Koeller2001}.

Using \pfomt the reaction conditions were screened, to assess if any would promote the methylation of non-catecholic substrates.
Changes in the pH of the medium effect enzymatic activity, especially if charged groups are part of the catalytic mechanism, which was proposed for \pfomt \citep{Brandt2015}.
Furthermore, \pfomt is a magnesium dependent enzyme and the activity is effected by altering the concentration of \cf{Mg^2+}.
Thus, the pH was chosen to be varied along with \cf{Mg^2+} concentration in order to study the influence of those two factors on the methylation reaction.

\subsection{\glsentrytext{pfomt} pH-profiles are influenced by \cf{Mg^2+}}
\label{sec:pfomtph}

\pfomt was dialyzed extensively against buffer containing \SI{5}{\milli\molar} \gls{edta} to obtain enzyme that was virtually free of bound divalent cations.
pH-profiles (pH 5.5 -- 9.5) of three different substrates (caffeic acid, \textit{iso}-ferulic acid, eriodictyol) were obtained.
The pH-profiles were measured without and with the addition of \SI{10}{\milli\molar} \cf{MgCl2}.

The catecholic substrates caffeic acid and eriodictyol were converted by \pfomt much more quickly than \ifa, which is a (3-hydroxy-4-methoxy)-substituted cinnamic acid (\autoref{fig:e_pHprofile}).
The highest rate of \ifa conversion was two orders of magnitude lower than the highest rate for conversion of the other two substrates.
Nonetheless conversion was observed for \ifa with increasing pH and even an influence of magnesium was observed.
Addition of \SI{10}{\milli\molar} \cf{Mg^2+} increased the rate of \ifa conversion by 3-fold from \SI{7}{\pico\katal\per\milli\gram} to \SI{21}{\pico\katal\per\milli\gram} at pH 9.45 (\autoref{tab:specific.activity}).

The specific activities observed for the conversion of caffeic acid are comparable to published data \citep{Dippe2015}.
For the two catecholic substrates, the pH-optimum shifted from neutral to alkaline pHs, when \cf{Mg^2+} was omitted.
However, the maximum activity remained roughly the same.
This was not true for \ifa, which was hardly converted at all without the addition of \cf{Mg^2+}.

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.65\textwidth]{usr/graphics/static/tmp_pHprofile}
  \end{figureframe}
  \caption{Initial rate/pH-profiles for the conversion of three different substrates (red -- caffeic acid, green -- eriodictyol, blue -- \textit{iso}-ferulic acid) by \pfomt. The non-catecholic substrate \textit{iso}-ferulic acid is converted much less quickly than the catecholic substrates. \bweigel{evtl alles schwarz weiÃ mit anderen linien/symbolen?}}
  \label{fig:e_pHprofile}
\end{figure}

\begin{table}[htpb]
  \caption{Maximum specific activity for the conversion of three different substrates with and without addition of magnesium. The pH at which the maximal activity was reached is indicated.}
  \label{tab:specific.activity}
  \centering\small
  \begin{tabularx}{\textwidth}{llrrr}
    \toprule
     substrate & \cf{Mg^2+} & pH & $A_\mathrm{sp} \big[\frac{\mathrm{U}}{\mathrm{mg}}\big]$ & $A_\mathrm{sp} \big[\frac{\mathrm{pkat}}{\mathrm{mg}}\big]$ \\ 
    \midrule
     caffeic acid & FALSE & 9.45 & 88 & 1466 \\ 
     caffeic acid & TRUE & 6.50 & 94 & 1572 \\ 
     eriodictyol & FALSE & 9.45 & 49 & 824 \\ 
     eriodictyol & TRUE & 6.50 & 81 & 1354 \\ 
     \ifa & FALSE & 9.45 & 0.4 & 7 \\ 
     \ifa & TRUE & 9.45 & 1.2 & 21 \\ 
    \bottomrule
  \end{tabularx}
\end{table}


Linear regression models of the form
\begin{equation}
  Y =\beta_0 + \beta_1X_1 + \beta_2X_2 + \cdots + \beta_pX_p + \epsilon
\end{equation}
were fitted to the data in order to make statistically sound inferences regarding a relationship between activity and pH/\cf{Mg^2+} addition.
However, to simplify this task two subsets were prepared first. 
The subsets split the data into substrates with catecholic and substrates without catecholic motifs.
The \ifa data was fit to the model
\begin{equation}\label{eq:ifa}
  activity = \beta_0 + \beta_1 \times \factor{Mg} + \beta_2 \times \factor{pH} + \beta_3 \times (\factor{Mg} \times \factor{pH}),
\end{equation}
which explains about \SI{93.5}{\percent} of the variance (\autoref{tab:lm-ifa}).
The small p-values of the coefficients for \factorx{Mg} and \factorx{pH}$\times$\factorx{Mg} suggest that there is a strong interaction between \cf{Mg^2+} addition and pH.
However, the pH as a main effect has little significance.

The pH profile for the catecholic substrates shows that there might be a quadratic relationship between pH and activity. 
This is true for most enzymatic reactions, where ionizable groups are involved the reaction.
A quadratic term was thus included into the linear model to capture this relationship:
\begin{equation}\label{eq:cat}
    activity = \beta_0 + \beta_1 \times \factor{Mg} + \beta_2 \times \factor{pH} + \beta_3 \times (\factor{Mg} \times \factor{pH}) + \beta_4 \times \factor{pH^2} + \beta_4 \times (\factor{pH^2} \times \factor{Mg}),
\end{equation}
This model describes the actual data reasonable well, with about \SI{68.6}{\percent} of the variance explained (\autoref{fig:e_pHlm}).
Also here the p-values suggest an interaction between \cf{Mg^2+} and pH (\autoref{tab:lm-cat}).
In addition more complex linear models were prepared and shrunken via the Lasso method and 5-fold cross validation (see \autoref{tab:lasso-ifa} and \ref{tab:lasso-catechol}) \citep{Tibshirani1994}.
The Lasso is a shrinkage method that can shrink coefficients to exactly zero and thus make a model more interpretable.
The results of the the shrunken models match well with the results obtained by linear modelling and are further statistical evidence that pH and \cf{Mg^2+} show main effects and also interaction effects which seem to be associated with the activity towards \ifa and catechol (eriodictyol, caffeic acid) methylation by \pfomt.
Nonetheless, all of these rather simple models can not reflect the reality of such complex systems as enzymes where lots of factors play important roles.
The shown results are solely included for purposes of making statistics based inferences in the context of domain knowledge.


\begin{table}[htpb]
  \caption{Coefficients of the model (\ref{eq:ifa}) for activity of \ifa methylation. The factor $\factor{Mg}$ is a categorical variable (addition/no addition) and can therefore only be 0 or 1.}
  \label{tab:lm-ifa}
  \centering\small
   \begin{tabularx}{\textwidth}{lrrrrr}
    \toprule\small
    & Estimate & Std. Error & t value & p-value &   \\ 
    \midrule
    $($Intercept) & -241.4238 & 420.1485 & -0.57 & 0.5864 &   \\ 
    \factorx{pH} & 38.4239 & 54.9778 & 0.70 & 0.5108 &   \\ 
    \factorx{Mg} & -2201.3084 & 594.1797 & -3.70 & 0.0100 & *  \\ 
 \factorx{pH}$\times$\factorx{Mg} & 373.8131 & 77.7503 & 4.81 & 0.0030 &  ** \\ 
    \bottomrule
  \end{tabularx}
\end{table}

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \begin{subfigure}{0.45\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_lm_ifa}
      \caption{\ifa ($R^2=0.9355$)}
    \end{subfigure}
    \begin{subfigure}{0.45\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_lm_catechols}
      \caption{Catecholic substrates ($R^2=0.6855$)}
    \end{subfigure}
  \end{figureframe}
  \caption{pH-profiles of substrate conversion. The linear regression models (blue, dashed lines) grasp the general trend of the data reasonably well to draw inferences. \SI{95}{\percent} prediction intervals are displayed as shaded areas.}
  \label{fig:e_pHlm}
\end{figure}

\subsection{Methylation of different chemical motifs}

The initial results, that showed the conversion of  non-catecholic \ifa by \pfomt promted additional experiments with other different non-catechols from multiple flavonoid subgroups (\autoref{tab:substitutions}, \autoref{fig:substitutions}).
The tested substrates were selected from four different compound groups (cinnamic acids, flavones, flavanones and flavonols) and each group contained each of the structural motifs tested (phenol, catechol, \gls{3o4m}, \gls{4o3m}), if possible.
Each substrate was assessed for conversion with two enzymes (wild-type and 4'-specific variant; Y51R N202W) at four different conditions (pH/\cf{Mg^2+}: low/no, low/yes, high/no, high/yes).
The low and high pH-values were 7.5 and 8.6, respectively.
When \cf{Mg^2+} was added the concentration was \SI{10}{\milli\molar}.
The reactions were incubated at \degC{30} for 16\,h (see section \ref{sec:omtassay}).

\begin{table}[htpb]
  \caption{Substrate grid that was tested for methylation with \pfomt.
    Four different groups of compounds were screened. The groups of flavones, flavanones and cinnamic acids each contained one representative of each motif (phenolic, catecholic, \glsentryfirst{3o4m}, \glsentryfirst{4o3m}. The substitution patterns refer to \autoref{fig:substitutions}.}
  \label{tab:substitutions}
  \begin{tabularx}{\textwidth}{Xrrccc}
    \toprule\small
    substrate & group & motif & R\textsuperscript{1} & R\textsuperscript{2} & R\textsuperscript{3} \\
    \midrule
    naringenin & flavanon & phenolic & H & OH  &  H \\
    eriodictyol & flavanon & catecholic & OH & OH  &  H \\
    hesperetin & flavanon & 3O4M & OH & OMe  &  H \\
    homoeriodictyol & flavanon & 4O3M & OMe & OH  &  H \\
    \midrule
    apigenin & flavone &  phenolic & H & OH  &  H \\
    luteolin & flavone & catecholic & OH & OH  &  H \\
    diosmetin & flavone & 3O4M & OH & OMe  &  H \\
    chrysoeriol & flavone & 4O3M & OMe & OH  &  H \\
    \midrule
    \textit{p}-coumaric acid & cinnamic acid &  phenolic & H & OH  &  H \\
    \textit{m}-coumaric acid & cinnamic acid &  phenolic & OH & H  &  H \\
    \textit{o}-coumaric acid & cinnamic acid &  phenolic & H & H  &  H \\
    caffeic acid & cinnamic acid & catecholic & OH & OH  &  H \\
    \ifa & cinnamic acid & 3O4M & OH & OMe  &  H \\
    ferulic acid & cinnamic acid & 4O3M & OMe & OH  &  H \\
    \midrule
    kaempferol & flavonole  &  phenolic & H & OH  &  H \\
    quercetin & flavonole  & catecholic & OH & OH  &  H \\
    myricetin &  flavonole & catecholic & OH & OH  &  OH \\
    \bottomrule
  \end{tabularx}
\end{table}

\begin{figure}[htpb]
  \begin{figureframe}
    \includegraphics[width=\textwidth]{usr/graphics/static/substrate_pattern}
  \end{figureframe}
  \caption{Substrates from four different groups were screened for methylation by \pfomt under different conditions (pH/\cf{Mg^2+}). Figure refers to \autoref{tab:substitutions}.}
  \label{fig:substitutions}
\end{figure}


There was almost complete conversion of the catecholic substrates under any condition after 16\,h of incubation, at least when the wild-type enzyme was used (\autoref{tab:pfomtconversion}).
This suggests, that the reaction period was chosen too long. 
Effects on the conversion of catecholic substrates will therefore be disregarded or only discussed superficially.
When the wild-type enzyme was employed conversion was observed for all substrates under at least one condition.  
The highest conversion of non-catecholic substrates was observed at high pH and high \cf{Mg^2+} conditions (\autoref{fig:pfomt_conversiopn}).
The trend in the data suggests, that methylation efficiency increases with pH, but especially when magnesium is also added.
Overall, methylation of \gls{3o4m} motifs was highest, with observed conversions of close to \SI{25}{\percent} for the cinnamic acid and flavanone substrates (\ifa and hesperetin).
For these substrates the conversion increases by almost 5-fold upon \cf{Mg^2+} addition, which is close to the observed increase of the initial rate of \ifa methylation (\ref{sec:pfomtph}).
At low pH-values and without addition of \cf{Mg^2+} barely any conversion of the non-catecholic substrates was observed.

The 4'-specific variant for the most part only showed conversion of the catecholic substrates, as expected.
Only some conversion of homoeriodictyol ($\approx$\SI{4}{\percent}) and \ifa ($\approx$\SI{3}{\percent}) was observed under high pH/\cf{Mg^2+} conditions.
However, for the catecholic substrates the same trend -- increasing pH/\cf{Mg^2+} increases activity -- as before holds true.

\begin{table}[htpb]
  \caption{Conversion of substrates after 16\,hours incubation. Only the maximum conversion is displayed, along with the conditions it was achieved under.
    \\ \footnotesize \textsuperscript{\textdagger} --  wild-type: substrate conversion was maximal for all pH/\cf{Mg^2+} combinations. \\ \textsuperscript{\textdaggerdbl} -- conversion of caffeic acid by the wild-type was set to 100\,\%.}
  \label{tab:pfomtconversion}
  \begin{tabularx}{\textwidth}{Xcccccc}
    \toprule\small
    \multirow{2}{*}{substrate} & \multicolumn{3}{c}{wild-type} & \multicolumn{3}{c}{4'-specific variant} \\
    \cmidrule(lr){2-4}\cmidrule(lr){5-7}
    & conversion & pH & \cf{Mg^2+} & conversion & pH & \cf{Mg^2+}  \\
    & \% & & & \% & & \\
    \midrule
    naringenin & 6  & \textglobrise  & \cmark  &   &   &   \\
    eriodictyol\textsuperscript{\textdagger} & 94  &   &   & 92  & \textglobrise  & \cmark  \\
    hesperetin &  22 & \textglobfall  & \cmark  &   &   &   \\
    homoeriodictyol & 6  & \textglobrise  & \cmark  & 4  & \textglobrise  & \cmark  \\
    \midrule
    apigenin & 5  & \textglobrise  & \cmark  &   &   &   \\ 
    luteolin\textsuperscript{\textdagger} &  95  &   &   & 92  & \textglobrise  &   \\ 
    diosmetin & 12  & \textglobrise  & \cmark  &   &   &   \\
    chrysoeriol & 7  & \textglobrise  &  \cmark &   &   &   \\
    \midrule
    \textit{p}-coumaric acid & 6  & \textglobrise  & \cmark  &   &   &   \\
    \textit{m}-coumaric acid & 6  & \textglobrise  & \cmark  &   &   &   \\
    \textit{o}-coumaric acid & 4  & \textglobrise  & \cmark  &   &   &   \\
    caffeic acid\textsuperscript{\textdagger} & 100\textsuperscript{\textdaggerdbl}  &   &   & 93  &  &  \\
    \ifa & 25  & \textglobrise  & \cmark  & 3  & \textglobrise  & \cmark  \\
    ferulic acid & 5  & \textglobrise  & \cmark  &   &   &   \\
    \midrule
    kaempferol & 6  & \textglobrise  & \cmark  & 7  & \textglobrise  & \cmark  \\
    quercetin & 93  & \textglobrise  & \xmark  & 86 & \textglobrise  &   \\
    myricetin &  129 & \textglobfall  & \xmark  & 102  & \textglobfall  & \cmark  \\
    \bottomrule
  \end{tabularx}
\end{table}


\begin{figure}[htpb]
  \begin{figureframe}
    \begin{subfigure}{0.48\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_conversion_substrates_WT}
      \caption{wild-type}
    \end{subfigure}
    \begin{subfigure}{0.48\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_conversion_substrates_variant}
      \caption{4'-specific variant}
    \end{subfigure}
  \end{figureframe}
  \caption{Conversion of multiple different substrates, catecholic and non catecholic, by \pfomt.
 pH/\cf{Mg^2+} color coded from light to dark: \textglobfall/\xmark, \textglobfall/\cmark, \textglobrise/\xmark, \textglobrise/\cmark.}
  \label{fig:pfomt_conversion}
\end{figure}

\section{Structural studies of flavonoids using \glsentryfirst{msms}}
\bweigel{extra kapitel?}

%\section{Consensus or Bioinformatic points-of-view (\glsentrytext{comt})???}

\section{Conclusion/Discussion}
