%%%-------------------------------------------------%%%
%%% Sub document for methylation of non-catechols %%%
%%%-------------------------------------------------%%%

\chapter{Enzymatic methylation of Non-catechols}


\ospSetTitle{Enzymatic methylation of non-catecholic aromatic hydroxyls using class I and class II methyl transferases}
\ospSetAuthors{\underline{Benjamin Weigel}\textsuperscript{1,a}, %
  Martin Dippe\textsuperscript{2,b}, %
  Annegret Laub\textsuperscript{1,b}, %
  Ludger A. Wessjohann\textsuperscript{1,d}}
\ospSetContacts{bweigel@ipb-halle.de\textsuperscript{a}, mdippe@ipb-halle.de\textsuperscript{b}, alaub@ipb-halle.de\textsuperscript{c}, law@ipb-halle.de\textsuperscript{d}}
\ospSetAffiliations{Leibniz-Institute of Plant Biochemistry, Department of Bioorganic Chemistry\textsuperscript{1}}
\ospSetKeywords{methyl transferase, SAM, biocatalysis}

\WEBmakeTitle{%

\textbf{Abstract}

 \Gls{pfomt} and \somt are \gls{sam}-dependent \glspl{mt}, belonging to classes I  (23--27\,kDa, cation-dependent) 
 and II (38--43\,kDa, cation-independent) 
 respectively.
 Methylation of non-catecholic aromatic hydroxyls (phenolic, \gls{3o4m}, \gls{4o3m}) examplified by different compound classes was achieved by both enzymes, although this has never been described for \pfomt.
 Active \somt could not be obtained for \invitro experiments, although soluble enyzme was obtained by optimizing refolding conditions using \gls{ffd} and \gls{doe}.
 The activity of \pfomt towards non-catechols is increased at high pH.
 %\Gls{FrFD} and \gls{doe} can be used to efficiently optimize \invitro refolding conditions.
 Adjusting the pH to more basic conditions can also partly remedy the negative effect of missing \cf{Mg^2+} for class I enzyme \pfomt.
}



\section{Introduction}
Non-catechols in nature (biosynthesis, mode of action?), chemical methylation???

phenol vs methoxyphenols vs catechols 

The 4'-hydroxyl of naringenin is non-catecholic in nature.
However it is much more acidic than the 4'-hydroxyl of eriodictyol (pKa 9.8 vs 12.7) and thus about equally as acidic as the 3'-hydroxyl of eriodictyol (pKa 9.7) \footnote{pKa values were calculated using ChemAxon's MarvinBeans 15.2.16.0} \citep{Schweigert2001a}.

Because \somt was already well characterized in the literature and acted on flavonoids as well as isoflavonoids, it was selected as a model candidate for enzymes that can methylate 4'-hydroxyls of non-catecholic flavonoids.

\section{\glsentrytext{somt}}


\subsection{\textit{In vivo} biotransformation in \textit{Nicotiana benthamiana}}
\bweigel{eventuell weglassen? wo ist der mehrwert fÃ¼r die arbeit???}
The group of Sylvestre Marillonet (IPB) established an efficient system to clone and assemble multi enzyme pathways in \textit{Nicotiana benthamiana}, using a modular cloning toolbox, which has already been used to produce flavonoids \cite{Konig2014}.
All the genes required to establish the pathway up to naringenin in \nico in theory had been cloned previously (\autoref{fig:somtpathway}).
Only the \textit{SOMT2} gene needed to be cloned into a suitable vector to be transiently expressed in \nico.
%The gene was first cloned into a level 0 module using BpiI and consequently subcloned into a level 1 module flanked by a promoter and a terminator using BsaI (\autoref{fig:cloning}).
%The 35S-promoter was used alonside the \gls{nos}-terminator to achieve the highest possible transcription rates. 
%
\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \begin{tikzpicture}[scale=0.75, every node/.style={transform shape}]
      \tikzstyle{cpd}=[rectangle, fill=blue!30]
      \path 	(0,0) node[cpd] (a) {phenyl alanine} 
      (3.5,0) node[cpd] (b) {cinnamic acid}
      (7,0) node[cpd] (c) {coumaric acid}
      (10.5, 0) node[cpd] (d) {coumaroyl CoA}
      (7.5, -0.5) node (da) {3 x malonyl CoA}
      (10.5, -2) node[cpd] (e) {naringenin chalcone}
      (10.5, -4) node[cpd] (f) {naringenin}
      (10.5, -6) node [draw=red, cpd] (g) {4'-O-methyl-naringenin};
		
      \draw[->] (a) -- node[above, gray] {PAL} (b);
      \draw[->] (b) -- node[above] {C4H} (c);
      \draw[->] (c) -- node[above] {4CL} (d);
      \draw[->] (d) -- node[right, gray] {CHS} (e);
      \draw[-] (da.east) .. controls +(right:1cm) and +(up:0.5cm) .. (e.north) ;
      \draw[->] (e) -- node[right] {CHI} (f);
      \draw[->] (f) -- node[right, red] {SOMT2} (g);
    \end{tikzpicture}
    \end{figureframe}
    \caption{Semi-synthetic pathway to naringenin and 4'-O-methyl naringenin in \textit{N. benthamiana}. Enzymes not endogenous to \textit{N. benthamiana} are in gray. PAL - phenylalanine ammonia lyase, C4H - cinnamic acid 4-hydroxylase, 4CL - 4-coumaric acid:CoA ligase, CHS - chalcone synthase, CHI - chalcone isomerase, SOMT2 - soy O-methyl transferase 2}
    \label{fig:somtpathway}
\end{figure}
%
%The \nico plants were infiltrated with transformed \agro (\autoref{sec:agronico}).
Infiltrated plants were harvested after 7 days (see \autoref{sec:agronico}).
The average weight loss after freeze drying was \SI{87.5}{\percent}.

The dried material was extracted and analyzed via \gls{hplc} to determine whether ponciretin or the ``down-stream'' glycosylated products (poncirin, didymine) were produced (\autoref{tab:naringenin}).
However, through comparison with authentic standards it was apparent, that none of the expected compounds were detected.
This finding suggest, that neither naringenin, nor any ``down-stream'' flavonoids (ponciretin, poncirin, didymin) were present in detectable amounts in the plant tissue at the time of harvest.
Although unlikely, it cannot be exluded that higher amounts of the compounds of interest were present at some point in the tissue.
%Addressing the issue of non-detection, however was outside the scope of this work.

\begin{table}[htpb]\centering
  %\ostTableEnv[label=tab:naringenin]{%
 %  \begin{tabularx}{\textwidth}{clll}%
  \caption{Naringenin and 4'-methylated derivatives that were inquired for in the plant samples via \gls{hplc}. The core structure of the compounds is displayed on the left.} 
  \label{tab:naringenin}
  \begin{tabular}{clll}
%  \hfill \\
    \toprule\small
    \multirow{4}{*}{%
      \includegraphics{usr/graphics/static/naringenin}%
    }%
    & R\textsuperscript{1} & R\textsuperscript{2} & name \\
    \cmidrule{2-4}
    & H & H & naringenin \\
    & \cf{CH3} & H & ponciretin \\
    & \cf{CH3} & rutinose \footnote{\textalpha -\textsc{l}-rhamnopyrasonyl-(1\ra6)-\textbeta -\textsc{d}-glucopyranose} & poncirin \\
    & \cf{CH3} & neohesperidose \footnote{\textalpha -\textsc{l}-rhamnopyrasonyl-(1\ra2)-\textbeta -\textsc{d}-glucopyranose} & didymine \\
    \bottomrule
  \end{tabular}
\end{table}
%}{%
%Naringenin derivatives.
%}
The \gls{hplc} chromatograms were analyzed by \gls{pca} after the data were aligned, centered and scaled, to assess whether the collected plant material samples were different from one another (\autoref{fig:e_pca} and \ref{fig:x_pca}).
The \gls{pca}-plot shows that the samples of the different leaf sides do not separate, indicating no difference between infiltration with the \textit{SOMT} gene and vector control between the first two principal components, which account for \SI{80}{\percent} of the variance.
However, there is a slight separation between top and bottom leaves in the second principal component and between plant 3 and plants (1 and 2) in the first principal component (appendix, \autoref{fig:x_pca}).
%
\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.4\textwidth]{usr/graphics/static/tmp_pca2}
  \end{figureframe}
  \caption{Scatterplot of the first two principal components from the \gls{pca} of the \gls{hplc} data obtained from leaf material extracts. The samples are colored by leaf side (left/SOMT-2: red, right/vector control: cyan).}
  \label{fig:e_pca}
\end{figure}
%
This suggest, that the chemical composition as detected by \gls{hplc} is slightly different in the top and botton leaves, as well as between the different plants.


\subsection{\textit{In vivo} biotransformation in \eco}

Kim \etal \citep{Kim2005b,Kim2005c,Kim2013} showed, that \somt could be used for the biotransformation of different flavonoids in \eco live cultures.
%Their technique was to be studied and expanded upon to gain additional insight into the enzyme.
The \textit{\somt} gene was cloned into the pET28a(+) and pET41a(+) vectors, to obtain constructs for the production of \somt without and with a N-terminal \gls{gst}-tag, respectively since both have been used successfully \citep{Kim2005b,Kim2005c,Kim2013}.
However, methylated flavonoids were not detected  when biotransformations were prepared according to the methods of the aforementioned authors (\autoref{fig:invivomethod_a}). 
Thus, the biotransformation medium was changed to \gls{ai}-medium with \SI{0.05}{\percent} glucose \citep{Studier2005}.
%Growth curves were measured, alongside glucose depletion, to establish the time at which substrate addition would be most beneficial.
%Furthermore, \sdspage samples were prepared to observe protein accumulation.
%
\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.4\textwidth]{usr/graphics/static/somt_method}
    \phantomsubcaption\label{fig:invivomethod_a}
    \phantomsubcaption\label{fig:invivomethod_b}
  \end{figureframe}
  \caption{Biotransformation methods as described by Kim \etal (\textbf{A}) and developed in this work (\textbf{B}). OD -- optiocal density at \SI{600}{\nano\meter}, LB -- \glsentrytext{lb}-medium, AI -- \glsentrytext{ai}-medium.}
  \label{fig:e_invivomethod}
\end{figure}
%
Measured growth curves showed, that the glucose present in the medium was depleted after about 5\,hours into growth (appendix, \autoref{fig:e_growthcurve}).
Expression of the \textit{\somt} gene is expected to begin at this time, because the catabolite repression on the \textit{lac} promoter would be relieved and the promoter would be activated by the lactose present in the \gls{ai}-medium.
Thus, \SI{0.1}{\milli\molar} of flavonoid substrate were added at 4\,hours to minimize its influence on growth and possible degradation.
Although \sdspage samples were prepared throughout the course of the experiment, accumulating \somt could not be clearly distinguished from endogenous \eco protein in the \sdspage gels (appendix, \autoref{fig:x_somtgel}).
Nonetheless, methylation of some of the tested substrates was observed over a course of 30\,hours (\autoref{tab:invivosomt}).
Therefore, the sampled medium was extracted using acidified ethyl acetate.
%
{\setlength\tabcolsep{1.75pt}
\begin{table}[!htb]\small\centering
  \caption{\textit{In vivo} biotransformation of different flavonoids, phenylpropanoids and anthraquinones by \somt in \eco. Conversion ratios were calculated for samples taken after 30 hours.
    Multiple substrates containing a 4'-hydroxyl were methylated.
    Calculation of conversion percentages are only rough estimates, because of the nature of crude medium extracts. Products were determined by \gls{lcmsms}.}
  \label{tab:invivosomt}
  \begin{tabular}{llcll}
     \toprule
    substrate & class & 4'-OH & conversion & product \\
    \midrule
    alizarin & anthraquinone & \xmark & \xmark & -- \\
    purpurin & anthraquinone & \xmark & \xmark & -- \\
    apigenin & flavone & \cmark & \cmark ($\geq$\SI{54}{\percent}) & 4'-\textit{O}-methyl apigenin \\  
    chrysin & flavone & \xmark & \xmark & -- \\
    genistein & isoflavone & \cmark & \cmark ($<$\SI{1}{\percent}) & Biochanin A\\%\footnote{4'-\textit{O}-methyl genistein} \\
    galangin & flavonol & \xmark & \xmark & -- \\ 
    kaempferol & flavonol & \cmark & \cmark ($\geq$\SI{6}{\percent}) & kaempferide\\%\footnote{4'-\textit{O}-methyl kaempferol} \\
    naringenin & flavanone & \cmark & \cmark ($\geq$\SI{55}{\percent}) & ponciretin \\
    eriodictyol & flavanone & \cmark & \cmark ($\geq$\SI{40}{\percent}) & hesperetin \\
    homoeriodictyol & flavanone & \cmark & \cmark ($>$\SI{6}{\percent})& 3',4'-(\textit{O,O})-dimethyl eriodictyol\\
    hesperetin & flavanone & \xmark & \xmark & -- \\
    phloretin & chalcone & \cmark & \xmark & -- \\
    resveratrol & stilbene & \cmark & \cmark ($\geq$\SI{86}{\percent})& 4'-\textit{O}-methyl resveratrol \\
    \textit{p}-coumaric acid & cinnamic acid & \cmark & \xmark & -- \\ 
    caffeic acid & cinnamic acid & \cmark & \xmark & -- \\
    reosmin & cinnamic acid\textsuperscript{\textdagger} & \cmark & \xmark & -- \\ 
    \bottomrule
    \multicolumn{5}{l}{\scriptsize \textsuperscript{\textdagger} dihydro cinnamic ketone}
  \end{tabular}
\end{table}
}
%
\Gls{lcms} was employed to determine the site of methylation, since this method is highly sensitive and numerous structural studies on flavonoids using tandem-mass spectrometry experiments have highlighted the feasability of this approach (see \autoref{ch:MSMS}) \citep{Fabre2001,Lee2005}.
\Gls{cid} was used to obtain structural information about the target molecules, since soft ionization techniques (e.g. \gls{esi}) used in \gls{lcms} instruments primarily produce protonated and deprotonated molecular ions, but rarely yield fragments \citep{Sleno2004}.
The \gls{cid} method collides the precursor ions with a neutral target gas while increasing the energy to induce fragmentation.
The produced fragments vary depending on the energy chosen for fragmentation.
Flavonoids follow certain different fragmentation pathways \citep{Fabre2001,Lee2005}. 
The fragmentation of interest in this work, was the one along the C-ring, which produces two fragments (A- and B-ring) (\autoref{fig:e_MSscheme}).
The mass of the A- and B-ring fragments gives strong evidence for the position (ring) at which methylation occured.
Using the \gls{cid} technique, an energy of \SI{30}{\electronvolt} proved sufficient to fragment most flavonoids along the C-ring as is shown here for the methylated naringenin (\autoref{fig:e_LCMS_ponciretin}).
The molecular ion [M+H]\textsuperscript{+} of the methylated naringenin has a \gls{mz} of 287.092.
The fragments helping to derive structural information are \mz{133} and \mz{153}, which can only be explained if the B-ring was methylated (\autoref{fig:e_MSscheme}).
If the A-ring was methylated, the expected fragment ions of A and B-ring would have \gls{mz}-values of 167 and 119 respectively.
%
\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \begin{subfigure}{\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/MS2_naringenin}
      \caption{CID spectrum (\SI{30}{\electronvolt}) of methylated naringenin.}
    \end{subfigure}
    
    \begin{subfigure}{\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/MS2_naringenin_scheme}
      \caption{Scheme displaying the fragmentation of ponciretin.}
      \label{fig:e_MSscheme}
    \end{subfigure}
%    
  \end{figureframe}
  \caption{The masses resulting from the fragmentation into A- and B-ring along the C-ring (dashed line, \textbf{b}) are evidence, that the 4'-hydroxyl on the B-ring is methylated by \somt}
  \label{fig:e_LCMS_ponciretin}
\end{figure}
%
The \gls{lcms} results suggest, that methylation occured exclusively at the 4'-hydroxyl, as there was no conversion detected, when the 4'-hydroxyl was absent (\autoref{tab:invivosomt}).
A free 4'-hydroxyl seems therefore necessary for a substance to be a substrate for \somt, which confirms the previous results \citep{Kim2005b, Kim2005c}. 
Conversion was observed for 4'-hydroxylated (iso)flavonoids and the stilbene resveratrol, however conversion rates of the isoflavone genistein were very low.
No conversion of anthraquinones, cinnamic acid derivates or chalcones was detected, which is also in accordance with previously published data \citep{Kim2005b, Kim2005c}.
\somt acts on phenolic, catecholic as well as (4-hydroxy-3-methoxy-phenyl)-moeities, as is suggested by the assay results that showed methylation of naringenin, eriodictyol and homoeriodictyol respectively.
The methylation of (4-hydroxy-3-methoxy-phenyl)-moiteties and of stilbenes are properties of \somt that have not been described before.

The conversion ratios were assessed, but are beset with large errors due to the nature of \eco rich medium extracts.
The highest conversions were observed for flavanones and flavones (up to $\geq$\SI{55}{\percent}). The tested isoflavones and flavonols showed much lesser conversion ratios (less then \SI{10}{\percent}).
The conversion ratios of apigenin ($\geq$\SI{54}{\percent}) and naringenin ($\geq$\SI{55}{\percent}) are comparable to the ones published previously \citep{Kim2005b, Kim2005c}.
However, genistein only showed minute conversions, which is in disagreement with the data previously published \citep{Kim2005b, Kim2005c}.
Conversion of eriodictyol, homoeriodictyol and kaempferol were not reported before.

The biotransformation of resveratrol to 3,5-dihydroxy-4'-methoxy-stilbene showed a conversion ratio of $\geq$\SI{86}{\percent} in 30 hours.
This is roughly double the conversion which was recently reported for \invivo biotransformations using the specific resveratrol \gls{omt} sbCOM1, which only achieved \SI{42}{\percent} conversion in 36 hours \citep{Kang2014}.


\subsection{\textit{In vitro} studies using recombinantly produced \glsentrytext{somt}}

\textit{In vivo} biotransformations are an important tool for the primary characterization of enzymes.
However, because live organisms are used and lots of variables are unknown, these systems can cause large errors and are not fit to thouroughly characterize an enzyme.
Initially, \somt was to be purified to homogeneity to be later thourougly characterized \invitro, since the recombinant production of \somt in \eco as a fusion protein with an N-terminal \gls{T7tag} was previosuly shown. 
However, the recombinant enzyme had not been characterized \cite{Kim2005c}. 

\subsubsection{Protein production test}
Initial protein production tests were carried out using \textit{\somt} cloned into pET28a(+) with an N-terminal \gls{histag}.
However, \somt was not produced in soluble form (\autoref{fig:e_somtgel_exp}).
Numerous systems were tested for the expression of \textit{\somt}. 
\eco strains used for the trials included BL21(DE3), Rosetta(DE3), Origami(DE3), C41(DE3), C43(DE3), C41(DE3) pLys, C43(DE3) pLys and DH5\textalpha. 
The \textit{\somt} gene was cloned into multiple other vectors, inluding pET20b for periplasmic protein proiduction, pET32 for expression with an \gls{trx} and vectors that carry promoters for induction by rhamnose.
Multiple media, including \gls{tb}, \gls{lb} and autoinductiuon media were used along with different inducers (e.g. lactose, rhamnose, \gls{iptg}) at different temperatures.
Nonetheless \somt could not be produced in a soluble form and expression only resulted in \glspl{ib}.

\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \includegraphics[scale=0.25]{usr/graphics/static/WEB214_SDSPAGE}
  \end{figureframe}
  \caption{\sdspage of pET28a(+) \textit{\somt} expressed in \eco BL21(DE3) in autoinduction medium at different temperatures (shown above). The insoluble fractions show a protein band the same height as the \SI{40}{\kilo\dalton} marker band, which corresponds to the \somt protein (\SI{40425}{\dalton}). M -- protein size marker, S -- soluble fraction, I -- insoluble fraction}
  \label{fig:e_somtgel_exp}
\end{figure}

\subsubsection{\textit{In vitro} protein refolding}

Since the \somt protein could not be obtained in soluble formen, when recombinantly expressed in \eco, the \glspl{ib} were prepared \cite{Rudolph1996} and used for \invitro refolding studies.
For \textit{in vitro} protein refolding, \glspl{ib} are solubilized using denaturants such as \gls{gua} or urea.
The native tertiary structure of the protein is then restored by removal of the denaturant under the ``right'' conditions (e.g. pH, salt, additives, etc.).
However, this is no trivial task since initially the ``right'' conditions have to be found by trial and error.
The refolding process competes with misfolding and aggregation processes and refolding buffers have to be optimized in order to obtain an efficient refolding system with the best possible results \cite{Willis2005,Yamaguchi2014,Rudolph1996}.
Refolding efficiency is best measured via biological activity, but even with adequate assays refolding studies are a time-consuming process.
The number of experiments required to even test only four variables, for example pH, salt, temperature and protein concentration with 3 states each (e.g. low, medium, high) in all possible combinations results in $3^4=81$.
An experimental setup, which accounts for all possible variable (factor) combinations is also called a \emph{full factorial design}.
These setups capture main effects, as well as higher level interaction effects \citep{Montgomery2001,Box2005}.
However, for screening purposes only a fraction of the experiments can be run.
The objective of these \glsentryfirst{ffd} experiments is to identify the variables, which have large effects and are worth expanding the experimental investigation upon.
\Glspl{ffd} have been successfully used for a number of protein refolding trials \citep{Willis2005,Vincentelli2004,Tobbell2002}.

The following factors were studied for the \invitro refolding of \somt: pH, arginine addition, glycerol addition, addition of divalent cations, ionic strength, redox system, cyclodextrin addition and effector (\gls{sah}) addition.
These factors were used, because all have been shown to influence refolding success \cite{Tsumoto2004,Anselment2010,Fujimoto2010,Willis2005,Yamaguchi2014,Chen2008a,Vincentelli2004,Akbari2010,Benoit2007}.
Two factor levels were used in a twelve-run design. This is sufficient to find some main effects, however no statement about interaction effects can be made.
For a complete listing of the buffers and conditions the reader is referred to the materials and methods chapter (Tables \ref{tab:ffd_factors} and \ref{tab:ffd_matrix}).

%The \sdspage gels that were prepared of the soluble and insoluble fraction already show 
Big differences between soluble and insoluble fractions of different refolding buffers could already be noticed from the \sdspage gels (\autoref{fig:e_refold_gel}, see \autoref{subsec:doerefolding}).
Refolding buffers 2,3 and 8--11 mainly produced insoluble protein, whereas the majority of the protein in refolding buffers 1, 4--7 and 12 was in soluble form after an overnight refolding reaction (Tables \ref{tab:ffd_factors} and \ref{tab:ffd_matrix}, \autoref{fig:e_refold_gel}). 
After rebuffering the cleared refolding reactions into a unified buffer the protein concentrations were estimated by \textsc{Bradford}-assays \citep{Bradford1976}.
The protein concentrations measured by \textsc{Bradford}'s method were consistent with the observations from the \sdspage gels (compare \autoref{fig:e_refold_BF} and \ref{fig:e_refold_gel}).
Soluble protein was obtained for buffers 1, 4--7 and 12.
The highest amount of soluble protein was present, when the refolding reaction took place in buffers 5 or 7.
%
\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/refold_sdspage}
  \end{figureframe}
  \caption{\sdspage of the insoluble and soluble fractions of the refolding reactions. Refolding reactions 2,3,8-11 seem to mainly produce misfolded insoluble protein, while the other refolding buffers (1,4-7,12) produce soluble protein.}
  \label{fig:e_refold_gel}
\end{figure}
%
\begin{figure}[htb]
  \centering
  \begin{figureframe}
    
    \begin{subfigure}{0.32\textwidth}\raggedright
      \includegraphics[width=0.98\textwidth]{usr/graphics/static/e_refold_BF}
      \caption{}
      \label{fig:e_refold_BF}
    \end{subfigure}%       
    \begin{subfigure}{0.64\textwidth}\raggedleft
      \includegraphics[width=0.98\textwidth]{usr/graphics/static/MEplot_BF}
      \caption{}
      \label{fig:MEplot_BF}
    \end{subfigure}
    
    \begin{subfigure}{0.32\textwidth}\raggedright
      \includegraphics[width=0.98\textwidth]{usr/graphics/static/tmp_refold_hplc}
      \caption{}
      \label{fig:e_refold_activity}
    \end{subfigure}%
    \begin{subfigure}{0.64\textwidth}\raggedleft
      \includegraphics[width=0.98\textwidth]{usr/graphics/static/tmp_MEP_activity}
      \caption{}
      \label{fig:MEplot_activity}
    \end{subfigure}
    
  \end{figureframe}
  \caption{Results of \invitro protein refolding trials. Measured data (left) is presented alongside the \glspl{meplot} (right).  The dashed line through the \glspl{meplot} illustrates the overall mean.
    \textbf{a} -- Protein concetration after refolding and rebuffering into a universal buffer. 
    The highest yield of soluble protein was achieved in buffers 5 and 7. 
    The \glspl{meplot} (\textbf{b}) illustrate the connection between a factor and the measured protein concentration, suggesting that high pH and arginine concentration might have been beneficial in the refolding reactions. 
    \textbf{c} -- Calculated conversion of naringenin to ponciretin by the refolded protein fractions. 
    Protein refolded in buffers 1 and 7 seem to afforded the most active protein by conversion ($\sim$volume activity). 
    The \glspl{meplot} for the conversion (\textbf{d}) show that the redox state (reducing) of the refolding environment was important to achieve active protein.
    }
  \label{fig:e_refolding_analysis}
\end{figure}
%
The common denominator of those buffers is that all of them contained arginine, whose addition has proven beneficial for many refolding applications \citep{Chen2008a,Tsumoto2004,Fujimoto2010}.

\textbf{\Glsentryfirstplural{meplot}} illustrate the difference between level means for each factor.
Therefore, the mean of the measured property (i.e. protein concetration) for each level of every factor, described by  + or -, of the used refolding buffers is plotted in relation to the overall mean. 
For example, the levels of the factor pH are ``low'' (-) and ``high'' (+). 
When $x^-_i$ is the measured concentration from a refolding reaction in buffer $i$ with ``low'' pH and $x^+_j$ is the measured concentration from a refolding reaction in buffer $j$ with ``high'' pH, then the level means $\overline{x}^{-}$ and $\overline{x}^{+}$ are calculated as follows:
\begin{eqnarray*}
  \overline{x}^- = \frac{1}{n} \cdot \sum_{i=1}^n x^-_i \\
  \overline{x}^+ = \frac{1}{n} \cdot \sum_{j=1}^n x^+_j
\end{eqnarray*}
In the presented experiment ``low'' pH buffers were buffers 2, 4, 6, 8, 10 and 12, whereas ``high'' pH buffers were buffers 1, 3, 5, 7, 9 and 11, thus the level means were calculated as such:
\begin{eqnarray*}
  \overline{x}^+ = \frac{{x}^+_1 + {x}^+_3 + {x}^+_5 + {x}^+_7 + {x}^+_9 + {x}^+_{11}}{6} = 24.63 \\
  \overline{x}^- = \frac{{x}^-_2 + {x}^-_4 + {x}^-_6 + {x}^-_8 + {x}^-_{10} + {x}^-_{12}}{6} = 11.34
\end{eqnarray*}

The \glspl{meplot} for protein concentration suggest, that arginine was likely an important factor for the refolding of \somt (\autoref{fig:MEplot_BF}).
Furthermore, the addition of \gls{sah} or glycerin and the pH seemed to have an influence, whereas the other factors seemed to play only minor roles to achieve high concentrations of soluble protein after refolding.
However, the \aov test, which gave a $p$-value of \num{0.0158} for factor \emph{arginine}, suggests that only arginine addition had a significant influence on refolding, when the significance level is set to \SI{5}{\percent} (appendix, \autoref{tab:aov_BF}).
The other $p$-values are all higher than \num{0.05}, which suggests the other factors had no influence on the yield of soluble protein. 
Only the $p$-value for factor \emph{\gls{sah}} (\num{0.0897}) would suggest significance, if the significance level was raised to \SI{10}{\percent}.

However, soluble protein is not necessarily active. 
Therefore activity tests were conducted with the refolded protein samples to check for naringenin conversion (\autoref{fig:e_refold_activity}).
The conversion of naringenin to ponciretin was calculated from the \gls{auc} of the susbtrate and product peaks as follows:
\begin{equation*}
%  conversion = \frac{\mathrm{AUC_{ponciretin}} / \mathrm{AUC_{AC-9}}}{\mathrm{AUC_{naringenin}}/\mathrm{AUC_{AC-9}}  + \mathrm{AUC_{ponciretin}} / \mathrm{AUC_{AC-9}}}
    conversion = \frac{\mathrm{AUC_{ponciretin}}}{\mathrm{AUC_{naringenin}}  + \mathrm{AUC_{ponciretin}}}
\end {equation*}
Although the substrate naringenin was already contaminated with about \SI{2.5}{\percent} ponciretin, this value was not subtracted from the measured conversions to avoid introduction of unnecessary errors.
The protein activity in the refolded samples was generally very low, as suggested by the low conversions after an overnight activity assay.
The maximum conversion of about \SI{8.7}{\percent} (\SI{6.2}{\percent}) was observed for the protein sample refolded in buffer 7.
The activity of the protein samples did not correlate well with the amount of soluble protein (\autoref{fig:e_refolding_analysis}).
This becomes clear from the samples refolded in buffers 4 and 5, where the amount of soluble protein was high but the observed activity was at a baseline level.

The \meplot suggests that the main effects for obtaining high amounts of soluble protein and obtaining active protein after refolding are different (\autoref{fig:MEplot_activity}). 
Most notably, the redox state of the refolding reaction seemed to have a big influence on the protein sample's activity.
The redox state however had almost no influence on the yield of soluble protein (\autoref{fig:MEplot_BF}).
Indeed, the \aov test suggests that using reducing refolding conditions (\gls{dtt}) over a redox-shuffling system (GSH:GSSG, oxidizing) has a significant influence on methylation activity judged by the $p$-value \num{0.0218} of the factor \emph{redox} (appendix, \autoref{tab:aov_activity}). 
However, there is the possibility for \somt to form intramolecular disulfide bridges, as the modelled structure suggests (\autoref{fig:x_somtmodel}).
There are also reports, which showed that intermolecular disulfide bridges can contribute to the stability of, mainly archeal, \glspl{mt} and have no influence on the enzymatic activity \cite{Guelorget2010,Jorda2011,Gee2005}.
Nevertheless most \glspl{mt} are only active under reducing conditions and literature suggests, that sometimes assays of \glspl{mt} are explicitly conducted under reducing conditions \cite{Ibdah2003,Zubieta2001a}.
Reducing environments reduce the chance of disulfide cross-linked protein mono- and oligomers and allow the enzyme to be more flexible, which might be important for catalysis.
When using a significance level of \SI{10}{\percent}, the $p$-value for \emph{arginine} is 0.0824, which also suggests a significant influence of this factor on the chance to obtain active protein after refolding.
This is plausible, since there cannot be any activity when no soluble protein is present, and refolding reactions without added arginine did not afford any soluble protein.
Judging from the \aov test, the remaining factors like \emph{glycerin}, \emph{ionic strength} or \emph{divalent cations} had no significant impact on the protein activity after refolding.

%\subsubsection{Scaleup of \invitro refolding reactions}
Due to the promising results obtained from the refolding trails, which implicated that the best overall refolding performance (i.e. soluble protein and activity) was achieved in buffer 7 (\SI{50}{\milli\molar} borate/NaOH, \SI{0.5}{\molar} arginine, \SI{2}{\milli\molar} \cf{CaCl2}, \SI{2}{\milli\molar} \cf{MgCl2}, \SI{10}{\milli\molar} \cf{NaCl}, \SI{0.5}{\milli\molar} \cf{KCl}, \SI{30}{\milli\molar} \textalpha-cyclodextrin, \SI{5}{\milli\molar} \gls{dtt} pH 8.5), this buffer was used to scale up the refolding reaction from a total volume of \SI{1.05}{\milli\litre} to a volume of \SI{50}{\milli\litre}.
After concentration of soluble protein from the scaled-up refolding reaction, activity tests were conducted.
Unfortunately, the refolded \somt showed no activity for naringenin methylation, which was evidence that the scaled-up refolding was unsuccessful.
Nonetheless, gel filtration chromatography and \gls{cd} spectroscopy were used as a tool to study the three-dimensional structure of the refolded \somt.
The retention volume for the major peak eluted during the gel filtration run was 12.47\,ml, but in the chromatogram a shoulder at \SI{14.26}{\milli\litre} was clearly distinguishable (\autoref{fig:x_somt_gefi}).
Molecular masses of the proteins corresponding to the peaks were estimated from commercial gel filtration protein standards.
The first peak corresponds to a molecular weight of approximately \SI{165}{\kilo\dalton}, whereas the shoulder at \SI{14.26}{\milli\litre} corresponds to a globular protein of approximately \SI{65.5}{\kilo\dalton}.
\SI{65.5}{\kilo\dalton} is roughly the weight of one \somt monomer (\SI{40}{\kilo\dalton}) or a dimer, whereas a mass of \SI{165}{\kilo\dalton} would indicate a tetramer.
These results were further indication, that the majority of the refolded protein was not in the expected native dimeric state.
However, the refolded \somt had adopted some kind of fold that allowed for it to be in soluble form.
%
%% CD spectrum of SOMT
\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/CD_SOMT}
  \end{figureframe}
  \caption{CD-spectrum of refolded \somt (black) compared to the spectrum that was calculated (red) by the K2D3 web service (\url{http://cbdm-01.zdv.uni-mainz.de/~andrade/k2d3//index.html}) from the \somt sequence. Secondary structure estimates from the measured spectrum are \SI{12.39}{\percent} \textalpha-helix and \SI{32.51}{\percent} \textbeta-sheet.}
  \label{fig:x_somt_cd}
\end{figure}
%
Furthermore, the \gls{cd} spectrum (\autoref{fig:x_somt_cd}) suggested, that the refolded \somt possessed a secondary structure and was not present as an unfolded random coil.
The secondary structure was estimated from the measured \gls{cd}-spectrum by the K2D3 web service \cite{Louis-Jeune2012}.
According to the K2D3 calculations, the secondary structure elements consisted of \SI{12.39}{\percent} \textalpha-helix and \SI{32.51}{\percent} \textbeta-sheet.
However, the calculated protein model (\autoref{fig:x_somtmodel}) suggests the \textalpha-helix content might be much higher (\SI{52.3}{\percent}), whereas the \textbeta-sheet content might accordingly be lower (\SI{15.4}{\percent}).
These findings further indicate, that the refolded protein was not in a native state, which might be the cause of the lack of enzymatic activity.
Even over the course of many trials a successful large scale refolding of \somt yielding active protein could not be achieved.

These results display that \gls{doe} combined with \gls{ffd} can be a valuable tool for the identification of main factors during protein refolding.
However, there still exists a discrepancy between small scale refolding reactions and the process of upscaling, which might not be trivial.

\section{\glsentrytext{pfomt}}
\subsection{Phenolic hydroxyls}

Phenolic hydroxyl groups have \pka-values of around 10 as demonstrated by four \textit{p}-cresole derivatives (\autoref{tab:pkaphenols}). 
Catecholic systems have two \pka-values, one for each hydroxyl group. 
The 3-hydroxyl (R\tup{2}) of the displayed example has a much smaller \pka than the 4-hydroxyl (R\tup{1}). 
This is in part due to the mesomeric (+M) and inductive (+I/-I) properties the substituents exhibit.
The M and I--effects let the 3-OH be deprotonated first, which in turn significantly lowers the acidity and thus increases the \pka of the 4-OH.
(4-hydroxy-3-methoxy)- and (3-hydroxy-4-methoxy)-derivatives have a similar \pka, with the \textit{meta}-position slightly more acidic due to the +I--effect of the methyl substituent.
The nucleophilicity of these phenolic groups happens to coincide with their \textsc{Br\o nsted} acidity.
Chemically speaking the hydroxyl group with the lower \pka always reacts first with an electrophile.
      
However, different enzymes are able to regioselectivly methylate the 3- or the 4-OH of such catecholic systems. \bweigel{cite dippe et al 2015}
Enzyme's active sites create a ``microclimate'', which can selectively raise or lower the effective \pka of functional groups and allows for the efficient manipulation of the macroscopically observed regioselectivity.


\begin{table}[htpb]\centering
    \caption{\pka-values of phenolic hydroxyl groups examplified by \textit{p}-cresole derivatives. Substituent positions on the aromatic ring are arbitrary and do not reflect conventions of the \glsentryfirst{iupac}.} 
  \label{tab:pkaphenols}
  \centering\small
  \begin{tabular}{cllll}
    \toprule
    \multirow{5}{*}{%
      \includegraphics{usr/graphics/static/phenols}%
    }%
    & R\textsuperscript{1} & R\textsuperscript{2} & \pka\textsuperscript{-R\textsuperscript{1}} &  \pka\textsuperscript{-R\textsuperscript{2}} \\
    \cmidrule{2-5}
    & OH & H & 10.36 & -- \\
    & OH & OH & 13.1 & 9.55 \\
    & OH & O-Me & 10.34 & -- \\
    & O-Me & OH & -- & 10.08 \\
    \bottomrule
  \end{tabular}
\end{table}

Previous studies have established that \pfomt is a 3'-\textit{O}-methyl transferase, which is not able to methylate substrates that bear either phenolic (e.g. naringenin), (3'-hydroxy-4-methoxy)- (e.g. hesperetin) or (4'-hydroxy-3-methoxy)-moieties (e.g. homoeriodictyol) \citep{Ibdah2003}.
In these previous studies, the reactions were all run under the same ``standard''  conditions.
However, the reaction buffer can have a tremendous impact on enzymes and their reactions. 
Therefore reaction conditions require optimization, just as the enzymes themselves, to augment an enzymatic process \citep{Bornscheuer2012,Koeller2001}.

Using \pfomt reaction conditions were screened, to assess if any would promote the methylation of non-catecholic substrates.
Although enzymes create a specific environment for catalysis, changes in the pH of the medium can still affect said environment and therefore enzymatic activity, especially if charged groups are part of the catalytic mechanism.
In the catalytic mechanism of \pfomt a catalytic triad of Lys-Asn-Asp, two of which are charged, is proposed to play a major role \citep{Brandt2015}.
Furthermore, \pfomt is a magnesium dependent enzyme and the activity is affected by altering the concentration of \cf{Mg^2+} or substitution of \cf{Mg^2+} by other divalent cations \cite{Ibdah2003}.
Thus, the pH was chosen to be varied along with \cf{Mg^2+} concentration in order to study the influence of those two factors on the methylation reaction.

\subsection{\glsentrytext{pfomt} pH-profiles are influenced by \cf{Mg^2+}}
\label{sec:pfomtph}

\pfomt was dialyzed against \SI{50}{\milli\molar} \gls{ssg}-buffer pH 7.5 containing \SI{5}{\milli\molar} \gls{edta} and again against the same buffer with the \gls{edta} omitted, to obtain enzyme that was virtually free of bound divalent cations.
pH-profiles (pH 5.5 -- 9.5) of three different substrates (caffeic acid, \textit{iso}-ferulic acid, eriodictyol) were obtained in the same unified buffer system (\glsentrydesc{ssg}, 2:7:7 molar ratio).
The pH-profiles were measured without and with the addition of \SI{10}{\milli\molar} \cf{MgCl2}.
Maximum methylation activity ($\approx$\SI{1500}{\pico\katal\per\milli\gram}) towards the catecholic substrates (i.e. caffeic acid, eriodictyol) was observed when magnesium was added and the pH was about 6.5  (\autoref{fig:e_pHprofile} and \autoref{tab:specific.activity}).
However, the observed maximum activity shifted towards basic pH values (pH 9.45), when magnesium was omitted from the reaction.
The maximum activity for methylation of \ifa was measured at pH 9.45, regardless of whether magnesium was added.
%
\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.65\textwidth]{usr/graphics/static/tmp_pHprofile}
  \end{figureframe}
  \caption{Specific activity/pH-profiles for the conversion of three different substrates (\textfilledpoint~~caffeic acid, \filledtriangleup~~eriodictyol, \textfilledsquare~~\textit{iso}-ferulic acid) by \pfomt. The specific activity for the non-catecholic substrate \textit{iso}-ferulic acid was much lower than the specific activity for the catecholic substrates.
  When magnesium is omitted, the activity is increased by increasing the pH}
  \label{fig:e_pHprofile}
\end{figure}
%
The catecholic substrates caffeic acid and eriodictyol were converted by \pfomt much more quickly than \ifa, which is a (3-hydroxy-4-methoxy)-substituted cinnamic acid (\autoref{fig:e_pHprofile}).

The highest specific activity $A_\mathrm{sp}$ for \ifa conversion was two orders of magnitude lower than the highest activity for the conversion of the other two substrates.
Nonetheless, conversion was observed for \ifa with increasing pH and even an influence of magnesium was observed (\autoref{fig:e_pHprofile} and \autoref{tab:specific.activity}).
\textit{Iso}-ferulic acid was converted rather slowly without the addition of \cf{Mg^2+} ($A_\mathrm{sp}=$\SI{7}{\pico\katal\per\milli\gram}).
However, addition of \SI{10}{\milli\molar} \cf{Mg^2+} increased the rate of \ifa conversion by 3-fold, from \SI{7}{\pico\katal\per\milli\gram} to \SI{21}{\pico\katal\per\milli\gram} at pH 9.45 (\autoref{tab:specific.activity}). 

The specific activities observed for the conversion of caffeic acid are comparable to published data \citep{Dippe2015}.
For the two catecholic substrates, the pH-optimum shifted from neutral to alkaline pHs, when \cf{Mg^2+} was omitted.
However, the maximum activity remained roughly the same, even though magnesium addition seemed to have a slight rate increasing effect.
Rate enhancements of up to 3-fold were observed at pH 9.45, when \cf{Mg^2+} was omitted compared to when it was present.
The maximum activities without magnesium were observed at pH 9.45 with \SI{1466}{\pico\katal\per\milli\gram} and \SI{824}{\pico\katal\per\milli\gram} for caffeic acid and eriodictyol respectively, while with \SI{10}{\milli\molar} \cf{Mg^2+} the maximum activities were recorded at pH 6.5 and increased to \SI{1572}{\pico\katal\per\milli\gram} and \SI{1354}{\pico\katal\per\milli\gram} respectively.
%
\begin{table}[htb]\centering
  \caption{Maximum specific activity for the conversion of three different substrates with and without addition of magnesium. The pH at which the maximal activity was reached is indicated by the column titled ``pH''.}
  \label{tab:specific.activity}
  \centering\small
  \begin{tabular}{llrrr}
    \toprule
     substrate & \cf{Mg^2+} & pH & $A_\mathrm{sp} \big[\frac{\mathrm{mU}}{\mathrm{mg}}\big]$ & $A_\mathrm{sp} \big[\frac{\mathrm{pkat}}{\mathrm{mg}}\big]$ \\ 
    \midrule
     caffeic acid & FALSE & 9.45 & 88 & 1466 \\ 
     caffeic acid & TRUE & 6.50 & 94 & 1572 \\ 
     eriodictyol & FALSE & 9.45 & 49 & 824 \\ 
     eriodictyol & TRUE & 6.50 & 81 & 1354 \\ 
     \ifa & FALSE & 9.45 & 0.4 & 7 \\ 
     \ifa & TRUE & 9.45 & 1.2 & 21 \\ 
    \bottomrule
  \end{tabular}
\end{table}
%

During the catalysis of a methyl transferase reaction, the acceptor moiety of the substrate is activated by abstraction of a proton.
%The increasing pH might directly abstract a proton 
At high pH-values, the substrates might already be deprotonated, thus increasing the rates of reaction, while the enzyme just acts as a scaffold.
High pH-values would also mean that the mainly the 4'-hydroxyl of the catechols would be deprotonated (\autoref{tab:pkaphenols}).
However, an increasing amount of 4'-methylation was not observed contradicting the notion, that an already deprotonated substrate entered the active site.
It is more likely, that the external milieu influences the enzymes active site and  makes the active side chains more basic.
However, why the addition of magnesium would shift the pH-optimum back to neutral pH-values is not clear.

To statistically support the presented findings, the collected data were also studied from a statisticians point of view, which is described in the following paragraph in a bit more detail.
The shown results are solely included for purposes of making statistics based inferences in the context of domain knowledge.
Relationships between independent variables $X_{1,\cdots,p}$ influencing a system and the outcome $Y$ of such a system can be mathematically described.
The simplest relationship between one independent variable $X_1$ and the dependent variable $Y$ is a linear one and is defined mathematically by a linear equation $Y = \beta_1X_1 + \beta_0$.
The coefficient $\beta_1$ describes how much $Y$ is altered by one unit $X$, while $\beta_0$ is the offset. 
Even seemingly non-linear outcomes of $Y$ might be sufficiently described by multi-term linear equations such as
\begin{equation*}
  Y =\beta_0 + \beta_1X_1 + \beta_2X_2 + \cdots + \beta_pX_p + \epsilon,
\end{equation*}%
when the independent variables $X_{1,\cdots,p}$ are known.
Linear regression models of this form can be used to make statistically sound inferences about a studied system and were thus used to assess the relationship between PFOMT's methylation activity and pH-modulation as well as \cf{Mg^2+} addition.
Two subsets of the activity data were prepared first.
The subsets split the data into substrates with catecholic (i.e. caffeic acid, eriodictyol) and substrates without catecholic (i.e. \ifa) motifs.
This was done to simplify the interpretability of the results, since the activities of the catecholic and non-catecholic  substrates differed greatly.
The \ifa data was fit to the linear model
\begin{equation}\label{eq:ifa}
  activity = \beta_0 + \beta_1 \times \factor{Mg} + \beta_2 \times \factor{pH} + \beta_3 \times (\factor{Mg} \times \factor{pH}),
\end{equation}
which contains one term, $\beta_3\times$\factorx{pH}$\times$\factorx{Mg}, to account for an interaction effect between magnesium and pH besides the main effects terms.
This model explains about \SI{93.6}{\percent} of the variance ($R^2=0.9355$) of the measured data (\autoref{tab:lm-ifa}).
Fitting the data in the \verb+R+ software using the \emph{stats::lm()} function also calculates $p$-values associated with each parameter estimate.
%
\begin{table}[htb]\centering
  \caption{Coefficients of the model (\autoref{eq:ifa}) for activity of \ifa methylation. The factor $\factor{Mg}$ is a categorical variable (addition/no addition) and can therefore only be 0 or 1.}
  \label{tab:lm-ifa}
  \centering\small
   \begin{tabular}{lrrrrr}
    \toprule
    & Estimate & Std. Error & t value & p-value &   \\ 
    \midrule
    $($Intercept) & -241.4238 & 420.1485 & -0.57 & 0.5864 &   \\ 
    \factorx{pH} & 38.4239 & 54.9778 & 0.70 & 0.5108 &   \\ 
    \factorx{Mg} & -2201.3084 & 594.1797 & -3.70 & 0.0100 & *  \\ 
 \factorx{pH}$\times$\factorx{Mg} & 373.8131 & 77.7503 & 4.81 & 0.0030 &  ** \\ 
    \bottomrule
    \multicolumn{6}{l}{\footnotesize significance codes: '*' \SI{5}{\percent} level; '**' \SI{1}{\percent} level}
  \end{tabular}
\end{table}
%
The smallest $p$-value 0.0030 was calculated for the interaction factor \factorx{pH}$\times$\factorx{Mg}, and strongly suggests that there is a significant interaction between methylation activity on \ifa and \cf{Mg^2+} addition combined with pH-modulation.
The parameter estimate $\beta_3$ for this term is almost 374.
Since in this case \factorx{Mg} is categorical and can only be 0 (no magnesium added) or 1 (magnesium added) the interaction term $\beta_3\times$\factorx{pH}$\times$\factorx{Mg} resolves to 0, if no magnesium was added.
This means that when \cf{Mg^2+} is added, the activity (in $\mathrm{AU\,min^{-1}}$) is increased by 374 for each unit the pH is raised.
The $p$-value for \factorx{pH} as a main effect is rather high, suggesting pH alone has no significant impact on the activity.
The parameter estimate $\beta_1$ for factor \factorx{Mg} however has a low $p$-value of 0.010, which suggest that its impact on activity is significant. 
Due to the categorical nature of \factorx{Mg}, this means, that the activity is decreased by \num{2201} $\mathrm{AU\,min^{-1}}$ when magnesium is added.
However, this information is only of importance when predicting the activity outside of the measured pH range, which was not the aim here.
Together, these results illustrate, that neither magnesium addition, nor the pH alone would have such a strong effect on the activity as both factors combined had.

The data of the second subset containing of the catecholic substrates was also modelled in a similar fashion (see \autoref{sec:apppfomtph}).
It was found, that the pH has a much stronger influence on the methylation activity of PFOMT towards catechols than \ifa.
Interaction between magnesium addition and pH-modulation were also suggested for the catecholic substrates from this data.
For a more thourough discussion see \autoref{sec:apppfomtph}.
In addition to pure inference based modelling, models can also be used to make predictions based on (new) independent variables (\autoref{fig:e_pHlm}).
However, it cannot be stressed enough that models do not reflect the truth, but are rather another tool to gain insight into a system.
%
\begin{figure}[htb]
  \centering
  \begin{figureframe}
    \begin{subfigure}{0.45\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_lm_ifa}
      \caption{\ifa ($R^2=0.9355$)}
    \end{subfigure}
    \begin{subfigure}{0.45\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_lm_catechols}
      \caption{Catecholic substrates ($R^2=0.6855$)}
    \end{subfigure}
  \end{figureframe}
  \caption{pH-profiles of substrate conversion along with predicted data. Predicted data from the linear regression models (blue, dashed lines) grasp the general trend of the data reasonably well to draw inferences. \SI{95}{\percent} prediction intervals are displayed as shaded areas.}
  \label{fig:e_pHlm}
\end{figure}
%



To the knowledge of the author, this is the first time the effects of \cf{Mg^2+} and pH on methyl transferase activity were systematically analyzed.
It was shown, that catecholic and non-catecholic substrates could be activated sufficiently by \pfomt at high pHs without the addition of \cf{Mg^2+}. 
It is improbable that, if the active site retained the same miromilieu under every reaction condition, an influence on the rate of reaction would be observed.
This could be a hint, that the enzyme relays the chemical information of the environment directly to the substrate to aid in activation.
Furthermore, omission of \cf{Mg^2+} shifts the pH-optimum of the reaction catalyzed by \pfomt to higher pH-values.
%Addition of \cf{Mg^2+}, seems to remedy the effect of high pHs on catalysis at least for caffeic acid, since the initial rates observed were not as high.
It would be of interest to analyze this behavior with further systematic studies and multiple levels of \cf{Mg^2+} concentrations.


\subsection{Methylation of different chemical motifs}

The previous section showed the conversion of  non-catecholic \ifa by \pfomt and promted additional experiments with other different non-catechols from multiple flavonoid subgroups (\autoref{tab:substitutions}, \autoref{fig:substitutions}).
The tested substrates were selected from four different compound groups (cinnamic acids, flavones, flavanones and flavonols) and each group contained each of four structural motifs (phenol, catechol, \glsentryfirst{3o4m}, \glsentryfirst{4o3m}), if possible.
Each substrate was assessed for conversion with two enzymes (\pfomt wild-type and 4'-specific variant \pfomt Y51R N202W) at four different conditions.
Magnesium addition and pH for the different conditions (pH/\cf{Mg^2+}: low/no, low/yes, high/no, high/yes).
The ``low'' and ``high'' pH-values were 7.5 and 8.6, respectively.
When \cf{Mg^2+} was added the concentration was \SI{10}{\milli\molar}.
The reactions were incubated at \degC{30} for 16\,h (see section \ref{sec:omtassay}).
%
\begin{figure}[htb]
  \begin{figureframe}
    \includegraphics[width=\textwidth]{usr/graphics/static/substrate_pattern}
  \end{figureframe}
  \caption{Substrates from four different phenylpropanoid groups were screened for methylation by \pfomt wild-type and variant under different conditions (pH and \cf{Mg^2+}). Figure refers to \autoref{tab:substitutions}.}
  \label{fig:substitutions}
\end{figure}
%
There was almost complete conversion of the catecholic substrates (eriodictyol, luteolin, caffeic acid and quercetin) after 16\,h of incubation regardless the reaction conditions, at least when the wild-type enzyme was used (\autoref{fig:pfomt_conversion} and \autoref{tab:pfomtconversion}).
This suggests, that the reaction period was chosen too long. 
Effects on the conversion of catecholic substrates will therefore be disregarded or only discussed superficially.
When the wild-type enzyme was employed conversion was observed for all substrates under at least one condition.  
The highest conversion of non-catecholic substrates was observed at high pH and high \cf{Mg^2+} conditions (\autoref{fig:pfomt_conversion}).
The trend in the data suggests, that methylation efficiency increases with pH, but especially when magnesium is also added.
%
\begin{table}[htb]\centering\small
  \caption{Substrate grid that was tested for methylation with \pfomt.
    Four different groups of compounds were screened. The groups of flavones, flavanones and cinnamic acids each contained one representative of each motif (phenolic, catecholic, \glsentryfirst{3o4m}, \glsentryfirst{4o3m}. The substitution patterns refer to \autoref{fig:substitutions}.}
  \label{tab:substitutions}
  \begin{tabular}{lrrccc}
    \toprule
    substrate & group & motif & R\textsuperscript{1} & R\textsuperscript{2} & R\textsuperscript{3} \\
    \midrule
    naringenin & flavanon & phenolic & H & OH  &  H \\
    eriodictyol & flavanon & catecholic & OH & OH  &  H \\
    hesperetin & flavanon & 3O4M & OH & OMe  &  H \\
    homoeriodictyol & flavanon & 4O3M & OMe & OH  &  H \\
    \midrule
    apigenin & flavone &  phenolic & H & OH  &  H \\
    luteolin & flavone & catecholic & OH & OH  &  H \\
    diosmetin & flavone & 3O4M & OH & OMe  &  H \\
    chrysoeriol & flavone & 4O3M & OMe & OH  &  H \\
    \midrule
    \textit{p}-coumaric acid & cinnamic acid &  phenolic & H & OH  &  H \\
    \textit{m}-coumaric acid & cinnamic acid &  phenolic & OH & H  &  H \\
    \textit{o}-coumaric acid & cinnamic acid &  phenolic & H & H  &  H \\
    caffeic acid & cinnamic acid & catecholic & OH & OH  &  H \\
    \ifa & cinnamic acid & 3O4M & OH & OMe  &  H \\
    ferulic acid & cinnamic acid & 4O3M & OMe & OH  &  H \\
    \midrule
    kaempferol & flavonole  &  phenolic & H & OH  &  H \\
    quercetin & flavonole  & catecholic & OH & OH  &  H \\
    myricetin &  flavonole & 3',4',5'-trihydroxy & OH & OH  &  OH \\
    \bottomrule
  \end{tabular}
\end{table}
%
The \gls{meplot} and \gls{iaplot} suggest the same, however there is no statistical evidence to support this notion (see \autoref{fig:x_mep_pfomt}, \ref{fig:x_iap_pfomt} \bweigel{evtÃ¶ hier noch ne tabelle von modelcoefficiebnts?}).
Although the statistical tests performed on this data do not back the hypothesis, this does not rule out an influence of pH or \cf{Mg^2+}.
It should be kept in mind, that enzymatic systems are quite complex and simple linear regression models that allow for inferences often cannot describe what is going on accurately and therefore fail.
Furthermore this very complex dataset is limited to only about 150 observations.
%
\begin{figure}[htb]
  \begin{figureframe}
    \begin{subfigure}{0.48\textwidth}
    %  \includegraphics[width=\textwidth]{usr/graphics/static/tmp_conversion_substrates_WT}
       \includegraphics[width=\textwidth]{usr/graphics/static/conversion_substrates_WT}
      \caption{wild-type}
    \end{subfigure}
    \begin{subfigure}{0.48\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_conversion_substrates_variant}
      \caption{4'-specific variant}
    \end{subfigure}
  \end{figureframe}
  \caption{Conversion of multiple different substrates, catecholic and non catecholic, by \pfomt.
 pH/\cf{Mg^2+} color coded from light to dark: \textglobfall/\xmark, \textglobfall/\cmark, \textglobrise/\xmark, \textglobrise/\cmark.}
  \label{fig:pfomt_conversion}
\end{figure}
%
Overall, methylation of \gls{3o4m} motifs was highest, with observed conversions of close to \SI{25}{\percent} for the cinnamic acid and flavanone substrates (\ifa and hesperetin).
For these substrates the conversion increased by almost 5-fold upon \cf{Mg^2+} addition, which is close to the observed increase of the initial rate of \ifa methylation (\ref{sec:pfomtph}).
Similar results have been shown for SaOMT5, an \gls{omt} from \textit{Streptomyces avermitilis} \citep{Youngdae2010}.
Conversion of the somewhat more rigid flavone diosmetin was lower by at least factor two compared to hespreretin and \ifa.
At low pH-values and without addition of \cf{Mg^2+} barely any conversion of the non-catecholic substrates was observed.
The fact, that conversion of \gls{3o4m}-moiety bearing substrates is greater than that of the 4'-phenolic and \gls{4o3m} substrates could be due to the fact that the wild-type of \pfomt by and large acts upon 3'-hydroxyls.

The 4'-specific variant for the most part only showed conversion of the catecholic substrates, as expected.
Only some conversion of homoeriodictyol ($\approx$\SI{4}{\percent}) and \ifa ($\approx$\SI{3}{\percent}) was observed under high pH/\cf{Mg^2+} conditions.
However, for the catecholic substrates the same trend -- increasing pH/\cf{Mg^2+} increases activity -- as before holds true.
Control experiments without enzyme at high pH and \SI{10}{\milli\molar} revealed, that no substrate conversion took place under these conditions.

The site of methylation was studied by LC-\gls{msms}. 
As expected, methylation only took place on the B-ring of the flavonoids.
For a throughout discussion refer to \autoref{ch:MSMS}.
%
\begin{table}[htb]\centering\small
  \caption{Conversion of substrates after 16\,hours incubation. Only the maximum conversion is displayed, along with the conditions it was achieved under.
     }
  \label{tab:pfomtconversion}
  \begin{tabular}{lcccccc}
    \toprule
    \multirow{2}{*}{substrate} & \multicolumn{3}{>{\centering}p{0.3\textwidth}}{wild-type} & \multicolumn{3}{>{\centering}p{0.3\textwidth}}{4'-specific variant} \\
    \cmidrule(lr){2-4}\cmidrule(lr){5-7}
    & conversion & pH & \cf{Mg^2+} & conversion & pH & \cf{Mg^2+}  \\
    & \% & & & \% & & \\
    \midrule
    naringenin & 6  & \textglobrise  & \cmark  &   &   &   \\
    eriodictyol\textsuperscript{\textdagger} & 94  &  \multicolumn{2}{c}{(all)}  & 92  & \textglobrise  & \cmark  \\
    hesperetin &  22 & \textglobfall  & \cmark  &   &   &   \\
    homoeriodictyol & 6  & \textglobrise  & \cmark  & 4  & \textglobrise  & \cmark  \\
    \midrule
    apigenin & 5  & \textglobrise  & \cmark  &   &   &   \\ 
    luteolin\textsuperscript{\textdagger} &  95  & \multicolumn{2}{c}{(all)}  & 92  & \textglobrise  &   \\ 
    diosmetin & 12  & \textglobrise  & \cmark  &   &   &   \\
    chrysoeriol & 7  & \textglobrise  &  \cmark &   &   &   \\
    \midrule
    \textit{p}-coumaric acid & 6  & \textglobrise  & \cmark  &   &   &   \\
    \textit{m}-coumaric acid & 6  & \textglobrise  & \cmark  &   &   &   \\
    \textit{o}-coumaric acid & 4  & \textglobrise  & \cmark  &   &   &   \\
    caffeic acid\textsuperscript{\textdagger} & 100\textsuperscript{\textdaggerdbl}  & \multicolumn{2}{c}{(all)} & 93  &  \multicolumn{2}{c}{(all)} \\
    \ifa & 25  & \textglobrise  & \cmark  & 3  & \textglobrise  & \cmark  \\
    ferulic acid & 5  & \textglobrise  & \cmark  &   &   &   \\
    \midrule
    kaempferol & 6  & \textglobrise  & \cmark  & 7  & \textglobrise  & \cmark  \\
    quercetin & 93  & \textglobrise  & \xmark  & 86 & \textglobrise  &   \\
    myricetin &  129 & \textglobfall  & \xmark  & 102  & \textglobfall  & \cmark  \\
    \bottomrule
    \multicolumn{7}{p{0.9\textwidth}}{\footnotesize \textsuperscript{\textdagger}wild-type: substrate conversion was maximal for all pH/\cf{Mg^2+} combinations. \newline \textsuperscript{\textdaggerdbl}conversion of caffeic acid by the wild-type was set to 100\,\%.}
  \end{tabular}
\end{table}


To the authors knowledge, this is the first time that methylation of a diverse set of non-catecholic substrates was described for a class I magnesium-dependent methyl transferase. 
A flavonoid-specific \gls{omt} from \textit{Catharanthus roseus} was described to methylate the 4'-position, when the substrates B-ring possessed a \gls{4o3m} substitution \citep{Schroder2004}. 
However, said enzyme only showed marginal activities towards catechols and \gls{3o4m} derivatives.
A class II \gls{omt} from wheat, named TaOMT2, is able to sequentially methylate the three hydroxyl-groups on the B-ring of tricetin, in the proposed order 3'-methyl \textrightarrow 3',5'-dimethyl \textrightarrow 3',4',5'-trimethyl \cite{Zhou2006}.
Methylation of dihydroxy-derivatives such as luteolin and eriodictyol by TaOMT2 only gave rise to 3'-mono-methylated products, which is similar to \pfomt.
However, there it was also demonstrated that TaOMT2 could methylate tamarixetin, the \gls{4o3m} derivative of quercetin, albeit at low activities.

Of the the two \pfomt enzymes, 4'-specific variant and wild-type, only the wild-type showed significant methylation of non-catecholic moieties. 
These findings support the previous results, that could show a pH and magnesium-dependent rate of methylation of \ifa (\autoref{sec:pfomtph}).
Although methylation of 3'-hydroxyl groups was preferred by the wild-type, a tendency to methylate 4'-hydroxyls, when these were the only ones present, could be demonstrated.

Whereas it is clear, that the N-terminus of \pfomt is important for the function of the enzyme, the role of it \invivo is still not fully understood \citep{Kopycki2008b,Vogt2004b,Ibdah2003}.
However, it cannot be ruled out that acts as a signal sequence that can direct the enzyme to different compartments.
The findings presented here might give some implications as to the regulation of \glspl{omt}, such as \pfomt, since the pH can be quite different in different cell compartments in plants \cite{Martiniere2013}.

\section{Conclusion/Discussion}

Enzymatic methylation of non-catecholic moieties, was studied using
the two methyl transferases \pfomt and \somt, of classes I and II
respectively.  Therefore multiple different flavonoid and
phenylproanoid substrates, displaying either single phenolic,
catecholic, \glsentrydesc{3o4m} or \glsentrydesc{4o3m} moieties, were tested. 
Furthermore, the influence of pH and magnesium addition on \pfomt was systematically studied.

In \invivo biotransformation experiments it could be shown, that the
class II \glsentrydesc{omt} \somt is able to methylate flavonoids and stilbenes
at the 4'-OH of the B-ring, regardless the exact moiety (phenolic,
catecholic, \glsentrydesc{4o3m}).  Although overall the conversions were very
low, the conversion of the stilbene resveratrol was superior over all
other tested substrates ($\geq$\SI{86}{\percent} vs. $\geq$\SI{55}{\percent}).
\somt showed methylating activity exclusively when a free 4'-OH was present, suggesting it only acts on 4'-hydroxyl groups. 
Unfortunately, these results are purely based on \invivo bitransformations carried out in \eco.
\somt could not be obtained in pure, soluble form for \invitro characterization.
Nonetheless, using \somt it was shown that \glsentryfirst{doe} and \glsentryfirst{ffd} can be valuable tools for the systematic determination of factors that influence refolding of \glspl{omt}.

\textit{In vitro} experiments using the class I \glsentrydesc{omt} \pfomt, showed that non-catecholic substrates could be methylated.
These findings are contrary to the belief, that \pfomt only acted on vicinal aromatic dihydroxyls that are present in compounds such as eriodictyol or caffeic acid.
The best conversion of non-catechols was achieved for substrates with \glsentrydesc{3o4m}-moieties (e.g. hesperetin, \ifa), even though conversion was observed for phenolic (e.g. naringenin) and \glsentrydesc{4o3m}-substrates (e.g. homoeriodictyol), thus demosntrating the ability of \pfomt to methylate both 3'- and 4'-hydroxyls.
The best conversions were obtained using the \pfomt wild-type at elevated pH and after \cf{Mg^2+} addition.
Magnesium addition and pH displayed synergistic effects, meaning the effects of both are not just additive.
pH optima shifted under conditions with and without addition of \cf{Mg^2+}.
When magnesium was omitted, it seemed as though the chemical environment surronding the enzyme was relayed into the active site.
Thus, non-catecholic substrates were methylated at high pH without magnesium, whereas they were hardly methylated at low pH.

These findings show, that the linear stepwise optimization of reaction condions might not always yield the best overall results, when it comes to such complex systems as enzymes and that synergistic effects need to be considered when looking for the best working conditions.

