%%%-------------------------------------------------%%%
%%% Sub document for appendix %%%
%%%-------------------------------------------------%%%

\part{Appendix}

\chapter{Figures}

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    <<struct.compare.appendix,  echo=F, tidy=TRUE, fig.height=2, fig.width=5.6, out.width='\\textwidth', dev='tikz', error=T>>=
    @
  \end{figureframe}
  \caption{Differences in the dihedrals \textpsi~(red) and \textphi~(black) of the solved \textit{apo}-PFOMT and the structure with bound \gls{sah} (pdb: 3C3Y).
  The secondary structure is displayed at the top. Helices are displayed as rectangles and sheets are shown as arrows.
  Graphical background annotations are used to display the binding sites of \gls{sah} (green) and the metal ion (plum).
  The orange bars indicate regions, where much movement seems to happen upon binding or release of the co-substrate.
  The blue bar shows the region that was annotated as "insertion loop" in previous studies.}
  \label{fig:app_struc.compare}
\end{figure}  

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=\textwidth]{usr/graphics/static/tmp_pca1}
  \end{figureframe}
  \caption{\gls{pca} of \nico leaves infiltraded by \agro carrying different genes.}
  \label{fig:x_pca}
\end{figure}


\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.45\textwidth]{usr/graphics/static/sds_somt}
  \end{figureframe}
  \caption{\sdspage gel of samples aquired during growth curve measurements. The arrow indicated the band that could correspond to the GST-tagged \somt protein.}
  \label{fig:x_somtgel}
\end{figure}

\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.85\textwidth]{usr/graphics/static/SOMT_SS-bridges}
  \end{figureframe}
  \caption{Graphical representation of a \somt model obtained from the PHYRE2 web portal (\url{http://www.sbg.bio.ic.ac.uk/phyre2/html/page.cgi?id=index}) \citep{Kelley2015}}
  \label{fig:x_somtmodel}
\end{figure}


\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/platzhalter}
  \end{figureframe}
  \caption{Chromatogram of the gel filtration analysis of refolded \somt.}
  \label{fig:x_somt_gefi}
\end{figure}

%% CD spectrum of SOMT
\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \includegraphics[width=0.75\textwidth]{usr/graphics/static/CD_SOMT}
  \end{figureframe}
  \caption{CD-spectrum of refolded \somt (black) compared to the spectrum that was calculated by the K2D3 web service (\url{http://cbdm-01.zdv.uni-mainz.de/~andrade/k2d3//index.html}). Secondary structure estimates from the spectrum are \SI{12.39}{\percent} \textalpha-helix and \SI{32.51}{\percent} \textbeta-sheet.}
  \label{fig:x_somt_cd}
\end{figure}


%% Main effects plot PFOMT conversion of non-catechols
\begin{figure}[htpb]
  \centering
  \begin{figureframe}
    \begin{subfigure}{0.73\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_MEP_pfomt}
      \caption{Main effects plot for PFOMT conversion of non-catechols. 
        Phenolics (A) are converted worst and conversion of \gls{3o4m} is generally best.
        There seem to be positive effects of \cf{Mg^2+} and pH.
        The wild-type enzyme shows better conversion than the 4'-specific variant.}
      \label{fig:x_mep_pfomt}
    \end{subfigure}
    %% Interaction plot Mg/pH
    \begin{subfigure}{0.23\textwidth}
      \includegraphics[width=\textwidth]{usr/graphics/static/tmp_pfomt_MgpH}
      \caption{Interaction plot of \cf{Mg^2+} and pH for the conversion of non-catechols and catechols by \pfomt.
        There seems to be an interaction between \cf{Mg^2+} addition and pH. However, statistical tests suggest this interaction might not be significant.\\
        blue -- \cf{Mg^2+} addition, red -- no \cf{Mg^2+}}
      \label{fig:x_iap_pfomt}
    \end{subfigure}
  \end{figureframe}
  \caption{}
\end{figure}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%% Tables
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\chapter{Tables}

%% Table of cloned constructs (appendix)
\ostTableEnv|[label = tab:molecularcloning]{%
  \scriptsize
  \begin{tabularx}{\textwidth}{l p{0.35\textwidth} p{0.1\textwidth} l X}
    \toprule
    construct name & description & entry constructs & destination & workflow steps (primers/cloning sites)\\ 
    \midrule
    pBEW101 & & & & \\
    pBEW102 & lsrA promoter & & & \\
    pBEW103 & pBEW102 with BamHI cloning site & pBEW102 & & \\
    pBEW104 & rhaP$_{\mathrm{BAD}}$ promoter & pBEW4b & pBEW103 & amplification (\textit{pRha1.fw/rv}), cloning (BglII, BamHI)\\
    pBEW106 & pICH413038-somt & pET28MC-somt & pICH413038 & amplification (\textit{somt1/2/3/4}), golden gate cloning (BpiI) \\
    pBEW107 & & pICH51266, pBEW106, pICH41421 & pICH75044 & golden gate cloning (BsaI) \\
    pBEW1a & & & & \\
    pBEW1b & & & & \\
    pBEW2a & & & & \\
    pBEW2b & & & & \\
    pBEW3a & & & & \\
    pBEW3b & & & & \\
    pBEW4a & & & & \\
    pBEW4b & & & & \\
    pET28-pfomt & \textit{pfomt} gene in pET-28a(+), endogenous NdeI site removed & pQE30-pfomt & pET-28a(+) & mutagenesis (\emph{pfomt1.fw/rv}), amplification (\emph{pfomt2.fw/rv}), cloning (NdeI, EcoRI) \\
    pET20-somt & N-terminal pelB-tag fusion for periplasmic expression & & pET20-b(+)& \\
    pET28-somt & & & pET28-a(+)& \\
    pET28MC-somt & & &  & \\
    pET32-somt & N-terminal TrX-tag fusion & & pET-32a(+) & \\
    pET41-somt & N-terminal GST-tag fusion & & pET-41a(+) & \\
    pUC19* & added BglII site & pUC19 & - & mutagenesis (\textit{pUC1.fw/rv}) \\
    pUCB1 & pUC19 derivative with lsrA promoter & lsr-XX-DAS & pUC19* & cloning (NdeI, BglII) \\ 
    pUCB1-sfGFP-DAS+4 & & & & \\
    
    \bottomrule
  \end{tabularx}
}{%
Overview over the contructs produced for the present thesis. Each step during the production of the construct is given in the workflow steps column. Primers (italic font) or restriction sites used during each step are displayed in parenthesis.
}
% }}}

\begin{lstTabularx}[1]{\linewidth}{>{\scriptsize}p{0.45\textwidth}XX>{\scriptsize}p{0.12\textwidth}}
  \caption{\gls{sam} analogues that have been used with \glspl{mt}. Targets: \emph{P} -- peptide/protein, \emph{D} -- DNA, \emph{R} -- RNA, \emph{S} -- small molecule.} \\
  \label{tab:mtlit}
  \hfill \\
     \toprule
	\small analogue & enzyme & target & \small references \\ 
     \midrule
     \endfirsthead
     \toprule
	\small analogue & enzyme & target & \small references \\ 
     \midrule
     \endhead
     \small\textit{SAM} & & &\\
    % \cf{-CH3} & & &\\
     \cf{-CH2-CH3} & PRMT1, M.TaqI, M.HhaI, M.BcnIB, RebM, RapM & S,P,D & \citep{Thomsen2013,Dalhoff2006,Singh2014,Law2015} \footnote{Singh \etal (2014) published a series of 44 biocatalytically synthesized \gls{sam} and \gls{seam} derivatives, most of which were not tested towards their alkyl donation potential in \gls{mt} reactions.}\\
     \cf{-CH2-CH2-CH3} & PRMT1, M.TaqI, M.HhaI, M.BcnIB & P,D & \citep{Thomsen2013,Dalhoff2006}\\
     \cf{-CH2-CH2-CH2-CH3} & PRMT1 & P & \citep{Thomsen2013}\\
     \cf{-CH2-C6H5} & NovO, CouO, PRMT1 & S,P & \citep{Stecher2009,Thomsen2013}\\
%    \cf{-CH2-CH2-C6H5} & & & \citep{Thomsen2013}\\ % not tested with MTs
%    \cf{-CH2-CH(-NH2)-COOH} & & & \citep{Thomsen2013}\\ % not tested with Mts
%     \cf{-CH2-C(=O)-NH2} & & & \citep{Thomsen2013}\\ % not tested with Mts
     \cf{-CH2-C(=O)-CH3} & COMT, TPMT, CazF & S & \citep{Lee2010,Winter2013}\\
     
     \cf{-CH2-CH=CH2} & NovO, CouO, RapM, PRMT1, M.TaqI, M.HhaI, M.BcnIB, RebM, Tgs & P,S,D & \citep{Stecher2009,Thomsen2013,Dalhoff2006,Singh2014,Wang2011,Wang2011a,Tengg2012a,Law2015,Schulz2013} \\
     \cf{-CH2-CH=CH-CH3} & NovO, CouO & S & \citep{Stecher2009}\\
  %  \cf{-CH2-CH=CH-CH2-CH3} & & & \citep{Wang2014}\\ %% only SAM synthase
    
     \cf{-CH2-C#CH} & Dim-5, \textit{Hs}MLL, TRM1, NovO, CouO, PRMT1 & P,R,S & \citep{Willnow2012,Stecher2009,Wang2011,Wang2011a,Islam2011}\\
     \cf{-CH2-C#N} & RebM & S & \citep{Singh2014} \\
     \cf{-CH2-CH2-C#CH} & PKMT & P & \citep{Islam2011}\\
     \cf{-CH2-CH2-CH2-C#CH} & PKMT & P & \citep{Islam2011}\\
     \cf{-CH2-C#C-CH3} & NovO, CouO, M.HhaI, M.TaqI, M.BcnIB & S,D & \citep{Stecher2009,Dalhoff2006,Lukinavicius2013}\\
     \cf{-CH2-C#C-CH2-CH3} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-C#C-CH2-NH2} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-C#C-CH2-NH-C(=O)(-CH2-)3-NH2} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-C#C(-CH2-)3-NH2} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-C#C(-CH2-)3-NH-C(=O)(-CH2-)3-NH2} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-C#C(-CH2-)3-C#CH} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-C#C(-CH2-)3-N3} & M.HhaI & D & \citep{Lukinavicius2013} \\
     \cf{-CH2-CH=CH-C#CH} & Dim-5, \textit{Hs}MLL, TRM1, PRMT1, Tgs & P,R & \citep{Willnow2012,Peters2010,Wang2011,Wang2011a,Motorin2011,Islam2011,Schulz2013}\\
     \cf{-CH2-CH=CH-CH2-C#CH} & & & \citep{Wang2011a,Islam2011} \\
     \cf{-CH2-CH=CH-CH2-O-CH2-C#CH} & PRMT1 & P & \citep{Wang2011,Wang2011a}\\
     
    \\
     \small\textit{SeAM} & & &\\
     \cf{-CH3} & & & \\
     \cf{-CH2-C#CH} & Dim-5, \textit{Hs}MLL, TRM1, RebM, CazF & P,R,S & \citep{Willnow2012,Singh2014,Bothwell2012, Winter2013}\\
     
     \\
     \small\textit{N}-mustard derivatives & & &\\
     \cf{-CH2-CH2-I} & RebM & S & \citep{Zhang2006} \\
     
     \bottomrule
\end{lstTabularx}


\ostTableEnv[label = tab:xtalstat]{%
  \begin{tabularx}{\textwidth}{Xll}%
% \caption{Crystallographic data, phasing and refinement statistics.} \\
%  \label{tab:d-xtalstat}
%    \small
%  \hfill \\
    \toprule
    & 140519\_PFOMT  & MC001413-G10.1 \\
    \midrule
 %    \endfirsthead
 %   \toprule
 %   & PFOMT  & \\
 %   \midrule
 %   \endhead
     
    \textit{data collection} &  & \\
    wavelength (\gls{a}) &  & \\
    resolution (\gls{a}) & 1.95  & \\
    total reflections & \num{392368} & \\
    unique reflections & \num{125822} & \\
    completeness (\%) & 99.12 & \\
    $I/\sigma (I)$ & 9.9 & \\
    $R^a_{sym}$ &  & \\
    redundancy &  & \\
    space group & $P2_12_12_1$ & \\
    cell dimensions (\gls{a}) &  & \\
    \textit{a} & 86.16 & 48.88 \\
    \textit{b} & 128 & 71.36 \\
    \textit{c} & 129.3 & 127.80 \\ \\
    
    \textit{refinement} &  & \\
    $R_{\mathrm{work}}$/$R_{\mathrm{free}}$ & 0.21369 / 0.24700 & \\
    \gls{rmsd} bond lengths (\gls{a}) & 0.0199  & \\
    \gls{rmsd} bond angles (Â°) & 2.0568 & \\
    B-values ($\mathrm{\AA^2}$) & 21.593  & \\
    water &  & \\
    ions &  & \\ \\
     
    \textit{Ramachandran plot} (\%) &  & \\
    favoured & 96.82 & \\
    allowed & 2.38 & \\
    outliers & 0.8 & \\
    \bottomrule
  \end{tabularx}
}{%
Crystallographic data, phasing and refinement statistics.
}


\begin{table}[htpb]
  \caption{Coefficients of the model (\ref{eq:cat}) for activity of catecholic methylation. The factor $\factor{Mg}$ is a categorical variable (addition/no addition) and can therefore only be 0 or 1.}
  \label{tab:lm-cat}
  \centering\small
  \begin{tabularx}{\textwidth}{lrrrrr}
    \toprule
    & Estimate & Std. Error & t value & p-value &  \\ 
    \midrule
    $($Intercept) & -421929.9946 & 356063.7085 & -1.18 & 0.2557 &  \\ 
    \factorx{Mg} & -839999.8874 & 503550.1257 & -1.67 & 0.1175 &  \\ 
    \factorx{pH} & 103271.3345 & 97739.1728 & 1.06 & 0.3086 &  \\ 
    $\factor{pH^2}$ & -4977.7406 & 6512.6996 & -0.76 & 0.4574 &  \\ 
    \factorx{Mg}$\times$\factorx{pH} & 266920.7964 & 138224.0638 & 1.93 & 0.0740 & . \\ 
    \factorx{Mg}$\times\factor{pH^2}$ & -19830.2264 & 9210.3481 & -2.15 & 0.0492 & * \\ 
    \bottomrule
  \end{tabularx}
\end{table}

\begin{table}
  \caption{Coefficients obtained for linear regression model suing the \ifa subset after shrinkage using the Lasso method and 5-fold cross validation. Only non-zero coefficients (variables actually do have an effect) are retained during the Lasso. Seed was set to 1337.}
  \label{tab:lasso-ifa}
  \centering\small
  \begin{tabularx}{\textwidth}{lr}
    \toprule
    variable & coefficient \\ 
    \midrule
    $($Intercept) & -509.8385 \\ 
    \factorx{pH} & 73.4085 \\ 
    \factorx{Mg} & -1606.1362 \\ 
    \factorx{pH}$\times$\factorx{Mg} & 296.0753 \\ 
    \bottomrule
  \end{tabularx}
\end{table}

\begin{table}
  \caption{Coefficients obtained for linear regression model using the catechols subset after shrinkage using the Lasso method and 5-fold cross validation. Only non-zero coefficients (variables actually do have an effect) are retained during the Lasso. Seed was set to 1336.} 
  \label{tab:lasso-catechol}
  \centering\small
  \begin{tabularx}{\textwidth}{lr}
    \toprule
    variable & coefficient \\ 
    \midrule
    $($Intercept) & -467632.3821 \\ 
    \factorx{pH} & 94469.8366  \\ 
    \factorx{pH}$\times$\factorx{Mg} & 19068.9540 \\ 
    \factorx{pH}$\times\factor{pH^2}$ & -381.5863 \\ 
    \factorx{pH}$\times$\factorx{Mg}$\times\factor{pH^2}$ & -292.3608 \\ 
    \bottomrule
  \end{tabularx}
\end{table}
