---
title: "WEB346"
author: "Benjamin Weigel"
date: "July 27th, 2015"
output: rmarkdown::tufte_handout
---

```{r, cache=FALSE, echo=FALSE}
knitr::opts_chunk$set(dev='tikz', error=T, warning=F, message=F, size='tiny', echo=F)
```

# Question

Do Mg\textsuperscript{2+}, pH, chemical motif (e.g. catecholic, phenolic, 3'-OMe, 4'-OMe) and the choice of enzyme (WT or 4'-variant) influence the observed conversion of flavonoids and phenylpropanoids by the O-methyltransferase PFOMT?

# Introduction

17 different flavonoid and phenyl propanoid substrates were tested for methylation.
These substrates can loosely be categorized into four groups by the chemical motif that is to be methylated (e.g. catecholic, phenolic, 3'-OMe, 4'-OMe).
Three other factors are studied that might also influence the conversion. The addition of Mg\textsuperscript{2+}, high (8.6) or low (7.5) pH and the enzyme variant (WT or 3'-variant) that is used are also varied.

\newthought{A total of 96 experiments} were conducted to cover each group at least with three independent observations. 

\begin{marginfigure}
$$\mathrm{Mg} \times \mathrm{pH} \times \mathrm{variant} \times \mathrm{motif} $$
$$2 \times 2 \times 2 \times 4 = 32$$
\caption{number of groups studied}
\end{marginfigure}

# Results

## Data

```{r, echo=F, message=FALSE, results='asis', fig.margin=T}
library(magrittr)
library(dplyr)
library(FrF2)
library(ggplot2)
library(xtable)
options(xtable.comment = FALSE)
options(xtable.booktabs = TRUE)

load("WEB346.results.rda")

motif <- data.frame(substrate = 1:17, 
                    motif = c(rep(c("phenolic", "catecholic", "4-O-Me", "3-O-Me"), 2), 
                              rep("phenolic", 2),
                              "phenolic", "catecholic", "3-O-Me", "4-O-Me", "phenolic", "catecholic", "catecholic"),
                    select = c(rep(T, 9), rep(F, 2), rep(T, 3), rep(F, 3)))
WEB346.results <- merge(WEB346.results, motif, by="substrate")
WEB346.results$pH <- ifelse(WEB346.results$pH > 8, "high", "low")
WEB346.results %<>% mutate(#pH = factor(pH, levels = c("low", "high"), ordered = T),
                           Mg = as.factor(Mg),#
                           motif = factor(motif, labels=c(LETTERS[1:4]) , levels=c("phenolic", "catecholic", "4-O-Me", "3-O-Me"))) 

data.blank <- WEB346.results %>% filter(select == T) %>%
  select(substrate, value, pH, Mg, variant, motif)

WEB346.results %<>% filter(select == T, variant != "none") %>%
  select(substrate, value, pH, Mg, variant, motif)
rm(motif)

xtable(head(WEB346.results), caption = "First rows of dataframe")
```

```{r, results='asis', fig.margin=TRUE}
levtab <- xtable(data.frame(labels=c(LETTERS[1:4]) , motif=c("phenolic", "catecholic", "4-O-Me", "3-O-Me")), caption="Labels in the data.frame and their corresponding motif.")
print(levtab, floating.environment='margintable')
```

## Significance no enzyme vs. enzyme

Do the amounts of produced product vary significantly between treatments were no enzyme was added and trreatm,ents with enzyme?
Only ran blanks at high pH with no magnesium added.
Subset data to only include high pH, no Mg, no catechols. The catechols were removed to avoid the bias associated with an autocorrelation between product and catecholic moiety.

\newthought{From the ANOVA table it is clear}, that conversion is not due to chance. The p-value for the variant factor is almost 0.
This means, that we should be able to use the data for analyis.

```{r ,fig.margin=F, out.width='0.2\\linewidth', fig.height=7, fig.width=7, fig.cap="Diagnostic plots of the ANOVA", results='asis', echo=1}
lm(value~variant*motif, data = data.blank %>% filter(pH =="high", Mg == F, motif!="B")) %>% aov() %>% 
  summary() %>% 
  xtable(., caption="ANOVA table for comparison of blanks and samples. There is a stitistical significance between treatments, where o enzyme was added and treatments that contained enzymne (the wt).")

par(mfrow=c(2,2))
lm(value~variant*motif, data = data.blank %>% filter(pH =="high", Mg == F, motif!="B")) %>% aov() %>% plot()
```

```{r, fig.margin=T, fig.height=4, fig.cap="Main effects plot for factors that influence product amount."}
data.blank %<>% filter(pH =="high", Mg == F,  motif!="B")

a<-data.blank %>%
  group_by(motif) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(a)[1] <- "state"
a %<>% mutate(state = as.factor(state), ME = "motif")

d<-data.blank %>%
  group_by(variant) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(d)[1] <- "state"
d %<>% mutate(state = as.factor(state), ME = "variant")

MEgg <- rbind(a,d)

ggplot(MEgg, aes(x=state, y=value)) + 
  theme_bw() +
  theme(axis.text.x=element_text(angle=90, vjust = 0),
        axis.title.x=element_text(vjust = 0)) +
  geom_line(aes(group = ME), color="cornflowerblue", size=1) +
  geom_point(size=3) + 
  facet_wrap(~ME, nrow=1, scales="free_x") +
  labs(y="mean $\\frac{\\mathrm{SAH}}{\\mathrm{flavone}} (\\mathrm{AU})$", x="factor level") +
  geom_hline(aes(yintercept=mean(MEgg$value)), linetype=2, size=1)
```

## Main effects plots

The main effects plots (\ref{fig:mep}) give an overview of what happens. The motif clearly has the biggest influence on conversion. This makes sense, since PFOMT acts on catecholic moities.

```{r MEP, fig.height=4, fig.width=6, fig.cap=c("Main effetcs plots for the factor variables. Clearly the motif seems to have the biggest impact. Catecholic moieties are converted most effectively.", "Main effects plot, when catecholic moieties factor is omitted. Phenolics are converted worst. The wildtype has the best activity.")}

a<-WEB346.results %>%
  group_by(motif) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(a)[1] <- "state"
a %<>% mutate(state = as.factor(state), ME = "motif")

b<-WEB346.results %>%
  group_by(Mg) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(b)[1] <- "state"
b%<>% mutate(state = as.factor(state), ME = "Mg")

c<-WEB346.results %>%
  group_by(pH) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(c)[1] <- "state"
c %<>% mutate(state = as.factor(state), ME = "pH")

d<-WEB346.results %>%
  group_by(variant) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(d)[1] <- "state"
d %<>% mutate(state = as.factor(state), ME = "variant")

MEgg <- rbind(a,b,c, d)

ggplot(MEgg, aes(x=state, y=value)) + 
  theme_bw() +
  theme(axis.text.x=element_text(angle=90, vjust = 0),
        axis.title.x=element_text(vjust = 0)) +
  geom_line(aes(group = ME), color="cornflowerblue", size=1) +
  geom_point(size=3) + 
  facet_wrap(~ME, nrow=1, scales="free_x") +
  labs(y="mean $\\frac{\\mathrm{SAH}}{\\mathrm{flavone}} (\\mathrm{AU})$", x="factor level") +
  geom_hline(aes(yintercept=mean(MEgg$value)), linetype=2, size=1)

a<-WEB346.results %>% filter(motif != "B") %>%
  group_by(motif) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(a)[1] <- "state"
a %<>% mutate(state = as.factor(state), ME = "motif")

b<-WEB346.results %>% filter(motif != "B") %>%
  group_by(Mg) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(b)[1] <- "state"
b%<>% mutate(state = as.factor(state), ME = "Mg")

c<-WEB346.results %>% filter(motif != "B") %>%
  group_by(pH) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(c)[1] <- "state"
c %<>% mutate(state = as.factor(state), ME = "pH")

d<-WEB346.results %>% filter(motif != "B") %>%
  group_by(variant) %>%
  summarise_each(funs(mean(., na.rm=TRUE)), value)
names(d)[1] <- "state"
d %<>% mutate(state = as.factor(state), ME = "variant")

MEgg <- rbind(a,b,c, d)

ggplot(MEgg, aes(x=state, y=value)) + 
  theme_bw() +
  theme(axis.text.x=element_text(angle=90, vjust = 0),
        axis.title.x=element_text(vjust = 0)) +
  geom_line(aes(group = ME), color="cornflowerblue", size=1) +
  geom_point(size=3) + 
  facet_wrap(~ME, nrow=1, scales="free_x") +
  labs(y="mean $\\frac{\\mathrm{SAH}}{\\mathrm{flavone}} (\\mathrm{AU})$", x="factor level") +
  geom_hline(aes(yintercept=mean(MEgg$value)), linetype=2, size=1)
```

```{r IAP, results='asis', fig.height=3, fig.width=2.5, fig.margin=T, fig.cap='Interaction plot for Mg and pH. The lines suggest an interaction between pH and Mg, but this is not enough evidence to say wether that interaction is significant. red -- no magnesium, blue -- magnesium added.'}
d<-WEB346.results %>%
  group_by(Mg,pH) %>%
  summarise_each(funs(mean(., na.rm=TRUE), sd(., na.rm=TRUE)), value)

ggplot(d, aes(x=pH, y=mean, group = Mg)) + 
  theme_bw() +
  theme(axis.text.x=element_text(angle=90, vjust = 0),
        axis.title.x=element_text(vjust = 0),
        legend.position="none") +
  geom_line(aes(color = Mg), size=1) +
  geom_point(size=3) +
  labs(y="mean $\\frac{\\mathrm{SAH}}{\\mathrm{flavone}} (\\mathrm{AU})$", x="pH") +
  geom_hline(aes(yintercept=mean(d$mean)), linetype=2, size=1)
```

\newthought{The interaction plot}(\ref{fig:IAP}) of Mg and pH displays an interaction. When magnesium is added the activity tends to be higher. This effect is more pronounced at low pH values. 
It is not possible to say wether this effect is significant without the according satistical test.
Indeed the ANOVA results suggest, that there is no significance. In fact there doesn't even seem to be a statistical significance from Mg addition or pH alone. 

## Anova

Two-way ANOVA test for simple models and more complex models were prepared.
The p-values for the complex model are very low all over. However it is very likely that this is an overinterpretation of the data.
The data/experiment might be too complex to derive anything of value from the information. 
The use of more complex modelling techniques mnight shed some light.
However, when the substrates with catecholic moieties are excluded from the results, there is statistical significance at leas for Mg and pH.

```{r, results='asis', echo=TRUE}
data.lm <- lm(value ~ pH*Mg, data=WEB346.results)
summary(aov(data.lm)) %>% xtable(.,caption="ANOVA table.")

data.lm <- lm(value ~ pH*Mg*motif*variant, data=WEB346.results)
summary(aov(data.lm)) %>% xtable(.,caption="ANOVA table of the most complex model. Significance is everywhere ... :O")


data.lm <- lm(value ~ pH*Mg+variant, data=WEB346.results %>% filter(motif != "B"))
summary(aov(data.lm)) %>% xtable(.,caption="ANOVA table when catecholics are excluded. At least the data suggest significance for pH and Mg.")
```


## Regression Tree

A regression tree can be built from the data. At first glance it shows, that the motif is especially important for conversion.
This is trivial, since PFOMT is a 4'-OMT that acts on catecholic motifs. The substrates with catecholic motifs are thus converted more efficiently.

\newthought{It seems as if the variant} is influenced by pH and metal addition more clearly thatn the wt-enzyme, since the tree splits up more.

```{r, fig.margin = F, fig.cap="RegressionTree of the data built with the `rpart`-package. The motif effects the conversion most. It seems that only the variant is influenced by pH and Mg.", dev='CairoPDF', echo=F, fig.width=10, fig.height=8}
library(rpart)
library(rpart.plot)
library(rattle)

data.tree <- rpart(value ~ ., 
                   data=WEB346.results %>% select(-c(substrate)),
                   control = c(minsplit = 3,
                               minbucket = 3,
                               cp = 0.01))
fancyRpartPlot(data.tree, digits=3)
rm(data.tree)
```

## Linear Models

At first it was checked, if pH and Mg have an influence on the conversion.
It does not seem that Mg or pH have any influence on the conversion, as the p-values are much too high.
This could be the result of the fact that the conversion reactions were incubated for 16 hours.
Possible intricacies in the conversion (due to different reaction velocities) can not be distinguished from one another if the reaction time is so long that all substrate is used up.

```{r linear.model1, results='asis'}
data.lm <- lm(value ~ pH*Mg, data=WEB346.results)
summary(data.lm) %>% xtable(.,caption="A linear model with pH and Mg as factors is not a lot better than random guesses. p-values are very high. Thus this alone does not explain the variance.")
```

\newthought{However, when the catecholics are omitted} as a factor level it becomes clear that Mg and pH DO in fact influence the conversion.
After backwards selection only the factors pH, Mg, variant and te interactions between variant and Mg and pH are retained.

```{r linear.model2, message=F, echo=c(1,2)}
data.lm <- lm(value ~ pH*Mg*variant, data=WEB346.results %>% filter(motif != "B"))
data.lm <- MASS::stepAIC(data.lm, direction = "backward")
#summary(data.lm) %>% xtable(.,caption="When the motif is included the model describes the data much better. The low p-values even suggest, that there is an interaction effect between pH and substrates with catecholic motifs, as well as an interaction pH:Mg:Catechol.")
```

```{r, results='asis'}
summary(aov(data.lm)) %>% xtable(., caption="When catecholics are omitted there is significance in the following terms: pH, Mg, variant, pH:variant, Mg:variant")
```

 \marginnote{This is a margin note.  Notice that there isn't a number preceding the note.}


```{r regression_tree, fig.margin = F, fig.cap="RegressionTree of the data of only substrates with phenolic, 3'-OMe and 4'-OMe moieties. Catecholic substrates are omitted.", dev='CairoPDF', fig.width=10, fig.height=8}
data.tree <- rpart(value ~ ., 
                   data=WEB346.results %>% filter(motif != "B") %>% select(-substrate),
                   control = c(minsplit = 3,
                               minbucket = 3,
                               cp = 0.01))
fancyRpartPlot(data.tree, digits=3)
```

## Model selection using lasso regression

Lasso regression can shrink model coefficients to zero. This helps in variable selection.
Variables that were shrunken to 0 tend to have little impact on the prediction capabilities of the model.

```{r lasso_sparse_matrix, fig.margin=TRUE, fig.cap="Lasso regression on the model."}
library(glmnet)

data <- WEB346.results# %>% filter(motif != "B")

# create model matrix for glmnet function
# needs to be a sparse matrix because of mixed numerical and factor variables
x<-sparse.model.matrix(value~pH*Mg*variant*motif, data = data)
y <- data[,2]

grid <- 10^seq(10,-10,length.out = 100)
lasso <- glmnet(x, y, family = "gaussian", alpha = 1, lambda=grid)
plot(lasso)
```


## Cross-validation to select best model

After cross-validation only 11 variuables have non-zero coefficients and are thus used during prediction.
All other variabkes were shrunken to zero.

```{r lasso1, eval=F, echo=T}
cv.lasso <- cv.glmnet(x,y,alpha=1,nfolds=5)
```


```{r lasso2, results='asis', fig.margin=T, fig.cap=c("Cross validation results for lasso regression. The best model only needs around 10 variables to describe the data with low error.", "The variables that have non-zero coefficients.")}
cv.lasso <- cv.glmnet(x,y,alpha=1,nfolds=5)
bestlam <- cv.lasso$lambda.min
lasso.coeff <- predict(cv.lasso, type="coefficients", s=bestlam)

plot(cv.lasso)

rm(levtab)
plot(lasso.coeff)
df <- data.frame(variable = dimnames(lasso.coeff)[[1]][summary(lasso.coeff)$i],
           coefficient = summary(lasso.coeff)$x)
```


# Answer

- the conversion is not by chance (significance between blank and enzyme treated groups)
- The biggest influence on conversion has the presence of a catechgolic moiety (surprise, surprise !)
- There is an effect of Mg and pH (main and interaction effect). 
  However the statistical test don't support this notion.
  $\rightarrow$ incubation time too large, this makes results badly interpretable !!!
  $\rightarrow$ when catecholic moieties are excluded there is significance for Mg addition and possibly for pH; however no interaction
- data is also very complex with possibly up to four factor interactions
- regression tree used for interpretation
  $\rightarrow$ pH and Mg can explain some of the variance



```{r, results='asis'}
xtable(df, caption="Variables and coefficients that were retained. Non-zero coefficients not shown.", digits = 4)
```

\newpage

# Conversion of substrates

The conversion of the substrate in % is of interest.

## Calculation of SAH and SAM concentration

The SAH and SAM concentration were estimated from the area-under-curve (AUC) of the SAM and SAH peaks.
The displayed formula also already include the conversion. $x_\mathrm{SAH}$ is a direct measure for the conversion.

\begin{marginfigure}
$$A_\mathrm{SAM} + A_\mathrm{SAM} = 1 \approx 500 \mathrm{uM}$$
$$x_\mathrm{SAH} = \frac{A_\mathrm{SAH}}{A_\mathrm{SAM} + A_\mathrm{SAM}}$$
$$c_\mathrm{SAH} = x_\mathrm{SAH} \times 500 \mathrm{uM}$$
\caption{Calculation of specific activity.}
\end{marginfigure}

0.4 mM substrate and 0.5 mM biologically active SAM were added.
That means the substrate conversion can be estimated by $conversion = \frac{0.4\mathrm{mM}}{0.5\mathrm{mM} \times  x_\mathrm{SAH}}$.

```{r concentration_estimation, fig.margin=T, fig.cap="Estimated SAH and SAM concentration plotted against the AUC. Linear best-fit models with 95\\% prediction intervals are included."}
load("WEB346.results.rda")

motif <- data.frame(substrate = 1:17, 
                    motif = c(rep(c("phenolic", "catecholic", "4-O-Me", "3-O-Me"), 2), 
                              rep("phenolic", 2),
                              "phenolic", "catecholic", "3-O-Me", "4-O-Me", "phenolic", "catecholic", "catecholic"),
                    select = c(rep(T, 9), rep(F, 2), rep(T, 3), rep(F, 3)),
                    group = c(rep("flavanone", 4), rep("flavone", 4), rep("cinnamic acid", 6), rep("flavonole", 3)))

WEB346.results <- merge(WEB346.results, motif, by="substrate")
WEB346.results$pH <- ifelse(WEB346.results$pH > 8, "high", "low")
WEB346.results %<>% mutate(pH = as.factor(pH),
  Mg = as.factor(Mg),
  motif = factor(motif))#, labels=c(LETTERS[1:4]) , levels=c("phenolic", "catecholic", "4-O-Me", "3-O-Me"))) 

rm(motif)

WEB346.results %<>% select(substrate, pH, Mg, variant, motif, select, SAH, SAM, flavon, group)

# conversion is x.SAH
WEB346.results %<>% mutate(x.SAH = (SAH/flavon)/((SAH/flavon)+(SAM/flavon)), 
                     c.SAH = (x.SAH*500/0.81),
                     x.SAM = (SAM/flavon)/((SAH/flavon)+(SAM/flavon)), 
                     c.SAM = (x.SAM*500/0.81))

res.save <- WEB346.results

## rearrange data.frame for plotting in ggplot
WEB346.results <- tidyr::gather(WEB346.results, key, area, SAH:SAM)
WEB346.results$c.SAM[with(WEB346.results, which(key == "SAH"))] <- NA
WEB346.results$c.SAH[with(WEB346.results, which(key == "SAM"))] <- NA
WEB346.results <- tidyr::gather(WEB346.results, key2, concentration, c.SAH, c.SAM)

## calcuate linear model for relationship concentration to area
sah.lm <- lm(concentration~area, data=WEB346.results %>% filter(key == "SAH"))
#sam.lm <- glm(concentration~area, data=WEB346.results %>% filter(key == "SAM"), family = gaussian(link = "log"))
sam.lm <- lm(concentration~area, data=WEB346.results %>% filter(key == "SAM"))
## predict new data
ypred <- data.frame(area=seq(0,4e6,length.out = 20))
ypred <- cbind(ypred, sah=predict(sah.lm, type="response", interval="prediction", newdata=ypred))
ypred <- cbind(ypred, sam=predict(sam.lm, type="response", interval="prediction", newdata=ypred))


ggplot() +
  theme_bw() +
  geom_point(data=WEB346.results, aes(x=area, y=concentration, shape=key), alpha=0.25, size=2) +
  geom_line(data=ypred[ypred$area<3e6,], aes(x=area, y=sah.fit), color="blue", fill="blue", linetype=2) +
  geom_ribbon(data=ypred[ypred$area<3e6,], aes(x=area, ymin=sah.lwr, ymax=sah.upr), alpha=0.1) +
  geom_line(data=ypred[ypred$area>1.5e6,], aes(x=area, y=sam.fit), color="red", linetype=2) +
  geom_ribbon(data=ypred[ypred$area>1.5e6,], aes(x=area, ymin=sam.lwr, ymax=sam.upr), fill="red", alpha=0.1)   +
  labs(x = "area-under-curve [AU]", y = "concentration $[\\mu M]$")
```

```{r, results='asis'}
xtable(summary(sah.lm), digits=-4)
```



\newpage
## Calculation of the coefficients for concentration~area bay bootstrapping

From the histogram of the coefficients obtained by the bootstrap it is clear that there is a non-gaussian distribution for the coefficients for SAM.
This could be due to the fact that there is a huge amount of samples with high SAM concentrations, which produces bias.

```{r, echo=c(6:13), fig.width=7, fig.height=7, fig.cap=c("Histogram of coefficient values obtained by bootstrap with n=1000", "Calculated data from the bootstrap.")}
load("WEB346.CV.rda")

library(boot)
set.seed(23)

boot.fn <- function(data, index){
  data <- data[index,]
  data %<>% mutate(x = (area/flavon)/((SAH/flavon)+(SAM/flavon)), 
                   concentration = (x*500/0.81)) # the total concentration of SAm is needed (R+L) --> 0.81 of SAm is biologicalle active
  return(coef(lm(concentration~area, data=data)))
}


lm.boot.sah <- boot(WEB346.CV %>% select(SAH, SAM, flavon) %>% mutate(area = SAH), boot.fn, 1000)
lm.boot.sam <- boot(WEB346.CV %>% select(SAH, SAM, flavon) %>% mutate(area = SAM), boot.fn, 1000)

pred.SAH <- data.frame(area =WEB346.CV$SAH, concentration=WEB346.CV$SAH*lm.boot.sah$t0[2]+lm.boot.sah$t0[1])
pred.SAM <- data.frame(area =WEB346.CV$SAM, concentration=WEB346.CV$SAM*lm.boot.sam$t0[2]+lm.boot.sam$t0[1])

par(mfrow=c(2,2))
hist(lm.boot.sah$t[,1], xlab="beta0.sah", main="")
hist(lm.boot.sah$t[,2], xlab="beta1.sah", main="")
hist(lm.boot.sam$t[,1], xlab="beta0.sam", main="")
hist(lm.boot.sam$t[,2], xlab="beta1.sam", main="")
#title("histogram of coefficient values obtained by bootstrap with n=1000", line = -2, outer = TRUE)


ggplot() +
  theme_bw() +
  geom_point(data=WEB346.results, aes(x=area, y=concentration, shape=key), alpha=0.25, size=2) +
  geom_point(data=pred.SAH, aes(x=area, y=concentration), color="red") +
  geom_point(data=pred.SAM, aes(x=area, y=concentration), color="blue") +
  #geom_line(data=ypred[ypred$area<3e6,], aes(x=area, y=sah.fit), color="blue", fill="blue", linetype=2) +
  #geom_ribbon(data=ypred[ypred$area<3e6,], aes(x=area, ymin=sah.lwr, ymax=sah.upr), alpha=0.1) +
  #geom_line(data=ypred[ypred$area>1.5e6,], aes(x=area, y=sam.fit), color="red", linetype=2) +
  #geom_ribbon(data=ypred[ypred$area>1.5e6,], aes(x=area, ymin=sam.lwr, ymax=sam.upr), fill="red", alpha=0.1)   +
  labs(x = "area-under-curve [AU]", y = "concentration $[\\mu M]$")

lm.boot.sah
lm.boot.sam
```

\newpage

```{r mf.labeller}
mf_labeller <- function(var, value){
   value <- as.character(value)
   if(var == "Mg"){
       value[value == "TRUE"] <- "10 mM Mg"
       value[value == "FALSE"] <- "no Mg"
   }
   if(var == "pH"){
       value[value == "low"] <- "low pH"
       value[value == "high"] <- "high pH"
   }
   
   return(value)
}
```

```{r, cache=FALSE, echo=FALSE}
knitr::opts_chunk$set(fig.height=4)
```

# Phenolic substrates

```{r plot_conversion_phenol, fig.margin=T, fig.cap="Comparison of conversion of phenolic substrates. dashed line -- 10 mM Mg, solid line -- no Mg"}
#WEB346.results$motif <- with(WEB346.results, factor(motif, levels = c("phenolic", "catecholic", "4-O-Me", "3-O-Me")))
#res.save <- WEB346.results
#WEB346.results <- res.save
themeweb <- list(theme(axis.text.x = element_text(angle=90),
                       legend.position="none"),
  facet_grid(variant~pH, labeller=mf_labeller),
  geom_line(aes(group=Mg, linetype=Mg), color="black"),
  labs(y="SAM-conversion"))

WEB346.results %<>% filter(variant != "none") %>% group_by(group, motif, pH, Mg, variant) %>% summarise(mean=mean(x.SAH), sd=sd(x.SAH))
WEB346.results %<>% ungroup %>% mutate(mean = mean/max(mean))
#EB346.results %<>% ungroup %>% mutate(mean = mean*(0.5/0.81)/0.4)
WEB346.results$mean[with(WEB346.results, which(is.na(mean)))] <- 0
WEB346.results$pH <- with(WEB346.results, factor(pH, levels = c("low", "high")))


ggplot(WEB346.results %>% filter(group != "flavonole", motif == "phenolic"), aes(x=group, y=mean)) +
  theme_bw() +  themeweb +
  geom_point(color="black") +
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=0, color="black") 
  
```

# 3'-O-methyl substrates

```{r plot_conversion_3Ome, fig.margin=T, fig.cap="Comparison of conversion of 3'-O-methyl substrates. dashed line -- 10 mM Mg, solid line -- no Mg"}
ggplot(WEB346.results %>% filter(group != "flavonole", motif == "3-O-Me"), aes(x=group, y=mean, color=Mg)) +
  theme_bw() +  themeweb +
  geom_point(color="black") +
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=0, color="black") 
  
```

# 4'-O-methyl substrates

```{r plot_conversion_4Ome, fig.margin=T, fig.cap="Comparison of conversion of 4'-O-methyl substrates. dashed line -- 10 mM Mg, solid line -- no Mg"}
ggplot(WEB346.results %>% filter(group != "flavonole", motif == "4-O-Me"), aes(x=group, y=mean, color=Mg)) +
  theme_bw() +  themeweb +
  geom_point(color="black") +
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=0, color="black") 
```

\newpage

# catecholic substrates

```{r plot_conversion_catechol, fig.margin=T, fig.cap="Comparison of conversion of catecholic substrates. dashed line -- 10 mM Mg, solid line -- no Mg"}
ggplot(WEB346.results %>% filter(group != "flavonole", motif == "catecholic"), aes(x=group, y=mean, color=Mg)) +
  theme_bw() +  themeweb +
  geom_point(color="black") +
  geom_errorbar(aes(ymin=mean-sd, ymax=mean+sd), width=0, color="black") 
```

# Calculation of all conversions

```{r, eval=T, echo=F, results='asis'}
WEB346.results <- res.save

max.nonmyr <- WEB346.results %>% filter(substrate != 17) %>% select(x.SAH) %>% max
conversions <- WEB346.results %>% filter(variant != "none") %>%
  mutate(conversion = x.SAH/max.nonmyr) %>% 
  group_by(substrate, variant) %>%
  summarise(max = {
    if(max(conversion) > 0){
      max(conversion)
    } else {return(NA)}
  }, 
  at.pH = {
    if(max(conversion) > 0){
      pH[which(conversion == max(conversion))]
    } else {return(NA)}
  }, 
  at.Mg = {
    if(max(conversion) > 0){
      Mg[which(conversion == max(conversion))]
    } else {return(NA)}})

conversions %>% filter(variant == "WT") %>% xtable
conversions %>% filter(variant != "WT") %>% xtable
```

```{r, fig.width=4, fig.height=5, fig.cap=c("Wild-type conversions. colors from light to dark: low/no, low/yes, high/no, high/yes", "Same as above, but some data is omitted.","4'-selective variant conversions. colors from light to dark: low/no, low/yes, high/no, high/yes")}
WEB346.results <- res.save

max.nonmyr <- WEB346.results %>% filter(substrate != 17) %>% select(x.SAH) %>% max
conversions <- WEB346.results %>% filter(variant != "none") %>%
  mutate(conversion = x.SAH/max.nonmyr) 

conversions$pH <- with(conversions, factor(pH, levels = c("low", "high")))

web <-   list(geom_point(aes(x=pH, y=conversion, group=Mg, linetype=Mg)), 
              geom_line(aes(x=pH, y=conversion, group=Mg, linetype=Mg)),
              facet_wrap(~substrate, ncol = 4),
              ylim(0,1.3),
              theme(legend.position="none",
                    axis.text.x = element_text(angle=90)))
withingroup <- data.frame(substrate = 1:17, with.in = c(rep(c("phenolic", "catecholic", "3O4M", "4O3M"), 2), "phenolic", 
                                                        "m-coumaric","o-coumaric", "catecholic","4O3M", "3O4M", "phenolic","catecholic","myricetin"))
withingroup$with.in <- with(withingroup, factor(with.in, levels = c("phenolic", "catecholic", "3O4M", "4O3M", "m-coumaric","o-coumaric", "myricetin")))
conversions <- merge(conversions, withingroup, by="substrate")
#ggplot(data = conversions %>% filter(variant == "WT")) +
#  web

#ggplot(data = conversions %>% filter(variant != "WT"), aes(x=pH, y=conversion, group=Mg, linetype=Mg)) +
#  web

conversions %<>% mutate(condition=paste(pH, Mg, sep="."))
conversions$condition <- with(conversions, factor(condition, levels = c("low.FALSE", "low.TRUE", "high.FALSE", "high.TRUE")))

web <- list(  geom_bar(stat = "identity", position = "dodge", aes(fill=condition)),
              facet_grid(group~., scale="free_x"),
              labs(x="motif"),
              scale_fill_grey(start=0.8, end=0.2),
              #scale_fill_brewer(palette="Set1"),
              theme_bw(),
              theme(legend.position="none",
                    axis.text.x = element_text(angle=90))
              )

ggplot(data = conversions %>% filter(variant == "WT"), aes(x=with.in, y=conversion)) +
  web

ggplot(data = conversions %>% filter(variant == "WT", with.in != "catecholic" & with.in != "myricetin"), aes(x=with.in, y=conversion)) +
  web

ggplot(data = conversions %>% filter(variant != "WT"), aes(x=with.in, y=conversion)) +
  web


ggplot(data = conversions %>% filter(variant == "WT", group != "flavonole", with.in!="catecholic"), aes(x=with.in, y=conversion)) +
  web


```

